<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"  
  "http://www.w3.org/TR/html4/loose.dtd">  
<html > 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<!-- html --> 
<meta name="src" content="Manual.tex"> 
<meta name="date" content="2014-11-23 17:41:00"> 
<link rel="stylesheet" type="text/css" href="Manual.css"> 
</head><body 
>
<!--l. 30--><p class="noindent" >
                                                                         

                                                                         
<!--l. 40--><p class="noindent" ><span 
class="cmcsc-10x-x-172">G<span 
class="small-caps">e</span><span 
class="small-caps">n</span><span 
class="small-caps">o</span><span 
class="small-caps">m</span><span 
class="small-caps">e</span> C<span 
class="small-caps">o</span><span 
class="small-caps">m</span><span 
class="small-caps">p</span><span 
class="small-caps">i</span><span 
class="small-caps">l</span><span 
class="small-caps">e</span><span 
class="small-caps">r</span></span><br /><br />
___________________________________________<br /><br />
<span 
class="cmbx-12x-x-172">User Manual</span><br /><br />
_______________________________________________<br /><br />
<span 
class="cmcsc-10x-x-144">V.0.5</span><br /><br />
<span 
class="cmr-12">23rd</span><span 
class="cmr-12">&#x00A0;November</span><span 
class="cmr-12">&#x00A0;2014</span><br /><br />
<span 
class="cmcsc-10x-x-172"><span 
class="small-caps">d</span><span 
class="small-caps">r</span><span 
class="small-caps">a</span><span 
class="small-caps">f</span><span 
class="small-caps">t</span></span><br /><br />

<img 
src="pictures/genomecompiler.png" alt="PIC"  
>
                                                                         

                                                                         
<!--l. 73--><p class="noindent" >
                                                                         

                                                                         
<h3 class="likesectionHead"><a 
 id="x1-1000"></a>Contents</h3>
<div class="tableofcontents">
<span class="sectionToc" >1 <a 
href="#x1-20001" id="QQ2-1-2">Tutorials</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.1 <a 
href="#x1-30001.1" id="QQ2-1-3">Getting Started</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.1 <a 
href="#x1-40001.1.1" id="QQ2-1-4">Creating an Account</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.2 <a 
href="#x1-50001.1.2" id="QQ2-1-6">Logging in</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.2 <a 
href="#x1-60001.2" id="QQ2-1-8">Genome Compiler User Interface</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.1 <a 
href="#x1-70001.2.1" id="QQ2-1-9">Overview of Genome Compiler User Interface</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.2 <a 
href="#x1-80001.2.2" id="QQ2-1-14">Tooltips</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.3 <a 
href="#x1-90001.2.3" id="QQ2-1-16">Minimizing Materials box and Properties</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.4 <a 
href="#x1-100001.2.4" id="QQ2-1-19">Maximizing Materials box and Properties</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.5 <a 
href="#x1-110001.2.5" id="QQ2-1-22">Re-arranging Project Tabs</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.6 <a 
href="#x1-120001.2.6" id="QQ2-1-28">Viewing Projects</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7 <a 
href="#x1-130001.2.7" id="QQ2-1-34">Abstraction Layers</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.8 <a 
href="#x1-140001.2.8" id="QQ2-1-39">Numbering of Base Pairs and Amino Acids</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.9 <a 
href="#x1-150001.2.9" id="QQ2-1-43">Properties Section</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.10 <a 
href="#x1-160001.2.10" id="QQ2-1-49">Options Menu</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.3 <a 
href="#x1-170001.3" id="QQ2-1-52">Importing and Exporting</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.1 <a 
href="#x1-180001.3.1" id="QQ2-1-53">Importing Files</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.2 <a 
href="#x1-190001.3.2" id="QQ2-1-58">Importing Parts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.3 <a 
href="#x1-200001.3.3" id="QQ2-1-61">Exporting Files</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.4 <a 
href="#x1-210001.4" id="QQ2-1-64">Editing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.1 <a 
href="#x1-220001.4.1" id="QQ2-1-65">Basic Copy/Cut/Paste/Delete/Undo/Redo</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.2 <a 
href="#x1-230001.4.2" id="QQ2-1-69">Drag and Drop Functionality</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.3 <a 
href="#x1-240001.4.3" id="QQ2-1-72">Adding New Parts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.4 <a 
href="#x1-250001.4.4" id="QQ2-1-77">Adding Amino Acids</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.5 <a 
href="#x1-260001.4.5" id="QQ2-1-79">Reverse Complement</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.6 <a 
href="#x1-270001.4.6" id="QQ2-1-82">Merging and Splitting Parts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.7 <a 
href="#x1-280001.4.7" id="QQ2-1-87">Selecting Strand</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.8 <a 
href="#x1-290001.4.8" id="QQ2-1-93">Mutation Wheel</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.4.9 <a 
href="#x1-300001.4.9" id="QQ2-1-97">Basic Local Alignment Search Tool (BLAST)</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.5 <a 
href="#x1-310001.5" id="QQ2-1-99">Go to</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.1 <a 
href="#x1-320001.5.1" id="QQ2-1-100">Openging the &#8221;Go to&#8221; Dialog</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.2 <a 
href="#x1-330001.5.2" id="QQ2-1-104">&#8221;Go to&#8221; Dialog</a></span>
                                                                         

                                                                         
<br />&#x00A0;<span class="subsectionToc" >1.6 <a 
href="#x1-340001.6" id="QQ2-1-106">Search in the Materials box</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.6.1 <a 
href="#x1-350001.6.1" id="QQ2-1-107">Searching in the Materials box</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.6.2 <a 
href="#x1-360001.6.2" id="QQ2-1-111">Example</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.7 <a 
href="#x1-370001.7" id="QQ2-1-113">Search in Projects</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.7.1 <a 
href="#x1-380001.7.1" id="QQ2-1-114">Searching in Projects</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.8 <a 
href="#x1-390001.8" id="QQ2-1-119">NCBI Search</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.8.1 <a 
href="#x1-400001.8.1" id="QQ2-1-120">Searching in the NCBI Database</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.8.2 <a 
href="#x1-410001.8.2" id="QQ2-1-124">Downloaded and Downloading Files Window</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.9 <a 
href="#x1-420001.9" id="QQ2-1-126">Sharing and Collaboration</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.9.1 <a 
href="#x1-430001.9.1" id="QQ2-1-127">Cloud Folders</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.9.2 <a 
href="#x1-440001.9.2" id="QQ2-1-129">Creating New Folders</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.9.3 <a 
href="#x1-450001.9.3" id="QQ2-1-131">Creating Subfolders</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.9.4 <a 
href="#x1-460001.9.4" id="QQ2-1-135">Sharing Cloud Folders</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.9.5 <a 
href="#x1-470001.9.5" id="QQ2-1-142">Creating New Folders via the &#8221;Cloud Folder Settings&#8221; dialog</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.10 <a 
href="#x1-480001.10" id="QQ2-1-144">User Comments</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.10.1 <a 
href="#x1-490001.10.1" id="QQ2-1-145">User Comments Indicator</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.10.2 <a 
href="#x1-500001.10.2" id="QQ2-1-147">Navigating to Comments</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.10.3 <a 
href="#x1-510001.10.3" id="QQ2-1-149">Adding Comments</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.10.4 <a 
href="#x1-520001.10.4" id="QQ2-1-154">Multiple Users making Comments in a Single Project</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.10.5 <a 
href="#x1-530001.10.5" id="QQ2-1-156">User Comments Summary Table</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.11 <a 
href="#x1-540001.11" id="QQ2-1-159">Track Changes</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.11.1 <a 
href="#x1-550001.11.1" id="QQ2-1-160">Track Changes Indicator</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.11.2 <a 
href="#x1-560001.11.2" id="QQ2-1-162">Navigating to Changes</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.11.3 <a 
href="#x1-570001.11.3" id="QQ2-1-164">Multiple Users making Changes in a Single Project</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.11.4 <a 
href="#x1-580001.11.4" id="QQ2-1-166">Track Changes Summary Table</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.12 <a 
href="#x1-590001.12" id="QQ2-1-169">Ordering and Pricing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.12.1 <a 
href="#x1-600001.12.1" id="QQ2-1-170">Pricing for Projects</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.12.2 <a 
href="#x1-610001.12.2" id="QQ2-1-173">Pricing for Primers</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.13 <a 
href="#x1-620001.13" id="QQ2-1-177">Compilation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.13.1 <a 
href="#x1-630001.13.1" id="QQ2-1-178">Selecting Rules for Activation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.13.2 <a 
href="#x1-640001.13.2" id="QQ2-1-182">Compiling Projects</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.13.3 <a 
href="#x1-650001.13.3" id="QQ2-1-185">Error and Warning Indicators</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.13.4 <a 
href="#x1-660001.13.4" id="QQ2-1-189">Compilation Summary Table</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.13.5 <a 
href="#x1-670001.13.5" id="QQ2-1-192">Navigating to Errors and Warnings</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.14 <a 
href="#x1-680001.14" id="QQ2-1-194">Back Translation</a></span>
                                                                         

                                                                         
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.14.1 <a 
href="#x1-690001.14.1" id="QQ2-1-195">How to Start Translating</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.14.2 <a 
href="#x1-700001.14.2" id="QQ2-1-198">&#8220;Translation Settings&#8221; Dialog</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.14.3 <a 
href="#x1-710001.14.3" id="QQ2-1-203">Proportion of each triplet</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.15 <a 
href="#x1-720001.15" id="QQ2-1-206">Combinatorial Design</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.15.1 <a 
href="#x1-730001.15.1" id="QQ2-1-207">Circuit Part Combinatorial</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.15.2 <a 
href="#x1-740001.15.2" id="QQ2-1-211">Protein Engineering Combinatorial</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.16 <a 
href="#x1-750001.16" id="QQ2-1-217">Managing Restriction Sites</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.16.1 <a 
href="#x1-760001.16.1" id="QQ2-1-218">Opening the Dialog</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.16.2 <a 
href="#x1-770001.16.2" id="QQ2-1-224">&#8221;Restriction Site&#8221; Settings Dialog</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.17 <a 
href="#x1-780001.17" id="QQ2-1-229">Virtual Digest</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.17.1 <a 
href="#x1-790001.17.1" id="QQ2-1-230">Running a Digest</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.17.2 <a 
href="#x1-800001.17.2" id="QQ2-1-232">&#8220;Restriction Sites&#8221; Window</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.17.3 <a 
href="#x1-810001.17.3" id="QQ2-1-234">&#8221;Digest&#8221; Summary Table</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.18 <a 
href="#x1-820001.18" id="QQ2-1-240">Primer Design</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.18.1 <a 
href="#x1-830001.18.1" id="QQ2-1-241">Opening the Primer Design Dialog</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.18.2 <a 
href="#x1-840001.18.2" id="QQ2-1-242">Primer Design Dialog</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.18.3 <a 
href="#x1-850001.18.3" id="QQ2-1-247">Primer Summary Table</a></span>
<br />&#x00A0;<span class="subsectionToc" >1.19 <a 
href="#x1-860001.19" id="QQ2-1-250">Cloning Wizard</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.1 <a 
href="#x1-870001.19.1" id="QQ2-1-251">Opening the Cloning Wizard</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.2 <a 
href="#x1-880001.19.2" id="QQ2-1-253">Cloning Procedure</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.3 <a 
href="#x1-890001.19.3" id="QQ2-1-255">In-house Cloning</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.4 <a 
href="#x1-900001.19.4" id="QQ2-1-256">In-house Cloning: Destination Vector</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.5 <a 
href="#x1-910001.19.5" id="QQ2-1-259">In-house Cloning: Select Insert</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.6 <a 
href="#x1-920001.19.6" id="QQ2-1-263">In-house Cloning: Design Insert</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.7 <a 
href="#x1-930001.19.7" id="QQ2-1-265">Design Insert: Order Insert</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.8 <a 
href="#x1-940001.19.8" id="QQ2-1-267">Design Insert: PCR Insert</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.19.9 <a 
href="#x1-950001.19.9" id="QQ2-1-272">In-house Cloning: Finalize</a></span>
<br /><span class="sectionToc" >2 <a 
href="#x1-960002" id="QQ2-1-275">Flows</a></span>
<br />&#x00A0;<span class="subsectionToc" >2.1 <a 
href="#x1-970002.1" id="QQ2-1-276">Sequence Annotation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.1.1 <a 
href="#x1-980002.1.1" id="QQ2-1-277">Pasting your Sequence</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.1.2 <a 
href="#x1-990002.1.2" id="QQ2-1-281">Annotating your Sequence</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.1.3 <a 
href="#x1-1000002.1.3" id="QQ2-1-283">Renaming your part</a></span>
<br />&#x00A0;<span class="subsectionToc" >2.2 <a 
href="#x1-1010002.2" id="QQ2-1-285">Gene Amplification</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.2.1 <a 
href="#x1-1020002.2.1" id="QQ2-1-286">Finding your Gene of Interest</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.2.2 <a 
href="#x1-1030002.2.2" id="QQ2-1-288">Commenting the Part</a></span>
                                                                         

                                                                         
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.2.3 <a 
href="#x1-1040002.2.3" id="QQ2-1-290">Designing Primers</a></span>
<br />&#x00A0;&#x00A0;<span class="subsubsectionToc" >2.2.4 <a 
href="#x1-1050002.2.4" id="QQ2-1-291">Ordering</a></span>
</div>
                                                                         

                                                                         
<h3 class="sectionHead"><span class="titlemark">1   </span> <a 
 id="x1-20001"></a>Tutorials</h3>
<!--l. 82--><p class="noindent" >
<h4 class="subsectionHead"><span class="titlemark">1.1   </span> <a 
 id="x1-30001.1"></a>Getting Started</h4>
<!--l. 84--><p class="noindent" >Welcome to Genome Compiler! This tutorial will show you how to get started with using our
software.
<!--l. 86--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.1.1   </span> <a 
 id="x1-40001.1.1"></a>Creating an Account</h5>
     <ul class="itemize1">
     <li class="itemize">If you do not have an account yet, sign up at <a 
href="https://designer.genomecompiler.com/users/sign_up" ><span 
class="cmtt-10">designer.genomecompiler.com/</span>
     <span 
class="cmtt-10">users/sign_up</span></a> (Figure&#x00A0;<a 
href="#x1-4001r1">1.1.1.1<!--tex4ht:ref: fig:sign_up --></a>). You will receive an email welcoming you to the
     software. Activate your account by clicking the link provided in the email.
     <!--l. 92--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-4001r1"></a> <img 
src="pictures/getting_started_screenshots/sign_up.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.1.1.1: </span><span  
class="content">Signing up to Genome Compiler.</span></div><!--tex4ht:label?: x1-4001r1 -->
     <!--l. 97--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 101--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.1.2   </span> <a 
 id="x1-50001.1.2"></a>Logging in</h5>
     <ul class="itemize1">
     <li class="itemize">Once  you  have  an  account,  simply  login  at  <a 
href="https://designer.genomecompiler.com/" ><span 
class="cmtt-10">designer.genomecompiler.com</span></a>
     (Figure&#x00A0;<a 
href="#x1-5001r1">1.1.2.1<!--tex4ht:ref: fig:login --></a>).
     <!--l. 108--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-5001r1"></a> <img 
src="pictures/getting_started_screenshots/login.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.1.2.1: </span><span  
class="content">Logging in to Genome Compiler.</span></div><!--tex4ht:label?: x1-5001r1 -->
     <!--l. 113--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You are now ready to go! Check out the rest of the manual to learn more about the
     different features of our software.
     </li></ul>
<!--l. 119--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.2   </span> <a 
 id="x1-60001.2"></a>Genome Compiler User Interface</h4>
<!--l. 125--><p class="noindent" >Genome Compiler offers a customizable user friendly interface.
<!--l. 127--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.1   </span> <a 
 id="x1-70001.2.1"></a>Overview of Genome Compiler User Interface</h5>
<!--l. 129--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                         

                                                                         
<a 
 id="x1-7001r1"></a>
                                                                         

                                                                         
<!--l. 130--><p class="noindent" >                                   <img 
src="pictures/UI_screenshots/general_overview.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.1.1: </span><span  
class="content">General view of the software.</span></div><!--tex4ht:label?: x1-7001r1 -->
                                                                         

                                                                         
<!--l. 133--><p class="noindent" ></div><hr class="endfigure">
     <ul class="itemize1">
     <li class="itemize">The  toolbar  at  the  top  of  the  screen  (Figure&#x00A0;<a 
href="#x1-7002r2">1.2.1.2<!--tex4ht:ref: fig:toolbar_description --></a>)  contains  icons  for  file
     operations, standard editing (see section&#x00A0;<a 
href="#x1-210001.4">1.4<!--tex4ht:ref: subsec:editing --></a>) and tools.
     <!--l. 139--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-7002r2"></a>                                                       <img 
src="pictures/UI_screenshots/toolbar_description.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.1.2: </span><span  
class="content">The toolbar.</span></div><!--tex4ht:label?: x1-7002r2 -->
     <!--l. 143--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The Materials box on the left of the screen (Figure&#x00A0;<a 
href="#x1-7003r3">1.2.1.3<!--tex4ht:ref: fig:materials_box_description --></a>) gives you access to all your
     files. The search tool helps you find your files easily (see section&#x00A0;<a 
href="#x1-340001.6">1.6<!--tex4ht:ref: subsec:search_materials_box --></a>). You can also use it
     to search through the NCBI database (see section&#x00A0;<a 
href="#x1-390001.8">1.8<!--tex4ht:ref: subsec:NCBI --></a>). The Materials box provides a
     number of icons for file operations and tools.
     <!--l. 147--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-7003r3"></a> <img 
src="pictures/UI_screenshots/materials_box_description.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.1.3: </span><span  
class="content">The Materials box.</span></div><!--tex4ht:label?: x1-7003r3 -->
     <!--l. 152--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The properties section at the bottom of the screen (Figure&#x00A0;<a 
href="#x1-7004r4">1.2.1.4<!--tex4ht:ref: fig:properties_section_description --></a>) contains different
     tabs providing information on your project, parts, selection, compilation and
     restriction digests (more information on the properties section can be found in
     section&#x00A0;<a 
href="#x1-150001.2.9">1.2.9<!--tex4ht:ref: subsubsec:properties_section --></a>).
     <!--l. 156--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-7004r4"></a>                                                       <img 
src="pictures/UI_screenshots/properties_section_description.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.1.4: </span><span  
class="content">The properties section.</span></div><!--tex4ht:label?: x1-7004r4 -->
     <!--l. 160--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.2   </span> <a 
 id="x1-80001.2.2"></a>Tooltips</h5>
     <ul class="itemize1">
     <li class="itemize">Many tooltips are available, describing dialogs, icons, etc. Simply hover over the
     item to see the tooltip. For example, information regarding the amino acids is
     displayed when you hover over them (Figure&#x00A0;<a 
href="#x1-8001r1">1.2.2.1<!--tex4ht:ref: fig:aa_tooltip --></a>).
     <!--l. 170--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-8001r1"></a> <img 
src="pictures/UI_screenshots/aa_tooltip.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.2.1: </span><span  
class="content">The tooltip describing the amino acids.</span></div><!--tex4ht:label?: x1-8001r1 -->
     <!--l. 175--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 179--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.3   </span> <a 
 id="x1-90001.2.3"></a>Minimizing Materials box and Properties</h5>
     <ul class="itemize1">
     <li class="itemize">The Materials box and the properties section can be minimized by clicking the
     appropriate icon (Figures&#x00A0;<a 
href="#x1-9001r1">1.2.3.1<!--tex4ht:ref: fig:normal --></a> and&#x00A0;<a 
href="#x1-9002r2">1.2.3.2<!--tex4ht:ref: fig:minimized --></a>).
     <!--l. 185--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-9001r1"></a> <img 
src="pictures/UI_screenshots/normal.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.3.1: </span><span  
class="content">Minimizing buttons for the Materials box and the properties section.</span></div><!--tex4ht:label?: x1-9001r1 -->
     <!--l. 190--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 192--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-9002r2"></a> <img 
src="pictures/UI_screenshots/minimized.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.3.2: </span><span  
class="content">Minimized Materials box and properties section.</span></div><!--tex4ht:label?: x1-9002r2 -->
     <!--l. 197--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 202--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.4   </span> <a 
 id="x1-100001.2.4"></a>Maximizing Materials box and Properties</h5>
     <ul class="itemize1">
     <li class="itemize">The Materials box and the properties section sizes can be adjusted (Figure&#x00A0;<a 
href="#x1-10001r1">1.2.4.1<!--tex4ht:ref: fig:normal_before_enlarged --></a>
     and&#x00A0;<a 
href="#x1-10002r2">1.2.4.2<!--tex4ht:ref: fig:enlarged --></a>).
     <!--l. 208--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-10001r1"></a> <img 
src="pictures/UI_screenshots/normal_before_enlarged.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.4.1: </span><span  
class="content">Buttons to adjust the size of the Materials box and the properties
section.</span></div><!--tex4ht:label?: x1-10001r1 -->
     <!--l. 213--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 215--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-10002r2"></a> <img 
src="pictures/UI_screenshots/enlarged.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.4.2: </span><span  
class="content">Enlarged Materials box and properties section.</span></div><!--tex4ht:label?: x1-10002r2 -->
     <!--l. 220--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 225--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.5   </span> <a 
 id="x1-110001.2.5"></a>Re-arranging Project Tabs</h5>
     <ul class="itemize1">
     <li class="itemize">Project tabs can be moved and organized. When multiple projects are open, they
     can appear in two different views. In tile view, you can adjust the size of each
     project&#8217;s pane by moving the bar in between them (Figure&#x00A0;<a 
href="#x1-11001r1">1.2.5.1<!--tex4ht:ref: fig:tabs_view1 --></a>).
     <!--l. 231--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-11001r1"></a>                                                       <img 
src="pictures/UI_screenshots/tabs_view1.png" alt="PIC"  
>
                                                                         

                                                                         
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.5.1: </span><span  
class="content">Tile view with multiple projects.</span></div><!--tex4ht:label?: x1-11001r1 -->
     <!--l. 235--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Alternatively, in full view, you can have only one project visible at a time. Navigate to
     each project by clicking the tab (Figure&#x00A0;<a 
href="#x1-11002r2">1.2.5.2<!--tex4ht:ref: fig:tabs_view2 --></a>).
     <!--l. 239--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-11002r2"></a>                                                       <img 
src="pictures/UI_screenshots/tabs_view2.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.5.2: </span><span  
class="content">Full view with multiple projects.</span></div><!--tex4ht:label?: x1-11002r2 -->
     <!--l. 243--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Navigate between both views by moving the tabs or by clicking the appropriate icon
     (Figure&#x00A0;<a 
href="#x1-11005r4">1.2.5.4<!--tex4ht:ref: fig:switch_view_icons --></a>).
     <!--l. 247--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-11005r4"></a> <!--l. 250--><p class="noindent" ><img 
src="pictures/UI_screenshots/full_view.png" alt="PIC"  
> <a 
 id="x1-11003r1"></a>
<span 
class="cmr-9">The swith to full view icon.</span>
     &#x00A0; <!--l. 257--><p class="noindent" ><img 
src="pictures/UI_screenshots/tile_view.png" alt="PIC"  
> <a 
 id="x1-11004r2"></a>
<span 
class="cmr-9">The switch to tile view icon.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.5.4: </span><span  
class="content">Switch view icons.</span></div><!--tex4ht:label?: x1-11005r4 -->
     <!--l. 262--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 266--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.6   </span> <a 
 id="x1-120001.2.6"></a>Viewing Projects</h5>
     <ul class="itemize1">
     <li class="itemize">You can choose between two different views to visualize projects. The split view
     shows DNA on the left pane and the circular or linear view on the right pane
     (Figure&#x00A0;<a 
href="#x1-12001r1">1.2.6.1<!--tex4ht:ref: fig:split_view_project --></a>). Alternatively, the expanded view allows you to select one view
     which fills the whole screen (Figure&#x00A0;<a 
href="#x1-12002r2">1.2.6.2<!--tex4ht:ref: fig:expanded_view_project --></a>).
     <!--l. 272--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-12001r1"></a>                                                       <img 
src="pictures/UI_screenshots/split_view_project.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.6.1: </span><span  
class="content">The split view.</span></div><!--tex4ht:label?: x1-12001r1 -->
     <!--l. 276--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 278--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-12002r2"></a>                                                       <img 
src="pictures/UI_screenshots/expanded_view_project.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.6.2: </span><span  
class="content">The expanded view.</span></div><!--tex4ht:label?: x1-12002r2 -->
     <!--l. 282--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Navigate between the views using the &#8221;Expand View&#8221; and &#8221;Split View&#8221; buttons
     (Figure&#x00A0;<a 
href="#x1-12005r4">1.2.6.4<!--tex4ht:ref: fig:switch_view_icons_project --></a>).
                                                                         

                                                                         
     <!--l. 286--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-12005r4"></a> <!--l. 289--><p class="noindent" ><img 
src="pictures/UI_screenshots/expand_view_button.png" alt="PIC"  
> <a 
 id="x1-12003r1"></a>
<span 
class="cmr-9">The &#8221;Expand View&#8221; button.</span>
     &#x00A0; <!--l. 296--><p class="noindent" ><img 
src="pictures/UI_screenshots/split_view_button.png" alt="PIC"  
> <a 
 id="x1-12004r2"></a>
<span 
class="cmr-9">The &#8221;Split View&#8221; button.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.6.4: </span><span  
class="content">Switch view icons.</span></div><!--tex4ht:label?: x1-12005r4 -->
     <!--l. 301--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 305--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7   </span> <a 
 id="x1-130001.2.7"></a>Abstraction Layers</h5>
     <ul class="itemize1">
     <li class="itemize">There are three abstraction layers: the circular view, the linear view and the DNA
     view. You can navigate to each layer view by clicking its icon (Figures&#x00A0;<a 
href="#x1-13001r1">1.2.7.1<!--tex4ht:ref: fig:linear_view_UI --></a>
     and&#x00A0;<a 
href="#x1-13002r2">1.2.7.2<!--tex4ht:ref: fig:dna_view_UI --></a>).
     <!--l. 311--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-13001r1"></a>                                                       <img 
src="pictures/UI_screenshots/linear_view.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.7.1: </span><span  
class="content">The linear view and its icon.</span></div><!--tex4ht:label?: x1-13001r1 -->
     <!--l. 315--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 317--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-13002r2"></a>                                                       <img 
src="pictures/UI_screenshots/dna_view.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.7.2: </span><span  
class="content">The DNA view and its icon.</span></div><!--tex4ht:label?: x1-13002r2 -->
     <!--l. 321--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The DNA view contains an amino acid layer (Figure&#x00A0;<a 
href="#x1-13003r3">1.2.7.3<!--tex4ht:ref: fig:aa_layer --></a>). Methionine is shown in
     green, stop codons are represented by a red star, and incomplete amino acids are marked
     by a red exclamation point.
     <!--l. 325--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-13003r3"></a>                                                       <img 
src="pictures/UI_screenshots/aa_layer.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.7.3: </span><span  
class="content">The amino acid layer in the DNA view.</span></div><!--tex4ht:label?: x1-13003r3 -->
     <!--l. 329--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can show or unshow the amino acid layer by checking or unchecking the box in the
     toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-13004r4">1.2.7.4<!--tex4ht:ref: fig:aa_layers_menu --></a>).
     <!--l. 333--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-13004r4"></a> <img 
src="pictures/UI_screenshots/aa_layers_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.7.4: </span><span  
class="content">The &#8221;Amino Acid&#8221; checkbox in the toolbar drop down Layers menu.</span></div><!--tex4ht:label?: x1-13004r4 -->
     <!--l. 338--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 342--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.8   </span> <a 
 id="x1-140001.2.8"></a>Numbering of Base Pairs and Amino Acids</h5>
     <ul class="itemize1">
     <li class="itemize">Base pairs and amino acids are numbered in the DNA view (Figure&#x00A0;<a 
href="#x1-14001r1">1.2.8.1<!--tex4ht:ref: fig:start --></a>). After
     each thousand, counting starts again from 0 and a label indicates the current
     thousand (Figure&#x00A0;<a 
href="#x1-14002r2">1.2.8.2<!--tex4ht:ref: fig:1000 --></a>).
     <!--l. 348--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-14001r1"></a>                                                       <img 
src="pictures/UI_screenshots/start.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.8.1: </span><span  
class="content">Numbered base pairs and amino acids in the DNA view.</span></div><!--tex4ht:label?: x1-14001r1 -->
     <!--l. 352--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">After each thousand, counting starts again for base pairs from 0 and a label indicates
     the current thousand (Figure&#x00A0;<a 
href="#x1-14002r2">1.2.8.2<!--tex4ht:ref: fig:1000 --></a>).
     <!--l. 356--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-14002r2"></a> <img 
src="pictures/UI_screenshots/1000.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.8.2: </span><span  
class="content">Numbered base pairs in the DNA view, after a thousand.</span></div><!--tex4ht:label?: x1-14002r2 -->
     <!--l. 361--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The number of the first base pair of each part is indicated in the linear view
     (Figure&#x00A0;<a 
href="#x1-14003r3">1.2.8.3<!--tex4ht:ref: fig:number_part --></a>).
     <!--l. 365--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-14003r3"></a>                                                       <img 
src="pictures/UI_screenshots/number_part.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.8.3: </span><span  
class="content">The number of the first base pair of each part in the linear view.</span></div><!--tex4ht:label?: x1-14003r3 -->
     <!--l. 369--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 375--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.9   </span> <a 
 id="x1-150001.2.9"></a>Properties Section</h5>
     <ul class="itemize1">
     <li class="itemize">The properties section at the bottom of the screen contains different tabs providing
     information on your project (Figure&#x00A0;<a 
href="#x1-15001r1">1.2.9.1<!--tex4ht:ref: fig:project_tab_UI --></a>), parts (Figure&#x00A0;<a 
href="#x1-15002r2">1.2.9.2<!--tex4ht:ref: fig:part_tab_UI --></a>) (the tab is
     labeled &#8221;Multiple&#8221; when multiple parts are selected), selection (Figure&#x00A0;<a 
href="#x1-15005r5">1.2.9.5<!--tex4ht:ref: fig:selection_tab_UI --></a>),
     compilation (Figure&#x00A0;<a 
href="#x1-15003r3">1.2.9.3<!--tex4ht:ref: fig:compilation_tab_UI --></a>) and restriction digests (Figure&#x00A0;<a 
href="#x1-15004r4">1.2.9.4<!--tex4ht:ref: fig:digest_tab_UI --></a>).
     <!--l. 383--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-15001r1"></a>                                                       <img 
src="pictures/UI_screenshots/properties_section_description.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.9.1: </span><span  
class="content">The project tab.</span></div><!--tex4ht:label?: x1-15001r1 -->
     <!--l. 387--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 389--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-15002r2"></a>                                                       <img 
src="pictures/UI_screenshots/part_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.9.2: </span><span  
class="content">The part tab.</span></div><!--tex4ht:label?: x1-15002r2 -->
                                                                         

                                                                         
     <!--l. 393--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 395--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-15003r3"></a>                                                       <img 
src="pictures/UI_screenshots/compilation_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.9.3: </span><span  
class="content">The compilation tab.</span></div><!--tex4ht:label?: x1-15003r3 -->
     <!--l. 399--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 401--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-15004r4"></a>                                                       <img 
src="pictures/UI_screenshots/digest_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.9.4: </span><span  
class="content">The digest tab.</span></div><!--tex4ht:label?: x1-15004r4 -->
     <!--l. 405--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">When you select a strand of DNA in the DNA view, the selection tab (Figure&#x00A0;<a 
href="#x1-15005r5">1.2.9.5<!--tex4ht:ref: fig:selection_tab_UI --></a>)
     displays:
         <ul class="itemize2">
         <li class="itemize">the selection count
         </li>
         <li class="itemize">the GC content
         </li>
         <li class="itemize">the melting temperature (calculated according to <a 
href="http://www.pnas.org/content/95/4/1460.full" ><span 
class="cmtt-10">SantaLucia 1998</span></a>)
         </li>
         <li class="itemize">the start of the selection
         </li>
         <li class="itemize">the end of the selection
         </li>
         <li class="itemize">the number of parts contained in the selection
         </li>
         <li class="itemize">the isolelectric point
         </li>
         <li class="itemize">the hydrophobicity
         </li>
         <li class="itemize">the molecular weight</li></ul>
     <!--l. 420--><p class="noindent" >as well as the name, linear formula and type of the amino acid, when a single amino acid
     is selected.
     <!--l. 422--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-15005r5"></a>                                                       <img 
src="pictures/UI_screenshots/selection_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.9.5: </span><span  
class="content">The selection tab.</span></div><!--tex4ht:label?: x1-15005r5 -->
     <!--l. 426--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 430--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.2.10   </span> <a 
 id="x1-160001.2.10"></a>Options Menu</h5>
                                                                         

                                                                         
     <ul class="itemize1">
     <li class="itemize">The &#8221;Options&#8221; drop down menu at the top right (Figure&#x00A0;<a 
href="#x1-16001r1">1.2.10.1<!--tex4ht:ref: fig:options_menu --></a>) allows you
     to  find  information  about  the  software,  to  give  feedback  and  to  access  your
     preferences.
     <!--l. 436--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-16001r1"></a> <img 
src="pictures/UI_screenshots/options_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.10.1: </span><span  
class="content">The &#8221;Options&#8221; drop down menu.</span></div><!--tex4ht:label?: x1-16001r1 -->
     <!--l. 441--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">By default, projects left open in previous sessions are reopened in new sessions. This can
     be changed by selecting &#8221;No&#8221; in the &#8221;Preferences&#8221; dialog (Figure&#x00A0;<a 
href="#x1-16002r2">1.2.10.2<!--tex4ht:ref: fig:preferences --></a>). You can also
     reset all dialogs by clicking &#8221;Reset all dialogs&#8221;.
     <!--l. 445--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-16002r2"></a> <img 
src="pictures/UI_screenshots/preferences.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.2.10.2: </span><span  
class="content">The &#8221;Preferences&#8221; dialog.</span></div><!--tex4ht:label?: x1-16002r2 -->
     <!--l. 450--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 455--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.3   </span> <a 
 id="x1-170001.3"></a>Importing and Exporting</h4>
<!--l. 460--><p class="noindent" >Genome Compiler supports import of Genome Compiler (.gcproj), INSD (.xml), GenBank
(.gb), FASTA (.fasta, .fas, .fa), Raw DNA (.txt), Geneious (.geneious) and Vector NTI archive
(.ma4) file formats. Genome Compiler supports the export of Genome Compiler, GenBank,
FASTA and Raw DNA file formats.
<!--l. 462--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.3.1   </span> <a 
 id="x1-180001.3.1"></a>Importing Files</h5>
     <ul class="itemize1">
     <li class="itemize">To import a file, click the &#8220;Import File&#8221; icon in the Materials box (Figure&#x00A0;<a 
href="#x1-18001r1">1.3.1.1<!--tex4ht:ref: fig:materials_import_icon --></a>).
     <!--l. 468--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-18001r1"></a> <img 
src="pictures/importexport_screenshots/materials_import_icon.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.1.1: </span><span  
class="content">The &#8221;Import File&#8221; icon in the Materials box.</span></div><!--tex4ht:label?: x1-18001r1 -->
     <!--l. 473--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">A dialog will pop up (Figure&#x00A0;<a 
href="#x1-18002r2">1.3.1.2<!--tex4ht:ref: fig:computer_files --></a>) to allow you to choose the file from your computer
     files.
     <!--l. 477--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-18002r2"></a>                                                       <img 
src="pictures/importexport_screenshots/computer_files.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.1.2: </span><span  
class="content">Choosing a file from your computer files.</span></div><!--tex4ht:label?: x1-18002r2 -->
     <!--l. 481--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Once you selected a file, a dialog will pop up (Figure&#x00A0;<a 
href="#x1-18003r3">1.3.1.3<!--tex4ht:ref: fig:save_cloud_dialog --></a>) to prompt you for a
     location. You can choose to save the file in an existing folder or to create a new
     folder.
     <!--l. 486--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-18003r3"></a> <img 
src="pictures/importexport_screenshots/save_cloud_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.1.3: </span><span  
class="content">The &#8221;Save on Cloud&#8221; dialog when importing a file.</span></div><!--tex4ht:label?: x1-18003r3 -->
     <!--l. 491--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To create a new folder, click &#8220;New Folder&#8221;. A dialog will pop up (Figure&#x00A0;<a 
href="#x1-18004r4">1.3.1.4<!--tex4ht:ref: fig:new_folder_file --></a>) to
     prompt you for a name and an optional folder description. Save your file by clicking
     &#8220;Save&#8221;. The file will now be saved as a new project in Genome Compiler and can be
     opened from the Materials box.
     <!--l. 495--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-18004r4"></a> <img 
src="pictures/importexport_screenshots/new_folder_file.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.1.4: </span><span  
class="content">The &#8221;Save on Cloud&#8221; dialog when creating a new folder.</span></div><!--tex4ht:label?: x1-18004r4 -->
     <!--l. 500--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 504--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.3.2   </span> <a 
 id="x1-190001.3.2"></a>Importing Parts</h5>
     <ul class="itemize1">
     <li class="itemize">To import a part, select &#8220;Parts&#8221; from the import drop down menu (Figure&#x00A0;<a 
href="#x1-19001r1">1.3.2.1<!--tex4ht:ref: fig:materials_import_parts --></a>).
     <!--l. 511--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-19001r1"></a> <img 
src="pictures/importexport_screenshots/materials_import_parts.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.2.1: </span><span  
class="content">The import drop down menu in the Materials box.</span></div><!--tex4ht:label?: x1-19001r1 -->
     <!--l. 516--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8220;Import Genetic Parts&#8221; dialog will appear (Figure&#x00A0;<a 
href="#x1-19002r2">1.3.2.2<!--tex4ht:ref: fig:import_genetic_parts_dialog --></a>). The &#8220;Create New Part&#8221;
     icon allows you to add a new part to the list. Select the appropriate feature from the
     &#8220;Feature Key&#8221; drop down menu and enter the part name and sequence. Click the
     &#8220;Delete&#8221; icon to delete a part.
     <!--l. 520--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-19002r2"></a> <img 
src="pictures/importexport_screenshots/import_genetic_parts_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.2.2: </span><span  
class="content">The &#8221;Import Genetic Parts&#8221; dialog and its &#8221;Feature Key&#8221; drop down
menu.</span></div><!--tex4ht:label?: x1-19002r2 -->
     <!--l. 525--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Import the listed parts by clicking &#8220;Import&#8221;. The flow for saving parts is the same for
     saving files. Just like imported files, the imported part is saved as a new project in the
     Materials box.
     </li></ul>
<!--l. 530--><p class="noindent" >
                                                                         

                                                                         
<h5 class="subsubsectionHead"><span class="titlemark">1.3.3   </span> <a 
 id="x1-200001.3.3"></a>Exporting Files</h5>
     <ul class="itemize1">
     <li class="itemize">To  export  a  file,  select  it  and  select  the  appropriate  file  format  from  the
     &#8220;Export&#8221; drop down menu in the Materials box (Figures&#x00A0;<a 
href="#x1-20001r1">1.3.3.1<!--tex4ht:ref: fig:export_dropdown --></a>) or in the toolbar
     (Figures&#x00A0;<a 
href="#x1-20002r2">1.3.3.2<!--tex4ht:ref: fig:export_toolbar --></a>). A dialog will pop up to prompt you for a name and a location.
     <!--l. 538--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-20001r1"></a> <img 
src="pictures/importexport_screenshots/export_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.3.1: </span><span  
class="content">The &#8220;Export&#8221; drop down menu in the Materials box.</span></div><!--tex4ht:label?: x1-20001r1 -->
     <!--l. 543--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 545--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-20002r2"></a> <img 
src="pictures/importexport_screenshots/export_toolbar.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.3.3.2: </span><span  
class="content">The &#8220;Export&#8221; drop down menu in the toolbar.</span></div><!--tex4ht:label?: x1-20002r2 -->
     <!--l. 550--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 553--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.4   </span> <a 
 id="x1-210001.4"></a>Editing</h4>
<!--l. 558--><p class="noindent" >All the standard editing tools are available in the software. You can copy, cut, paste, delete,
drag, drop, undo and redo, as well as perform reverse complements, create, merge and split
parts or manage DNA strands.
<!--l. 560--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.1   </span> <a 
 id="x1-220001.4.1"></a>Basic Copy/Cut/Paste/Delete/Undo/Redo</h5>
     <ul class="itemize1">
     <li class="itemize">You can copy and paste, cut and paste or delete parts, amino acids or base pairs
     within a project or in between different projects. Right click on the sequence and
     select from the drop down menu (Figure&#x00A0;<a 
href="#x1-22001r1">1.4.1.1<!--tex4ht:ref: fig:dropdown_menu --></a>), or use the icons in the toolbar
     (Figure&#x00A0;<a 
href="#x1-22002r2">1.4.1.2<!--tex4ht:ref: fig:toolbar_copy_etc --></a>) or the standard keyboard shortcuts. You can paste strands of
     DNA from external sources.
     <!--l. 567--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-22001r1"></a> <img 
src="pictures/editing_screenshots/dropdown_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.1.1: </span><span  
class="content">Cut, copy, paste and delete in the right click drop down menu.</span></div><!--tex4ht:label?: x1-22001r1 -->
     <!--l. 572--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 574--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-22002r2"></a> <img 
src="pictures/editing_screenshots/toolbar_copy_etc.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.1.2: </span><span  
class="content">Cut, copy, paste and delete icons in the toolbar.</span></div><!--tex4ht:label?: x1-22002r2 -->
     <!--l. 579--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can undo or redo changes via the icons in the toolbar (Figure&#x00A0;<a 
href="#x1-22003r3">1.4.1.3<!--tex4ht:ref: fig:undo_redo --></a>) or the
     standard keyboard shortcuts.
     <!--l. 583--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-22003r3"></a> <img 
src="pictures/editing_screenshots/undo_redo.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.1.3: </span><span  
class="content">Undo and redo icons in the toolbar.</span></div><!--tex4ht:label?: x1-22003r3 -->
     <!--l. 588--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 592--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.2   </span> <a 
 id="x1-230001.4.2"></a>Drag and Drop Functionality</h5>
     <ul class="itemize1">
     <li class="itemize">In addition to standard copy, cut, paste, delete, Genome Compiler also has drag
     and drop functionality.
     </li>
     <li class="itemize">You can drag and drop projects from the Materials box to open projects, or from
     open projects to other open projects.
                                                                         

                                                                         
     </li>
     <li class="itemize">You can also drag and drop parts, amino acids or base pairs within a project and
     between projects.
     </li>
     <li class="itemize">You can create a new project by dragging a selection to the toolbar (Figures&#x00A0;<a 
href="#x1-23001r1">1.4.2.1<!--tex4ht:ref: fig:drag --></a>
     and&#x00A0;<a 
href="#x1-23002r2">1.4.2.2<!--tex4ht:ref: fig:drop --></a>).
     <!--l. 604--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-23001r1"></a> <img 
src="pictures/editing_screenshots/drag.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.2.1: </span><span  
class="content">Drag and drop a selection to the toolbar to create a new project.</span></div><!--tex4ht:label?: x1-23001r1 -->
     <!--l. 609--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 611--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-23002r2"></a> <img 
src="pictures/editing_screenshots/drop.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.2.2: </span><span  
class="content">New project created from a dropped selection.</span></div><!--tex4ht:label?: x1-23002r2 -->
     <!--l. 616--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 620--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.3   </span> <a 
 id="x1-240001.4.3"></a>Adding New Parts</h5>
     <ul class="itemize1">
     <li class="itemize">You can add new parts to your projects. Right click where you would like your
     new part to be and select &#8220;New part&#8221; from the drop down menu (Figure&#x00A0;<a 
href="#x1-24001r1">1.4.3.1<!--tex4ht:ref: fig:new_part_dropdown --></a>),
     or click the &#8220;Create New Part&#8221; icon in the toolbar (Figure&#x00A0;<a 
href="#x1-24002r2">1.4.3.2<!--tex4ht:ref: fig:new_part_icon --></a>).
     <!--l. 626--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-24001r1"></a> <img 
src="pictures/editing_screenshots/new_part_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.3.1: </span><span  
class="content">&#8221;New part&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-24001r1 -->
     <!--l. 631--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 633--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-24002r2"></a> <img 
src="pictures/editing_screenshots/new_part_icon.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.3.2: </span><span  
class="content">The &#8221;New part&#8221; icon in the toolbar.</span></div><!--tex4ht:label?: x1-24002r2 -->
     <!--l. 638--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">This will create an empty part (Figure&#x00A0;<a 
href="#x1-24003r3">1.4.3.3<!--tex4ht:ref: fig:empty_part_parts_view --></a>) which can be edited in the DNA view
     (Figure&#x00A0;<a 
href="#x1-24004r4">1.4.3.4<!--tex4ht:ref: fig:empty_part_dna_view --></a>).
     <!--l. 642--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-24003r3"></a> <img 
src="pictures/editing_screenshots/empty_part_parts_view.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.3.3: </span><span  
class="content">New empty part in the linear view.</span></div><!--tex4ht:label?: x1-24003r3 -->
     <!--l. 647--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 649--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-24004r4"></a> <img 
src="pictures/editing_screenshots/empty_part_dna_view.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.3.4: </span><span  
class="content">New empty part in the DNA view.</span></div><!--tex4ht:label?: x1-24004r4 -->
     <!--l. 654--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 658--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.4   </span> <a 
 id="x1-250001.4.4"></a>Adding Amino Acids</h5>
     <ul class="itemize1">
     <li class="itemize">You can add amino acids in your projects. Right click on the amino acid after which
     you would like the amino acids to be added and select &#8220;Insert Amino Acids...&#8221;
     from the drop down menu (Figure&#x00A0;<a 
href="#x1-25001r1">1.4.4.1<!--tex4ht:ref: fig:insert_aa_dropdown --></a>).
     <!--l. 664--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-25001r1"></a> <img 
src="pictures/editing_screenshots/insert_aa_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.4.1: </span><span  
class="content">&#8221;Insert Amino Acids&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-25001r1 -->
     <!--l. 669--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Insert Amino Acid&#8221; dialog will appear. Type in the amino acids (use the 1 letter
     representation).
     </li>
     <li class="itemize">The &#8221;Translation Settings&#8221; dialog will appear. You can choose to translate according to
     codon usage or to a translation table. For more details about translation see
     section&#x00A0;<a 
href="#x1-680001.14">1.14<!--tex4ht:ref: subsec:translation --></a>.
     </li></ul>
<!--l. 681--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.5   </span> <a 
 id="x1-260001.4.5"></a>Reverse Complement</h5>
     <ul class="itemize1">
     <li class="itemize">To perform a reverse complement, right click on the part and select &#8220;Reverse
     complement&#8221; from the drop down menu (Figure&#x00A0;<a 
href="#x1-26001r1">1.4.5.1<!--tex4ht:ref: fig:reverse_compl_dropdown --></a>), or click the &#8220;Reverse
     Complement&#8221; icon in the toolbar (Figure&#x00A0;<a 
href="#x1-26002r2">1.4.5.2<!--tex4ht:ref: fig:reverse_compl_toolbar --></a>).
     <!--l. 687--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-26001r1"></a> <img 
src="pictures/editing_screenshots/reverse_compl_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.5.1: </span><span  
class="content">&#8221;Reverse complement&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-26001r1 -->
     <!--l. 692--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 694--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-26002r2"></a> <img 
src="pictures/editing_screenshots/reverse_compl_toolbar.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.5.2: </span><span  
class="content">The &#8221;Reverse Complement&#8221; icon in the toolbar.</span></div><!--tex4ht:label?: x1-26002r2 -->
     <!--l. 699--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 703--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.6   </span> <a 
 id="x1-270001.4.6"></a>Merging and Splitting Parts</h5>
                                                                         

                                                                         
     <ul class="itemize1">
     <li class="itemize">You can merge and split parts. To merge parts, select them and select &#8220;Merge&#8221; in
     the toolbar (Figure&#x00A0;<a 
href="#x1-27001r1">1.4.6.1<!--tex4ht:ref: fig:merge --></a>), or right click and select &#8220;Merge selected parts&#8221; from
     the drop down menu (Figure&#x00A0;<a 
href="#x1-27002r2">1.4.6.2<!--tex4ht:ref: fig:merge_right_click --></a>). A new merged part containing the selection
     will be created.
     <!--l. 709--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-27001r1"></a> <img 
src="pictures/editing_screenshots/merge.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.6.1: </span><span  
class="content">The &#8221;Merge&#8221; icon in the toolbar.</span></div><!--tex4ht:label?: x1-27001r1 -->
     <!--l. 714--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 716--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-27002r2"></a> <img 
src="pictures/editing_screenshots/merge_right_click.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.6.2: </span><span  
class="content">&#8221;Merge selected parts&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-27002r2 -->
     <!--l. 721--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To split a sequence, switch to DNA view and place the caret where you would like to
     split the sequence. Select &#8220;Split&#8221; in the toolbar (Figure&#x00A0;<a 
href="#x1-27003r3">1.4.6.3<!--tex4ht:ref: fig:split --></a>), or right click and select
     &#8220;Split part&#8221; from the dropdown menu (Figure&#x00A0;<a 
href="#x1-27004r4">1.4.6.4<!--tex4ht:ref: fig:split_right_click --></a>). New split parts will be
     created.
     <!--l. 725--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-27003r3"></a> <img 
src="pictures/editing_screenshots/split.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.6.3: </span><span  
class="content">The &#8221;Split&#8221; icon in the toolbar.</span></div><!--tex4ht:label?: x1-27003r3 -->
     <!--l. 730--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 732--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-27004r4"></a> <img 
src="pictures/editing_screenshots/split_right_click.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.6.4: </span><span  
class="content">&#8221;Split part&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-27004r4 -->
     <!--l. 737--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 741--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.7   </span> <a 
 id="x1-280001.4.7"></a>Selecting Strand</h5>
     <ul class="itemize1">
     <li class="itemize">You can select DNA strands using the &#8221;Minus strand&#8221; and &#8221;Plus strand&#8221; icons in
     the &#8221;Part&#8221; tab (Figure&#x00A0;<a 
href="#x1-28001r1">1.4.7.1<!--tex4ht:ref: fig:select_strand --></a>).
     <!--l. 747--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-28001r1"></a> <img 
src="pictures/editing_screenshots/select_strand.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.7.1: </span><span  
class="content">The &#8221;Minus strand&#8221; and &#8221;Plus strand&#8221; icons in the &#8221;Part&#8221; tab.</span></div><!--tex4ht:label?: x1-28001r1 -->
     <!--l. 752--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The selected strand will be indicated by a darker color, as well as identified by an arrow
     (Figure&#x00A0;<a 
href="#x1-28004r3">1.4.7.3<!--tex4ht:ref: fig:strands --></a>).
     <!--l. 756--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-28004r3"></a><!--l. 769--><p class="noindent" ><img 
src="pictures/editing_screenshots/plus_strand.png" alt="PIC"  
> <a 
 id="x1-28002r1"></a>
<span 
class="cmr-9">Plus strand selected.</span>                                               &#x00A0; <!--l. 769--><p class="noindent" ><img 
src="pictures/editing_screenshots/minus_strand.png" alt="PIC"  
> <a 
 id="x1-28003r2"></a>
<span 
class="cmr-9">Minus strand selected.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.7.3: </span><span  
class="content">DNA strands visualization.</span></div><!--tex4ht:label?: x1-28004r3 -->
     <!--l. 771--><p class="noindent" ></div><hr class="endfigure">
                                                                         

                                                                         
     </li>
     <li class="itemize">You can show or unshow the minus strand by checking or unchecking the box in the
     toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-28005r4">1.4.7.4<!--tex4ht:ref: fig:minus_strand_layers_menu --></a>).
     <!--l. 775--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-28005r4"></a> <img 
src="pictures/editing_screenshots/minus_strand_layers_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.7.4: </span><span  
class="content">The &#8221;Minus Strand&#8221; checkbox in the toolbar drop down Layers menu.</span></div><!--tex4ht:label?: x1-28005r4 -->
     <!--l. 780--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 784--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.8   </span> <a 
 id="x1-290001.4.8"></a>Mutation Wheel</h5>
     <ul class="itemize1">
     <li class="itemize">The mutation wheel allows you to mutate amino acids. To access the mutation
     wheel, click the drop down arrow below the amino acid and select &#8221;Mutate...&#8221;
     (Figure&#x00A0;<a 
href="#x1-29001r1">1.4.8.1<!--tex4ht:ref: fig:access_mutation_wheel --></a>) or right click on the amino acid and select &#8221;Mutate...&#8221; from the
     drop down menu (Figure&#x00A0;<a 
href="#x1-29002r2">1.4.8.2<!--tex4ht:ref: fig:access_mutation_wheel_right_click --></a>).
     <!--l. 790--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-29001r1"></a> <img 
src="pictures/editing_screenshots/access_mutation_wheel.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.8.1: </span><span  
class="content">&#8221;Mutate...&#8221; in the drop down arrow menu.</span></div><!--tex4ht:label?: x1-29001r1 -->
     <!--l. 795--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 797--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-29002r2"></a> <img 
src="pictures/editing_screenshots/access_mutation_wheel_right_click.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.8.2: </span><span  
class="content">&#8221;Mutate&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-29002r2 -->
     <!--l. 802--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Select amino acids from the mutation wheel (Figure&#x00A0;<a 
href="#x1-29003r3">1.4.8.3<!--tex4ht:ref: fig:mutation_wheel --></a>) and click the green tick.
     You can filter selection of amino acids with a specific property by checking the property
     checkbox on the left.
     <!--l. 806--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-29003r3"></a> <img 
src="pictures/editing_screenshots/mutation_wheel.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.8.3: </span><span  
class="content">The amino acids mutation wheel.</span></div><!--tex4ht:label?: x1-29003r3 -->
     <!--l. 811--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 815--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.4.9   </span> <a 
 id="x1-300001.4.9"></a>Basic Local Alignment Search Tool (BLAST)</h5>
     <ul class="itemize1">
     <li class="itemize">You can send sequences to Basic Local Alignment Search Tool (BLAST) directly
     from the software. Select a sequence in the DNA view or a part in the linear view,
     right click and select &#8221;Send to BLAST&#8221; from the drop down menu (Figure&#x00A0;<a 
href="#x1-30001r1">1.4.9.1<!--tex4ht:ref: fig:send_to_blast --></a>).
     <!--l. 821--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-30001r1"></a> <img 
src="pictures/editing_screenshots/send_to_blast.png" alt="PIC"  
>
                                                                         

                                                                         
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.4.9.1: </span><span  
class="content">&#8221;Send to BLAST&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-30001r1 -->
     <!--l. 826--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">This will open the BLAST website (<a 
href="http://blast.ncbi.nlm.nih.gov" ><span 
class="cmtt-10">blast.ncbi.nlm.nih.gov</span></a>) with the selected
     sequence pasted.
     </li></ul>
<!--l. 831--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.5   </span> <a 
 id="x1-310001.5"></a>Go to</h4>
<!--l. 836--><p class="noindent" >The go to feature allows you to navigate easily through projects.
     <h5 class="subsubsectionHead"><span class="titlemark">1.5.1   </span> <a 
 id="x1-320001.5.1"></a>Openging the &#8221;Go to&#8221; Dialog</h5>To open the &#8220;Go to&#8221; dialog, right click
     on your project canvas in the protein or DNA view and select &#8220;Go to&#x2026;&#8221; or
     &#8220;Select range&#x2026;&#8221; (Figure&#x00A0;<a 
href="#x1-32003r2">1.5.1.2<!--tex4ht:ref: fig:access_goto --></a>), or use the keyboard shortcut &#8221;control G&#8221;. <hr class="figure"><div class="figure" 
><a 
 id="x1-32003r2"></a>
     <!--l. 847--><p class="noindent" ><img 
src="pictures/goto_screenshots/goto_dropdown.png" alt="PIC"  
> <a 
 id="x1-32001r1"></a>
<span 
class="cmr-9">&#8221;Go to...&#8221;</span>
     &#x00A0; <!--l. 854--><p class="noindent" ><img 
src="pictures/goto_screenshots/select_range.png" alt="PIC"  
> <a 
 id="x1-32002r2"></a>
<span 
class="cmr-9">&#8221;Select range...&#8221;</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.5.1.2: </span><span  
class="content">&#8221;Go to...&#8221; and &#8221;Select range...&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-32003r2 -->
     <ul class="itemize1">
     <li class="itemize"></div><hr class="endfigure">
     </li></ul>
<!--l. 863--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.5.2   </span> <a 
 id="x1-330001.5.2"></a>&#8221;Go to&#8221; Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8220;Go to&#8221; dialog will pop up (Figure&#x00A0;<a 
href="#x1-33001r1">1.5.2.1<!--tex4ht:ref: fig:goto_dialog --></a>) and prompt you for a nucleotide
     number. You can enter a single number or a range for selection. Clicking &#8220;Go&#8221; will
     take you to the sequence.
     <!--l. 870--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-33001r1"></a> <img 
src="pictures/goto_screenshots/goto_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.5.2.1: </span><span  
class="content">The &#8221;Go to&#8221; dialog.</span></div><!--tex4ht:label?: x1-33001r1 -->
     <!--l. 875--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 878--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.6   </span> <a 
 id="x1-340001.6"></a>Search in the Materials box</h4>
<!--l. 883--><p class="noindent" >The search bar allows you to search through your projects in the Materials box. The search is
performed not only based on the projects names, but also according to the projects meta
data.
     <h5 class="subsubsectionHead"><span class="titlemark">1.6.1   </span> <a 
 id="x1-350001.6.1"></a>Searching in the Materials box</h5>Type a keyword in the search bar. Results
     will appear below (Figure&#x00A0;<a 
href="#x1-35001r1">1.6.1.1<!--tex4ht:ref: fig:search_name --></a>). <hr class="figure"><div class="figure" 
><a 
 id="x1-35001r1"></a> <img 
src="pictures/search_materialsbox_screenshots/search_name.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.6.1.1: </span><span  
class="content">Search for a keyword in the Materials box.</span></div><!--tex4ht:label?: x1-35001r1 -->
     <ul class="itemize1">
     <li class="itemize"></div><hr class="endfigure">
     </li>
     <li class="itemize">The search tool will look for your keyword in everything contained in the project meta
     data, including its name, comments, etc. Meta data for each project can be found in the
     project tab (Figure&#x00A0;<a 
href="#x1-35002r2">1.6.1.2<!--tex4ht:ref: fig:project_tab --></a>) located along the bottom of each opened project
     or in a tooltip while hovering over the closed project in the Materials box
     (Figure&#x00A0;<a 
href="#x1-35003r3">1.6.1.3<!--tex4ht:ref: fig:tooltip --></a>).
     <!--l. 900--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-35002r2"></a>                                                       <img 
src="pictures/search_materialsbox_screenshots/project_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.6.1.2: </span><span  
class="content">The project tab showing the project meta data.</span></div><!--tex4ht:label?: x1-35002r2 -->
     <!--l. 904--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 906--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-35003r3"></a> <img 
src="pictures/search_materialsbox_screenshots/tooltip.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.6.1.3: </span><span  
class="content">Tooltip showing the project meta data.</span></div><!--tex4ht:label?: x1-35003r3 -->
     <!--l. 911--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 915--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.6.2   </span> <a 
 id="x1-360001.6.2"></a>Example</h5>
     <ul class="itemize1">
     <li class="itemize">For example, you can find the genome of this strand of E.coli by looking for the
     GI number that appears in the comment section (Figure&#x00A0;<a 
href="#x1-36001r1">1.6.2.1<!--tex4ht:ref: fig:search_comments --></a>).
     <!--l. 922--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-36001r1"></a> <img 
src="pictures/search_materialsbox_screenshots/search_comments.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.6.2.1: </span><span  
class="content">Example: search for the GI number.</span></div><!--tex4ht:label?: x1-36001r1 -->
     <!--l. 927--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 932--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.7   </span> <a 
 id="x1-370001.7"></a>Search in Projects</h4>
<!--l. 937--><p class="noindent" >The search bar allows you to search for DNA sequences and for part meta data within your
projects.
<!--l. 939--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.7.1   </span> <a 
 id="x1-380001.7.1"></a>Searching in Projects</h5>
     <ul class="itemize1">
     <li class="itemize">Type a keyword or a DNA sequence in the search bar (Figure&#x00A0;<a 
href="#x1-38001r1">1.7.1.1<!--tex4ht:ref: fig:search_bar --></a>) in the top
     right of the screen and click enter. The arrows allow you to navigate through the
     results.
     <!--l. 945--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-38001r1"></a> <img 
src="pictures/search_project_screenshots/search_bar.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.7.1.1: </span><span  
class="content">The search bar in the top right of the screen.</span></div><!--tex4ht:label?: x1-38001r1 -->
     <!--l. 950--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The search tool will look for your keyword in everything contained in the part
     meta data, including its name, comments, etc. Meta data for each part can
     be found in the part tab (Figure&#x00A0;<a 
href="#x1-38002r2">1.7.1.2<!--tex4ht:ref: fig:part_tab --></a>) located along the bottom of each
     opened project or in a tooltip while hovering over the part (Figure&#x00A0;<a 
href="#x1-38003r3">1.7.1.3<!--tex4ht:ref: fig:tooltip_part --></a>). For
     example in this case, the keyword &#8221;transposase&#8221; was found in the &#8220;Product&#8221;
     field.
     <!--l. 954--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-38002r2"></a>                                                       <img 
src="pictures/search_project_screenshots/part_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.7.1.2: </span><span  
class="content">Part tab showing the part meta data.</span></div><!--tex4ht:label?: x1-38002r2 -->
     <!--l. 958--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 960--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-38003r3"></a> <img 
src="pictures/search_project_screenshots/tooltip.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.7.1.3: </span><span  
class="content">Tooltip showing the part meta data.</span></div><!--tex4ht:label?: x1-38003r3 -->
     <!--l. 965--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">DNA sequences are searched for on both strands (Figure&#x00A0;<a 
href="#x1-38004r4">1.7.1.4<!--tex4ht:ref: fig:search_dna --></a>).
     <!--l. 969--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-38004r4"></a> <img 
src="pictures/search_project_screenshots/search_dna.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.7.1.4: </span><span  
class="content">Search for DNA sequences using the toolbar.</span></div><!--tex4ht:label?: x1-38004r4 -->
     <!--l. 974--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 977--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.8   </span> <a 
 id="x1-390001.8"></a>NCBI Search</h4>
<!--l. 983--><p class="noindent" >The NCBI search tool allows you to search through the NCBI database directly from our
software.
<!--l. 985--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.8.1   </span> <a 
 id="x1-400001.8.1"></a>Searching in the NCBI Database</h5>
     <ul class="itemize1">
     <li class="itemize">Type a keyword in the search bar and click the NCBI icon (Figure&#x00A0;<a 
href="#x1-40001r1">1.8.1.1<!--tex4ht:ref: fig:searchbox --></a>).
     <!--l. 991--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-40001r1"></a> <img 
src="pictures/NCBI_screenshots/searchbox.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.8.1.1: </span><span  
class="content">The NCBI icon and the search bar in the Materials box.</span></div><!--tex4ht:label?: x1-40001r1 -->
     <!--l. 996--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">NCBI search results will appear (Figure&#x00A0;<a 
href="#x1-40002r2">1.8.1.2<!--tex4ht:ref: fig:NCBI_results --></a>). The icon on the right of each result
     takes you to the NCBI website. To download the file, click the green &#8220;import&#8221; icon. To
     close the NCBI search results, click the NCBI icon.
     <!--l. 1000--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-40002r2"></a>                                                       <img 
src="pictures/NCBI_screenshots/NCBI_results.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.8.1.2: </span><span  
class="content">NCBI search results, with - from left to right - the NCBI icon, the green
&#8221;import&#8221; icon and the &#8221;See item in the NCBI website&#8221; icon.</span></div><!--tex4ht:label?: x1-40002r2 -->
     <!--l. 1004--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The downloaded file will be automatically added to the cloud in the Materials box
     (Figure&#x00A0;<a 
href="#x1-40003r3">1.8.1.3<!--tex4ht:ref: fig:sequence_cloud --></a>).
     <!--l. 1008--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-40003r3"></a> <img 
src="pictures/NCBI_screenshots/sequence_cloud.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.8.1.3: </span><span  
class="content">The downloaded file in the Materials box.</span></div><!--tex4ht:label?: x1-40003r3 -->
     <!--l. 1013--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">If you have already found what you are looking for in the NCBI database, you can copy
     the name or the GI number, and paste it in the search bar (Figure&#x00A0;<a 
href="#x1-40001r1">1.8.1.1<!--tex4ht:ref: fig:searchbox --></a>). Click the
     NCBI icon, and download the file.
     </li></ul>
<!--l. 1019--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.8.2   </span> <a 
 id="x1-410001.8.2"></a>Downloaded and Downloading Files Window</h5>
                                                                         

                                                                         
     <ul class="itemize1">
     <li class="itemize">Downloaded   and   downloading   files   appear   in   a   window   at   the   bottom
     (Figure&#x00A0;<a 
href="#x1-41001r1">1.8.2.1<!--tex4ht:ref: fig:whole_screen_import --></a>). You can remove files from the list by clicking the &#8220;remove&#8221; icon.
     The  &#8220;view  in  NCBI&#8221;  icon  takes  you  to  the  corresponding  page  on  the  NCBI
     website. Click the &#8220;view in the Materials box&#8221; icon on the left of the downloaded
     project to view the file in the Materials box.
     <!--l. 1025--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-41001r1"></a>                                                       <img 
src="pictures/NCBI_screenshots/whole_screen_import.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.8.2.1: </span><span  
class="content">The downloaded and downloading files window.</span></div><!--tex4ht:label?: x1-41001r1 -->
     <!--l. 1029--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1041--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.9   </span> <a 
 id="x1-420001.9"></a>Sharing and Collaboration</h4>
<!--l. 1045--><p class="noindent" >The Cloud allows users to access projects from anywhere and share them with others.
<!--l. 1047--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.9.1   </span> <a 
 id="x1-430001.9.1"></a>Cloud Folders</h5>
     <ul class="itemize1">
     <li class="itemize">Folders saved in the cloud appear in the Materials box and are marked with a cloud
     icon. Refresh the cloud by clicking the &#8220;Refresh Cloud&#8221; icon (Figure&#x00A0;<a 
href="#x1-43001r1">1.9.1.1<!--tex4ht:ref: fig:materials_box_cloud --></a>).
     <!--l. 1053--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-43001r1"></a> <img 
src="pictures/cloud_screenshots/materials_box_cloud.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.1.1: </span><span  
class="content">The &#8221;Refresh Cloud&#8221; icon in the Materials box.</span></div><!--tex4ht:label?: x1-43001r1 -->
     <!--l. 1058--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1063--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.9.2   </span> <a 
 id="x1-440001.9.2"></a>Creating New Folders</h5>
     <ul class="itemize1">
     <li class="itemize">To create a new folder, right click and select &#8220;New Cloud Folder&#8221; (Figure&#x00A0;<a 
href="#x1-44001r1">1.9.2.1<!--tex4ht:ref: fig:right_click_cloud --></a>).
     You can also create new folders via the &#8221;Cloud Folder Settings&#8221; dialog as explained
     in section&#x00A0;<a 
href="#x1-470001.9.5">1.9.5<!--tex4ht:ref: subsubsec:creating_new_folders_via_cloud_folder_settings_dialog --></a>.
     <!--l. 1069--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-44001r1"></a> <img 
src="pictures/cloud_screenshots/right_click_cloud.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.2.1: </span><span  
class="content">&#8221;New Cloud Folder&#8221; in the right click menu.</span></div><!--tex4ht:label?: x1-44001r1 -->
     <!--l. 1074--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1078--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.9.3   </span> <a 
 id="x1-450001.9.3"></a>Creating Subfolders</h5>
     <ul class="itemize1">
     <li class="itemize">You can also create subfolders. Click the &#8220;New Folder&#8221; icon in the Materials box,
     or right click and select &#8220;New Folder&#8221; (Figure&#x00A0;<a 
href="#x1-45003r2">1.9.3.2<!--tex4ht:ref: fig:subfolders --></a>).
     <!--l. 1084--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-45003r2"></a> <!--l. 1087--><p class="noindent" ><img 
src="pictures/cloud_screenshots/new_folder_icon.png" alt="PIC"  
> <a 
 id="x1-45001r1"></a>
<span 
class="cmr-9">The  &#8221;New  Folder&#8221;  icon  in  the  Materials</span>
<span 
class="cmr-9">box.</span>
                                                                         

                                                                         
     &#x00A0; <!--l. 1094--><p class="noindent" ><img 
src="pictures/cloud_screenshots/right_click_new_folder.png" alt="PIC"  
> <a 
 id="x1-45002r2"></a>
<span 
class="cmr-9">&#8221;New Folder&#8221; in the right click drop down</span>
<span 
class="cmr-9">menu.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.3.2: </span><span  
class="content">Create Cloud subfolders in the Materials box.</span></div><!--tex4ht:label?: x1-45003r2 -->
     <!--l. 1099--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1103--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.9.4   </span> <a 
 id="x1-460001.9.4"></a>Sharing Cloud Folders</h5>
     <ul class="itemize1">
     <li class="itemize">Folders can be shared via the &#8220;Cloud Folder Settings&#8221; dialog. It can be accessed
     by  clicking  the  &#8220;Cloud  Settings&#8221;  icon  in  the  Materials  box  or  in  the  toolbar
     (Figure&#x00A0;<a 
href="#x1-46003r2">1.9.4.2<!--tex4ht:ref: fig:cloud_settings_access --></a>). In this dialog select the folder from the drop down menu which
     you would like to share.
     <!--l. 1109--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-46003r2"></a> <!--l. 1112--><p class="noindent" ><img 
src="pictures/cloud_screenshots/cloud_settings_icon.png" alt="PIC"  
> <a 
 id="x1-46001r1"></a>
<span 
class="cmr-9">The &#8221;Cloud Settings&#8221; icon in the Materials</span>
<span 
class="cmr-9">box.</span>
     &#x00A0; <!--l. 1119--><p class="noindent" ><img 
src="pictures/cloud_screenshots/toolbar_cloud.png" alt="PIC"  
> <a 
 id="x1-46002r2"></a>
<span 
class="cmr-9">The &#8221;Cloud Settings&#8221; icon in the toolbar.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.4.2: </span><span  
class="content">The &#8221;Cloud Settings&#8221; icon.</span></div><!--tex4ht:label?: x1-46003r2 -->
     <!--l. 1124--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can also share a folder directly by right clicking on a folder and selecting &#8220;Settings&#8221;
     (Figure&#x00A0;<a 
href="#x1-46004r3">1.9.4.3<!--tex4ht:ref: fig:right_click_settings --></a>). Please note that you cannot share subfolders.
     <!--l. 1128--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-46004r3"></a> <img 
src="pictures/cloud_screenshots/right_click_settings.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.4.3: </span><span  
class="content">&#8221;Settings&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-46004r3 -->
     <!--l. 1133--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can share folders with other users. Select which folder you would like to share from
     the drop down menu (Figure&#x00A0;<a 
href="#x1-46005r4">1.9.4.4<!--tex4ht:ref: fig:cloud_folder_settings_choosefolder --></a>). Enter their email address in the &#8221;Share with&#8221;
     field and click &#8220;Add&#8221;. They will receive an email telling them that you would like to
     share a folder.
     <!--l. 1138--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-46005r4"></a> <img 
src="pictures/cloud_screenshots/cloud_folder_settings_choosefolder.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.4.4: </span><span  
class="content">The &#8221;Cloud Folder Settings&#8221; dialog: choosing a folder.</span></div><!--tex4ht:label?: x1-46005r4 -->
     <!--l. 1143--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To manage permissions, select &#8220;Read/Write&#8221; or &#8220;Read&#8221; from the drop down menu
     (Figure&#x00A0;<a 
href="#x1-46006r5">1.9.4.5<!--tex4ht:ref: fig:cloud_folder_settings --></a>). To remove users from the list, click &#8220;Remove&#8221;.
     <!--l. 1147--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-46006r5"></a> <img 
src="pictures/cloud_screenshots/cloud_folder_settings.png" alt="PIC"  
>
                                                                         

                                                                         
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.4.5: </span><span  
class="content">The &#8221;Cloud Folder Settings&#8221; dialog: managing permissions.</span></div><!--tex4ht:label?: x1-46006r5 -->
     <!--l. 1152--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1156--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.9.5   </span> <a 
 id="x1-470001.9.5"></a>Creating New Folders via the &#8221;Cloud Folder Settings&#8221; dialog</h5>
     <ul class="itemize1">
     <li class="itemize">You  can  also  create  new  folders  from  the  &#8220;Cloud  Folder  Settings&#8221;  dialog
     (Figure&#x00A0;<a 
href="#x1-47001r1">1.9.5.1<!--tex4ht:ref: fig:cloud_folder_settings_new_folder --></a>) by clicking on the &#8220;New Folder&#8221; icon. Enter the new folder&#8217;s name
     and an optional folder description.
     <!--l. 1163--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-47001r1"></a> <img 
src="pictures/cloud_screenshots/cloud_folder_settings_new_folder.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.9.5.1: </span><span  
class="content">The &#8221;New Folder&#8221; icon in the &#8221;Cloud Folder Settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-47001r1 -->
     <!--l. 1168--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1171--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.10   </span> <a 
 id="x1-480001.10"></a>User Comments</h4>
<!--l. 1176--><p class="noindent" >The user comments feature allows you to make comments on all abstraction layers, view
comments from multiple users over the cloud manipulating a single sequence, and to view a
summary table of all comments in each project.
<!--l. 1178--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.10.1   </span> <a 
 id="x1-490001.10.1"></a>User Comments Indicator</h5>
     <ul class="itemize1">
     <li class="itemize">You can make comments on all abstraction layers. Comments are indicated by
     yellow flags (Figure&#x00A0;<a 
href="#x1-49001r1">1.10.1.1<!--tex4ht:ref: fig:yellow_flag --></a>).
     <!--l. 1184--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-49001r1"></a> <img 
src="pictures/comments_screenshots/yellow_flag.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.1.1: </span><span  
class="content">Yellow flags indicate comments.</span></div><!--tex4ht:label?: x1-49001r1 -->
     <!--l. 1189--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1193--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.10.2   </span> <a 
 id="x1-500001.10.2"></a>Navigating to Comments</h5>
     <ul class="itemize1">
     <li class="itemize">All project comments are also tracked as small indications on the scroll bar on the
     right (Figure&#x00A0;<a 
href="#x1-50001r1">1.10.2.1<!--tex4ht:ref: fig:scrollbar_comments --></a>). You can navigate to the comments by clicking on these
     indications.
     <!--l. 1200--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-50001r1"></a> <img 
src="pictures/comments_screenshots/scrollbar_comments.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.2.1: </span><span  
class="content">The scroll bar allows navigation to the comments.</span></div><!--tex4ht:label?: x1-50001r1 -->
     <!--l. 1205--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1210--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.10.3   </span> <a 
 id="x1-510001.10.3"></a>Adding Comments</h5>
     <ul class="itemize1">
     <li class="itemize">To comment a whole part/gene, right click and select &#8220;Add comment&#8221; from the
     drop down menu (Figure&#x00A0;<a 
href="#x1-51001r1">1.10.3.1<!--tex4ht:ref: fig:add_comment_part --></a>).
     <!--l. 1216--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-51001r1"></a> <img 
src="pictures/comments_screenshots/add_comment_part.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.3.1: </span><span  
class="content">&#8221;Add comment&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-51001r1 -->
                                                                         

                                                                         
     <!--l. 1221--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Add your comment in the comment tool tip (Figure&#x00A0;<a 
href="#x1-51002r2">1.10.3.2<!--tex4ht:ref: fig:comment_part --></a>). You can include direct
     links. To navigate to links, click on them while holding the control key. You can delete
     comments by clicking the &#8220;Delete&#8221; icon.
     <!--l. 1226--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-51002r2"></a> <img 
src="pictures/comments_screenshots/comment_part.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.3.2: </span><span  
class="content">The comment tool tip: including a direct link.</span></div><!--tex4ht:label?: x1-51002r2 -->
     <!--l. 1231--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To comment a stretch of nucleotides, right click on them and select &#8220;Add comment&#8221;
     from the drop down menu (Figure&#x00A0;<a 
href="#x1-51003r3">1.10.3.3<!--tex4ht:ref: fig:add_comment_dna --></a>).
     <!--l. 1235--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-51003r3"></a> <img 
src="pictures/comments_screenshots/add_comment_dna.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.3.3: </span><span  
class="content">&#8221;Add comment&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-51003r3 -->
     <!--l. 1240--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Add your comment in the tool tip (Figure&#x00A0;<a 
href="#x1-51004r4">1.10.3.4<!--tex4ht:ref: fig:comment_dna --></a>).
     <!--l. 1244--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-51004r4"></a> <img 
src="pictures/comments_screenshots/comment_dna.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.3.4: </span><span  
class="content">The comment tool tip.</span></div><!--tex4ht:label?: x1-51004r4 -->
     <!--l. 1249--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1255--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.10.4   </span> <a 
 id="x1-520001.10.4"></a>Multiple Users making Comments in a Single Project</h5>
     <ul class="itemize1">
     <li class="itemize">Multiple users can comment the same sequence (Figure&#x00A0;<a 
href="#x1-52001r1">1.10.4.1<!--tex4ht:ref: fig:2_users_comment --></a>).
     <!--l. 1262--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-52001r1"></a> <img 
src="pictures/comments_screenshots/2_users_comment.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.4.1: </span><span  
class="content">Comments from multiple users on a single sequence.</span></div><!--tex4ht:label?: x1-52001r1 -->
     <!--l. 1267--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1272--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.10.5   </span> <a 
 id="x1-530001.10.5"></a>User Comments Summary Table</h5>
     <ul class="itemize1">
     <li class="itemize">To view a summary table of all comments, click the &#8220;Summary table&#8221; icon in the
     toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-53001r1">1.10.5.1<!--tex4ht:ref: fig:summary_table_dropdown --></a>). You can remove the comments
     from the views by unchecking the box. However, you will still be able to add
                                                                         

                                                                         
     comments.
     <!--l. 1278--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-53001r1"></a> <img 
src="pictures/comments_screenshots/summary_table_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.5.1: </span><span  
class="content">The comments &#8221;Summary table&#8221; icon in the toolbar drop down Layers
menu.</span></div><!--tex4ht:label?: x1-53001r1 -->
     <!--l. 1283--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8220;User Comments Summary&#8221; table (Figure&#x00A0;<a 
href="#x1-53002r2">1.10.5.2<!--tex4ht:ref: fig:user_comments_summary --></a>) displays all the comments, as
     well as their location, author and the time and date that they were made. You can
     navigate to each comment in the sequence by clicking on it. Click the &#8220;Show comments
     and changes&#8221; box to include in this list the track changes that were also made in this
     project.
     <!--l. 1287--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-53002r2"></a>                                                       <img 
src="pictures/comments_screenshots/user_comments_summary.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.10.5.2: </span><span  
class="content">The &#8221;User Comments Summary&#8221; table.</span></div><!--tex4ht:label?: x1-53002r2 -->
     <!--l. 1291--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1294--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.11   </span> <a 
 id="x1-540001.11"></a>Track Changes</h4>
<!--l. 1298--><p class="noindent" >The track changes feature allows you to track changes in real time, to see changes made to
projects by multiple users over the cloud and to view a summary table of all changes made to
each project. Changes include additions, deletions, substitution mutations, combinatorial
changes, etc.
<!--l. 1300--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.11.1   </span> <a 
 id="x1-550001.11.1"></a>Track Changes Indicator</h5>
     <ul class="itemize1">
     <li class="itemize">DNA and amino acid changes are tracked and indicated in the form of a blue flag
     on the sequence (Figure&#x00A0;<a 
href="#x1-55001r1">1.11.1.1<!--tex4ht:ref: fig:blue_flag --></a>).
     <!--l. 1306--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-55001r1"></a> <img 
src="pictures/track_changes_screenshots/blue_flag.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.11.1.1: </span><span  
class="content">Blue flags indicate changes.</span></div><!--tex4ht:label?: x1-55001r1 -->
     <!--l. 1311--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1315--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.11.2   </span> <a 
 id="x1-560001.11.2"></a>Navigating to Changes</h5>
     <ul class="itemize1">
     <li class="itemize">All project changes are also tracked as small indications on the scroll bar on the
     right (Figure&#x00A0;<a 
href="#x1-56001r1">1.11.2.1<!--tex4ht:ref: fig:changes_scrollbar --></a>). You can navigate to the tracked changes by clicking on
     these indications.
     <!--l. 1321--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-56001r1"></a> <img 
src="pictures/track_changes_screenshots/changes_scrollbar.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.11.2.1: </span><span  
class="content">The scroll bar allows navigation to the changes.</span></div><!--tex4ht:label?: x1-56001r1 -->
     <!--l. 1326--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1330--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.11.3   </span> <a 
 id="x1-570001.11.3"></a>Multiple Users making Changes in a Single Project</h5>
     <ul class="itemize1">
     <li class="itemize">Multiple users can make changes on the same sequence (Figure&#x00A0;<a 
href="#x1-57001r1">1.11.3.1<!--tex4ht:ref: fig:2_users_changes --></a>).
     <!--l. 1336--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-57001r1"></a> <img 
src="pictures/track_changes_screenshots/2_users_changes.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.11.3.1: </span><span  
class="content">Changes from multiple users on a single sequence.</span></div><!--tex4ht:label?: x1-57001r1 -->
                                                                         

                                                                         
     <!--l. 1341--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1345--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.11.4   </span> <a 
 id="x1-580001.11.4"></a>Track Changes Summary Table</h5>
     <ul class="itemize1">
     <li class="itemize">To view a summary table of all changes, click the track changes &#8220;Summary table&#8221;
     icon in the toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-58001r1">1.11.4.1<!--tex4ht:ref: fig:track_changes_dropdown_menu --></a>). You can remove the
     changes from the views by unchecking the box.
     <!--l. 1351--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-58001r1"></a> <img 
src="pictures/track_changes_screenshots/track_changes_dropdown_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.11.4.1: </span><span  
class="content">The track changes &#8221;Summary table&#8221; icon in the toolbar drop down
Layers menu.</span></div><!--tex4ht:label?: x1-58001r1 -->
     <!--l. 1356--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8220;Track Changes Summary&#8221; table (Figure&#x00A0;<a 
href="#x1-58002r2">1.11.4.2<!--tex4ht:ref: fig:track_changes_summary --></a>) displays all the changes in the
     project, as well as their location, author, the time and date that they were
     made and prospective comments. You can navigate to each change in the
     DNA sequence by clicking on it. Click the &#8220;Show comments and changes&#8221;
     box to include in this list any user comments which were also made in this
     project.
     <!--l. 1360--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-58002r2"></a>                                                       <img 
src="pictures/track_changes_screenshots/track_changes_summary.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.11.4.2: </span><span  
class="content">The &#8220;Track Changes Summary&#8221; table.</span></div><!--tex4ht:label?: x1-58002r2 -->
     <!--l. 1364--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1368--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.12   </span> <a 
 id="x1-590001.12"></a>Ordering and Pricing</h4>
<!--l. 1375--><p class="noindent" >Genome Compiler allows you to request quotes and order DNA directly from the software!
Pricing is available for projects as well as for primers.
<!--l. 1378--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.12.1   </span> <a 
 id="x1-600001.12.1"></a>Pricing for Projects</h5>
     <ul class="itemize1">
     <li class="itemize">To request a price quote for synthesising a whole project, go to the &#8221;Pricing&#8221; icon
     in the toolbar and click &#8221;Project...&#8221; (Figure&#x00A0;<a 
href="#x1-60001r1">1.12.1.1<!--tex4ht:ref: fig:pricing_select_project --></a>).
     <!--l. 1384--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-60001r1"></a> <img 
src="pictures/pricing_ordering_screenshots/pricing_select_project.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.12.1.1: </span><span  
class="content">&#8221;Project&#8221; in the &#8221;Pricing&#8221; drop down menu in the toolbar.</span></div><!--tex4ht:label?: x1-60001r1 -->
     <!--l. 1389--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Pricing settings&#8221; dialog will appear (Figure&#x00A0;<a 
href="#x1-60002r2">1.12.1.2<!--tex4ht:ref: fig:pricing_project --></a>). Select a supplier from the
     list and click &#8221;Get quote&#8221;. A representative from Genome Compiler will contact you to
     continue with the ordering process.
     <!--l. 1393--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-60002r2"></a>                                                       <img 
src="pictures/pricing_ordering_screenshots/pricing_project.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.12.1.2: </span><span  
class="content">The &#8221;Pricing settings&#8221; dialog for projects.</span></div><!--tex4ht:label?: x1-60002r2 -->
     <!--l. 1397--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1401--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.12.2   </span> <a 
 id="x1-610001.12.2"></a>Pricing for Primers</h5>
     <ul class="itemize1">
     <li class="itemize">To  get  pricing  for  primers,  go  to  the  &#8221;Pricing&#8221;  icon  in  the  toolbar  and  click
     &#8221;Primers...&#8221; (Figure&#x00A0;<a 
href="#x1-61001r1">1.12.2.1<!--tex4ht:ref: fig:pricing_select_primers --></a>).
     <!--l. 1407--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-61001r1"></a> <img 
src="pictures/pricing_ordering_screenshots/pricing_select_primers.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.12.2.1: </span><span  
class="content">&#8221;Primers&#8221; in the &#8221;Pricing&#8221; drop down menu in the toolbar.</span></div><!--tex4ht:label?: x1-61001r1 -->
     <!--l. 1412--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Primers settings&#8221; dialog will appear (Figure&#x00A0;<a 
href="#x1-61002r2">1.12.2.2<!--tex4ht:ref: fig:settings_primers --></a>). Select the primers you
     would like to order from the list and click &#8221;Pricing&#8221;.
     <!--l. 1416--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-61002r2"></a>                                                       <img 
src="pictures/pricing_ordering_screenshots/settings_primers.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.12.2.2: </span><span  
class="content">The &#8221;Primers settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-61002r2 -->
                                                                         

                                                                         
     <!--l. 1420--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Pricing settings&#8221; dialog will appear (Figure&#x00A0;<a 
href="#x1-61003r3">1.12.2.3<!--tex4ht:ref: fig:pricing_primers --></a>). Select a supplier from the
     list and click &#8221;Next&#8221;. This will redirect you to the website of the selected
     supplier.
     <!--l. 1424--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-61003r3"></a>                                                       <img 
src="pictures/pricing_ordering_screenshots/pricing_primers.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.12.2.3: </span><span  
class="content">The &#8221;Pricing settings&#8221; dialog for primers.</span></div><!--tex4ht:label?: x1-61003r3 -->
     <!--l. 1428--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1431--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.13   </span> <a 
 id="x1-620001.13"></a>Compilation</h4>
<!--l. 1437--><p class="noindent" >The compilation engine allows you to debug sequences. It encompasses specific synthesis
manufacturability rules as well as standard synthetic biology constraints.
<!--l. 1439--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.13.1   </span> <a 
 id="x1-630001.13.1"></a>Selecting Rules for Activation</h5>
     <ul class="itemize1">
     <li class="itemize">Before you start using the compilation engine, you should select the rules you
     would like to use. Rules can be selected from the &#8220;Compilation Settings&#8221; dialog
     (Figure&#x00A0;<a 
href="#x1-63003r3">1.13.1.3<!--tex4ht:ref: fig:compilation_settings_dialog --></a>). It can be accessed primarily by clicking on the corresponding
     &#8220;Settings&#8221; icon from the toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-63001r1">1.13.1.1<!--tex4ht:ref: fig:compilation_settings_toolbar --></a>), or
     by clicking the &#8220;Settings&#8221; icon on the far right in the &#8220;Compilation&#8221; tab after a
     compilation has been carried out (Figure&#x00A0;<a 
href="#x1-63002r2">1.13.1.2<!--tex4ht:ref: fig:compilation_tab_annotated --></a>).
     <!--l. 1445--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-63001r1"></a> <img 
src="pictures/compilation_screenshots/compilation_settings_toolbar.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.1.1: </span><span  
class="content">The &#8221;Compilation Settings&#8221; dialog icon in the toolbar drop down
Layers menu.</span></div><!--tex4ht:label?: x1-63001r1 -->
     <!--l. 1450--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1452--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-63002r2"></a>                                                       <img 
src="pictures/compilation_screenshots/compilation_tab_annotated.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.1.2: </span><span  
class="content">&#8221;Settings&#8221; icons in the &#8221;Compilation&#8221; summary table.</span></div><!--tex4ht:label?: x1-63002r2 -->
     <!--l. 1456--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Rules are divided into two groups: &#8220;design rules&#8221; and &#8220;synthesis rules&#8221;. These are
     predefined design rules, as well as manufacturability rules specific to the main DNA
     synthesis providers. You can request a new rule by emailing one of our customer service
     representatives.
     </li>
     <li class="itemize">You can select the rules you would like to apply by checking and unchecking their box in
     the dialog (Figure&#x00A0;<a 
href="#x1-63003r3">1.13.1.3<!--tex4ht:ref: fig:compilation_settings_dialog --></a>).
     </li>
     <li class="itemize">To ignore warnings, simply click the &#8220;Ignore warnings&#8221; button at the top of the dialog
     box (Figure&#x00A0;<a 
href="#x1-63003r3">1.13.1.3<!--tex4ht:ref: fig:compilation_settings_dialog --></a>).
     <!--l. 1464--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-63003r3"></a>                                                       <img 
src="pictures/compilation_screenshots/compilation_settings_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.1.3: </span><span  
class="content">The &#8220;Compilation Settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-63003r3 -->
     <!--l. 1468--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 1472--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.13.2   </span> <a 
 id="x1-640001.13.2"></a>Compiling Projects</h5>
     <ul class="itemize1">
     <li class="itemize">To compile your project, click the compilation icon in the toolbar (Figure&#x00A0;<a 
href="#x1-64001r1">1.13.2.1<!--tex4ht:ref: fig:toolbar_compilation --></a>),
     or right click on your project canvas and click &#8220;Compile project&#8221; (Figure&#x00A0;<a 
href="#x1-64002r2">1.13.2.2<!--tex4ht:ref: fig:right_click_compile --></a>).
     <!--l. 1478--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-64001r1"></a> <img 
src="pictures/compilation_screenshots/toolbar_compilation.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.2.1: </span><span  
class="content">The compilation icon in the toolbar.</span></div><!--tex4ht:label?: x1-64001r1 -->
     <!--l. 1483--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1485--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-64002r2"></a> <img 
src="pictures/compilation_screenshots/right_click_compile.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.2.2: </span><span  
class="content">&#8221;Compile project&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-64002r2 -->
     <!--l. 1490--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1494--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.13.3   </span> <a 
 id="x1-650001.13.3"></a>Error and Warning Indicators</h5>
     <ul class="itemize1">
     <li class="itemize">Errors  and  warnings  correspond  to  violation  of  design  rules.  Errors  (red  X
     flags) and warnings (yellow exclamation mark flags) will appear in the sequence
     (Figure&#x00A0;<a 
href="#x1-65003r2">1.13.3.2<!--tex4ht:ref: fig:error_warning --></a>).
     <!--l. 1500--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-65003r2"></a> <!--l. 1503--><p class="noindent" ><img 
src="pictures/compilation_screenshots/error.png" alt="PIC"  
> <a 
 id="x1-65001r1"></a>
<span 
class="cmr-9">Error.</span>
     &#x00A0; <!--l. 1510--><p class="noindent" ><img 
src="pictures/compilation_screenshots/warning.png" alt="PIC"  
> <a 
 id="x1-65002r2"></a>
<span 
class="cmr-9">Warning.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.3.2: </span><span  
class="content">Errors and warnings.</span></div><!--tex4ht:label?: x1-65003r2 -->
     <!--l. 1515--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1519--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.13.4   </span> <a 
 id="x1-660001.13.4"></a>Compilation Summary Table</h5>
     <ul class="itemize1">
     <li class="itemize">A  summary  of  the  errors  and  warnings  in  the  project  is  displayed  in  the
     &#8220;Compilation&#8221; summary table (Figure&#x00A0;<a 
href="#x1-66002r2">1.13.4.2<!--tex4ht:ref: fig:compilation_tab --></a>) at the bottom of the screen. This
     table can be accessed by clicking the corresponding icon from the toolbar drop
     down Layers menu (Figure&#x00A0;<a 
href="#x1-66001r1">1.13.4.1<!--tex4ht:ref: fig:access_track_summary_table --></a>). You can remove the viewing of errors and
     warnings from the different abstraction layers views by unchecking the compilation
                                                                         

                                                                         
     box from the toolbar drop down Layers menu.
     <!--l. 1525--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-66001r1"></a> <img 
src="pictures/compilation_screenshots/access_track_summary_table.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.4.1: </span><span  
class="content">The &#8221;Compilation&#8221; summary table icon in the toolbar drop down
Layers menu.</span></div><!--tex4ht:label?: x1-66001r1 -->
     <!--l. 1530--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1532--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-66002r2"></a>                                                       <img 
src="pictures/compilation_screenshots/compilation_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.4.2: </span><span  
class="content">The &#8221;Compilation&#8221; summary table.</span></div><!--tex4ht:label?: x1-66002r2 -->
     <!--l. 1536--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1540--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.13.5   </span> <a 
 id="x1-670001.13.5"></a>Navigating to Errors and Warnings</h5>
     <ul class="itemize1">
     <li class="itemize">You can navigate to each error or warning in the DNA sequence by clicking the
     desired rule violation in the &#8220;Compilation&#8221; summary table (Figure&#x00A0;<a 
href="#x1-66002r2">1.13.4.2<!--tex4ht:ref: fig:compilation_tab --></a>) or by
     using the project scroll bar on the right (Figure&#x00A0;<a 
href="#x1-67001r1">1.13.5.1<!--tex4ht:ref: fig:scrollbar_compilation --></a>).
     <!--l. 1546--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-67001r1"></a> <img 
src="pictures/compilation_screenshots/scrollbar_compilation.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.13.5.1: </span><span  
class="content">The scroll bar allows navigation to the errors and warnings.</span></div><!--tex4ht:label?: x1-67001r1 -->
     <!--l. 1551--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1554--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.14   </span> <a 
 id="x1-680001.14"></a>Back Translation</h4>
<!--l. 1559--><p class="noindent" >This feature allows you to back translate according to standard translation tables or codon
usage.
     <ul class="itemize1">
     <li class="itemize">Translation can occur in four different situations:
         <ul class="itemize2">
         <li class="itemize">Back translation of a whole project
         </li>
         <li class="itemize">Back translation of a part/gene
         </li>
         <li class="itemize">Drag &amp; drop a part/gene/AA/DNA
         </li>
         <li class="itemize">Copy &amp; paste part/gene/AA/DNA</li></ul>
     </li></ul>
<!--l. 1574--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.14.1   </span> <a 
 id="x1-690001.14.1"></a>How to Start Translating</h5>
     <ul class="itemize1">
     <li class="itemize">To back translate a whole project, go to the &#8220;Project&#8221; tab (Figure&#x00A0;<a 
href="#x1-69001r1">1.14.1.1<!--tex4ht:ref: fig:project_tab_host_organism --></a>) at the
     bottom of the opened project. You can select the appropriate host organism and
     translation table from the &#8216;Translation Settings&#8217; dialog which will appear upon
     selecting the host organism.
     <!--l. 1580--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-69001r1"></a>                                                       <img 
src="pictures/translation_screenshots/project_tab_host_organism.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.1.1: </span><span  
class="content">The &#8221;Project&#8221; tab.</span></div><!--tex4ht:label?: x1-69001r1 -->
     <!--l. 1584--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To back translate a part or a gene, right click on it and select &#8220;Back Translate&#x2026;&#8221;
     (Figure&#x00A0;<a 
href="#x1-69002r2">1.14.1.2<!--tex4ht:ref: fig:right_click_back_translate --></a>). This will take you to the &#8220;Translation Settings&#8221; dialog
     (Figure&#x00A0;<a 
href="#x1-70003r2">1.14.2.2<!--tex4ht:ref: fig:translation_settings_dialog --></a>).
     <!--l. 1588--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-69002r2"></a> <img 
src="pictures/translation_screenshots/right_click_back_translate.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.1.2: </span><span  
class="content">&#8221;Back Translate...&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-69002r2 -->
     <!--l. 1593--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Dragging and dropping sequence, or copying and pasting sequence from outside the
     project will also prompt the &#8221;Translation Settings&#8221; dialog (Figure&#x00A0;<a 
href="#x1-70003r2">1.14.2.2<!--tex4ht:ref: fig:translation_settings_dialog --></a>).
                                                                         

                                                                         
     </li></ul>
<!--l. 1599--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.14.2   </span> <a 
 id="x1-700001.14.2"></a>&#8220;Translation Settings&#8221; Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">You   can   translate   according   to   codon   usage   or   to   a   translation   table
     (Figure&#x00A0;<a 
href="#x1-70003r2">1.14.2.2<!--tex4ht:ref: fig:translation_settings_dialog --></a>). For codon usage, select the appropriate organism from the drop
     down menu. Similarly, you can select the appropriate translation table from the
     corresponding drop down menu.
     <!--l. 1605--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-70003r2"></a><!--l. 1618--><p class="noindent" ><img 
src="pictures/translation_screenshots/translation_settings_dialog_codon_usage.png" alt="PIC"  
> <a 
 id="x1-70001r1"></a>
<span 
class="cmr-9">Codon usage.</span>                                                        &#x00A0; <!--l. 1618--><p class="noindent" ><img 
src="pictures/translation_screenshots/translation_settings_dialog_transation_table.png" alt="PIC"  
> <a 
 id="x1-70002r2"></a>
<span 
class="cmr-9">Translation table.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.2.2: </span><span  
class="content">The &#8220;Translation Settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-70003r2 -->
     <!--l. 1620--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">In the case of translation of a strand of DNA, you can also choose to keep the DNA as
     original (Figure&#x00A0;<a 
href="#x1-70004r3">1.14.2.3<!--tex4ht:ref: fig:translation_settings_dna_as_original --></a>).
     <!--l. 1624--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-70004r3"></a> <img 
src="pictures/translation_screenshots/translation_settings_dna_as_original.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.2.3: </span><span  
class="content">The &#8220;Translation Settings&#8221; dialog: keep the DNA as original.</span></div><!--tex4ht:label?: x1-70004r3 -->
     <!--l. 1629--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Once you click the &#8220;Don&#8217;t ask again&#8221; box, translation will occur automatically according
     to your settings when you drag and drop or copy and paste a sequence.
     </li></ul>
<!--l. 1636--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.14.3   </span> <a 
 id="x1-710001.14.3"></a>Proportion of each triplet</h5>
     <ul class="itemize1">
     <li class="itemize">After a back translation, the percentages of each triplet for the organism are shown
     in the amino acid drop down menu (Figure&#x00A0;<a 
href="#x1-71001r1">1.14.3.1<!--tex4ht:ref: fig:percentages_dropdown --></a>) as well as in the amino acid
     combinatorial wheel (Figure&#x00A0;<a 
href="#x1-71002r2">1.14.3.2<!--tex4ht:ref: fig:percentages_wheel --></a>) and the amino acid mutation wheel.
     <!--l. 1642--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-71001r1"></a> <img 
src="pictures/translation_screenshots/percentages_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.3.1: </span><span  
class="content">Percentages in the amino acid drop down menu.</span></div><!--tex4ht:label?: x1-71001r1 -->
     <!--l. 1647--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1649--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-71002r2"></a> <img 
src="pictures/translation_screenshots/percentages_wheel.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.14.3.2: </span><span  
class="content">Percentages shown in the amino acid combinatorial wheel.</span></div><!--tex4ht:label?: x1-71002r2 -->
     <!--l. 1654--><p class="noindent" ></div><hr class="endfigure">
                                                                         

                                                                         
     </li></ul>
<!--l. 1657--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.15   </span> <a 
 id="x1-720001.15"></a>Combinatorial Design</h4>
<!--l. 1661--><p class="noindent" >The combinatorial design feature allows you to easily generate and filter multiple
combinations within your construct.
<!--l. 1663--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.15.1   </span> <a 
 id="x1-730001.15.1"></a>Circuit Part Combinatorial</h5>
     <ul class="itemize1">
     <li class="itemize">Drag and drop parts on the part that you would like to change. You can add part
     alternatives to as many different parts in the project as desired. The &#8221;Translation
     Settings&#8221; dialog will pop up (Figure&#x00A0;<a 
href="#x1-73001r1">1.15.1.1<!--tex4ht:ref: fig:translation_settings_dialog_codon_usage --></a>). You can choose to translate your
     sequence according to codon usage or to a translation table or to keep the DNA as
     original (more details about the translation feature can be found in section&#x00A0;<a 
href="#x1-680001.14">1.14<!--tex4ht:ref: subsec:translation --></a>).
     In the example below, the sequence is translated according to codon usage in E.
     coli.
     <!--l. 1669--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-73001r1"></a> <img 
src="pictures/combinatorial_screenshots/translation_settings_dialog_codon_usage.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.1.1: </span><span  
class="content">The &#8220;Translation Settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-73001r1 -->
     <!--l. 1674--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">A dropdown arrow will appear on the part. Click on it to see the dropdown menu
     showing all the different sequences (Figure&#x00A0;<a 
href="#x1-73002r2">1.15.1.2<!--tex4ht:ref: fig:combinatorial_parts_dropdown --></a>). To view a summary of all
     combinations, click &#8221;Combinatorial Summary...&#8221;.
     <!--l. 1678--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-73002r2"></a> <img 
src="pictures/combinatorial_screenshots/combinatorial_parts_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.1.2: </span><span  
class="content">The dropdown menu showing the different parts.</span></div><!--tex4ht:label?: x1-73002r2 -->
     <!--l. 1683--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Combinatorial Summary - Parts&#8221; will appear (Figure&#x00A0;<a 
href="#x1-73003r3">1.15.1.3<!--tex4ht:ref: fig:combinatorial_summary_parts --></a>). It displays the
     different combinations, which can be selected. You can choose which part to view in
     the main view by selecting it and clicking &#8221;Set as Current&#8221;. Get a pricing of
     your projects by clicking &#8221;Pricing&#8221;. The statistics of your selection are shown
     at the bottom of the dialog. On exhaustive mode, the library consists of all
     the possible combinations, whereas in one step mutation mode, the library
     consists of all combinations whereby only one alternative is replaced each
     time.
     <!--l. 1687--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-73003r3"></a>                                                       <img 
src="pictures/combinatorial_screenshots/combinatorial_summary_parts.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.1.3: </span><span  
class="content">The &#8221;Combinatorial Summary - Parts&#8221;.</span></div><!--tex4ht:label?: x1-73003r3 -->
     <!--l. 1691--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 1696--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.15.2   </span> <a 
 id="x1-740001.15.2"></a>Protein Engineering Combinatorial</h5>
     <ul class="itemize1">
     <li class="itemize">To test different amino acids on one location, either click the drop down arrow
     below  the  amino  acid  and  select  &#8221;Combinatorial...&#8221;  (Figure&#x00A0;<a 
href="#x1-74001r1">1.15.2.1<!--tex4ht:ref: fig:combinatorial_dropdown --></a>)  or  right
     click on the amino acid and select &#8221;Combinatorial...&#8221; from the drop down menu
     (Figure&#x00A0;<a 
href="#x1-74002r2">1.15.2.2<!--tex4ht:ref: fig:combinatorial_right_click --></a>). You can add amino acid alternatives to as many different amino
     acids in the part as desired.
     <!--l. 1702--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-74001r1"></a> <img 
src="pictures/combinatorial_screenshots/combinatorial_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.2.1: </span><span  
class="content">&#8221;Combinatorial...&#8221; in the drop down arrow menu.</span></div><!--tex4ht:label?: x1-74001r1 -->
     <!--l. 1707--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1709--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-74002r2"></a> <img 
src="pictures/combinatorial_screenshots/combinatorial_right_click.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.2.2: </span><span  
class="content">&#8221;Combinatorial...&#8221; in the right click dropdown menu.</span></div><!--tex4ht:label?: x1-74002r2 -->
     <!--l. 1714--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Select amino acids from the combinatorial wheel (Figure&#x00A0;<a 
href="#x1-74003r3">1.15.2.3<!--tex4ht:ref: fig:combinatorial_aa --></a>) and click the green
     tick. You can filter selection of amino acids with a specific property by checking the
     property checkbox on the left.
     <!--l. 1718--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-74003r3"></a> <img 
src="pictures/combinatorial_screenshots/combinatorial_aa.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.2.3: </span><span  
class="content">The amino acids combinatorial wheel.</span></div><!--tex4ht:label?: x1-74003r3 -->
     <!--l. 1723--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">A drop down arrow will appear on the amino acid. Click on it to see the drop down
     menu showing all the different amino acids (Figure&#x00A0;<a 
href="#x1-74004r4">1.15.2.4<!--tex4ht:ref: fig:combinatorial_aa_dropdown --></a>). To view a summary of all
     combinations, click &#8221;Combinatorial Summary...&#8221;.
     <!--l. 1727--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-74004r4"></a> <img 
src="pictures/combinatorial_screenshots/combinatorial_aa_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.2.4: </span><span  
class="content">The drop down menu showing the different amino acids.</span></div><!--tex4ht:label?: x1-74004r4 -->
     <!--l. 1732--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Combinatorial Summary - Proteins&#8221; will appear (Figure&#x00A0;<a 
href="#x1-74005r5">1.15.2.5<!--tex4ht:ref: fig:combinatorial_summary_proteins --></a>). It has the same
     features as for parts.
     <!--l. 1736--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-74005r5"></a>                                                       <img 
src="pictures/combinatorial_screenshots/combinatorial_summary_proteins.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15.2.5: </span><span  
class="content">The &#8221;Combinatorial Summary - Proteins&#8221;.</span></div><!--tex4ht:label?: x1-74005r5 -->
     <!--l. 1740--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1745--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.16   </span> <a 
 id="x1-750001.16"></a>Managing Restriction Sites</h4>
<!--l. 1750--><p class="noindent" >The &#8220;Restriction Sites&#8221; dialog allows you to manage restriction enzymes. You can filter them
for different criteria and save presets for further use.
<!--l. 1752--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.16.1   </span> <a 
 id="x1-760001.16.1"></a>Opening the Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">To open the &#8220;Restriction Sites&#8221; dialog, click on the Layers menu in the toolbar and
     select the &#8220;Restriction sites&#8221; settings from the drop down menu (Figure&#x00A0;<a 
href="#x1-76001r1">1.16.1.1<!--tex4ht:ref: fig:dropdown_menu_dialog --></a>).
     Selected cut sites will be visible throughout the different abstraction layers views
     (Figure&#x00A0;<a 
href="#x1-76005r3">1.16.1.3<!--tex4ht:ref: fig:cut_sites_copy --></a>). To remove them from the views, uncheck the corresponding
     checkbox in the Layers menu.
     <!--l. 1758--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-76001r1"></a> <img 
src="pictures/restriction_sites_dialog_screenshots/dropdown_menu_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.1.1: </span><span  
class="content">The &#8221;Restriction Sites&#8221; settings icon in the toolbar drop down Layers
menu.</span></div><!--tex4ht:label?: x1-76001r1 -->
     <!--l. 1763--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1765--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-76005r3"></a>  <!--l. 1785--><p class="noindent" ><img 
src="pictures/digest_screenshots/plasmid_view.png" alt="PIC"  
> <a 
 id="x1-76002r1"></a>
<span 
class="cmr-9">Circular view.</span>                     &#x00A0; <!--l. 1785--><p class="noindent" ><img 
src="pictures/digest_screenshots/parts_view.png" alt="PIC"  
> <a 
 id="x1-76003r2"></a>
<span 
class="cmr-9">Linear view.</span>                       &#x00A0; <!--l. 1785--><p class="noindent" ><img 
src="pictures/digest_screenshots/dna_view.png" alt="PIC"  
> <a 
 id="x1-76004r3"></a>
<span 
class="cmr-9">DNA view.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.1.3: </span><span  
class="content">Cut sites in the different views.</span></div><!--tex4ht:label?: x1-76005r3 -->
     <!--l. 1787--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1792--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.16.2   </span> <a 
 id="x1-770001.16.2"></a>&#8221;Restriction Site&#8221; Settings Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8220;Restriction Site&#8221; settings dialog (Figure&#x00A0;<a 
href="#x1-77001r1">1.16.2.1<!--tex4ht:ref: fig:restriction_sites_dialog --></a>) will appear, displaying the
     following information about the restriction enzymes:
         <ul class="itemize2">
         <li class="itemize">Number of cuts in the project
         </li>
         <li class="itemize">Length of the recognition sequence
         </li>
         <li class="itemize">Hang size
                                                                         

                                                                         
         </li>
         <li class="itemize">Ambiguity, i.e. whether different sequences can be recognized by the enzyme</li></ul>
     </li>
     <li class="itemize">In order to display the restriction enzymes cut sites in the project, select the check box
     for each enzyme in the main table and they will be automatically added to the
     &#8221;Selected&#8221; list on the right. You can remove enzymes from the list by unchecking their
     checkbox or clicking on their &#8220;Delete&#8221; icon.
     <!--l. 1807--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-77001r1"></a> <img 
src="pictures/restriction_sites_dialog_screenshots/restriction_sites_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.2.1: </span><span  
class="content">The &#8221;Restriction Sites&#8221; settings dialog.</span></div><!--tex4ht:label?: x1-77001r1 -->
     <!--l. 1812--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The drop down menu from each column heading can be used to filter your enzyme
     selection. In the example below (Figure&#x00A0;<a 
href="#x1-77002r2">1.16.2.2<!--tex4ht:ref: fig:double_cutters_only --></a>), we selected for double cutters only, so
     that only double cutters will appear on the list.
     <!--l. 1816--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-77002r2"></a> <img 
src="pictures/restriction_sites_dialog_screenshots/double_cutters_only.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.2.2: </span><span  
class="content">Example: show double cutters only.</span></div><!--tex4ht:label?: x1-77002r2 -->
     <!--l. 1821--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8220;Preset&#8221; drop down menu allows you to switch between presets. To create a new
     preset, select restriction enzymes (they will appear on the right) and click save. A dialog
     will pop up (Figure&#x00A0;<a 
href="#x1-77003r3">1.16.2.3<!--tex4ht:ref: fig:new_preset --></a>) to prompt you for a new &#8221;Preset name&#8221;. To delete a
     preset, click the corresponding delete icon .
     <!--l. 1825--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-77003r3"></a> <img 
src="pictures/restriction_sites_dialog_screenshots/new_preset.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.2.3: </span><span  
class="content">The &#8221;Save Preset&#8221; dialog (forefront).</span></div><!--tex4ht:label?: x1-77003r3 -->
     <!--l. 1830--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The small drop down arrow at the far right of the main table for each enzyme displays
     the restriction site locations (Figure&#x00A0;<a 
href="#x1-77004r4">1.16.2.4<!--tex4ht:ref: fig:restriction_site_locations_dropdown --></a>). Clicking on the locations will take you to
     the corresponding area in the sequence view and highlight the location in the plasmid
     view.
     <!--l. 1834--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-77004r4"></a> <img 
src="pictures/restriction_sites_dialog_screenshots/restriction_site_locations_dropdown.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.16.2.4: </span><span  
class="content">The &#8221;Restriction Sites&#8221; dialog: locations of the restriction sites.</span></div><!--tex4ht:label?: x1-77004r4 -->
     <!--l. 1839--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">If you check the &#8220;Show 1 cutters only&#8221; box at the bottom left, only 1 cutters will be
     visible throughout the different views.
     </li></ul>
<!--l. 1844--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.17   </span> <a 
 id="x1-780001.17"></a>Virtual Digest</h4>
<!--l. 1848--><p class="noindent" >This feature allows you to perform a hypothetical digest on a sequence selection or on the
whole of your sequence.
<!--l. 1850--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.17.1   </span> <a 
 id="x1-790001.17.1"></a>Running a Digest</h5>
     <ul class="itemize1">
     <li class="itemize">Right click on your project canvas or the project tab and select &#8220;Run Digest&#8221;
     (Figure&#x00A0;<a 
href="#x1-79001r1">1.17.1.1<!--tex4ht:ref: fig:right_click_digest --></a>).
     <!--l. 1856--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-79001r1"></a> <img 
src="pictures/digest_screenshots/right_click_digest.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.17.1.1: </span><span  
class="content">&#8221;Run Digest&#8221; in the right click drop down menu.</span></div><!--tex4ht:label?: x1-79001r1 -->
     <!--l. 1861--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1865--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.17.2   </span> <a 
 id="x1-800001.17.2"></a>&#8220;Restriction Sites&#8221; Window</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8220;Restriction Sites&#8221; window will appear (Figure&#x00A0;<a 
href="#x1-80001r1">1.17.2.1<!--tex4ht:ref: fig:restriction_sites_window --></a>). Select the desired
     restriction enzymes for your digest. This window works in a similar fashion to the
     &#8220;Restriction Sites&#8221; dialog (see section&#x00A0;<a 
href="#x1-750001.16">1.16<!--tex4ht:ref: subsec:restriction_sites_dialog --></a>).
     <!--l. 1871--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-80001r1"></a> <img 
src="pictures/digest_screenshots/restriction_sites_window.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.17.2.1: </span><span  
class="content">The &#8220;Restriction Sites&#8221; window.</span></div><!--tex4ht:label?: x1-80001r1 -->
     <!--l. 1876--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1880--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.17.3   </span> <a 
 id="x1-810001.17.3"></a>&#8221;Digest&#8221; Summary Table</h5>
     <ul class="itemize1">
     <li class="itemize">Click  &#8220;Run  Digest&#8221;  and  all  the  information  regarding  digested  fragment  sizes
     will be displayed in the &#8220;Digest&#8221; summary table at the bottom of your screen
     (Figure&#x00A0;<a 
href="#x1-81001r1">1.17.3.1<!--tex4ht:ref: fig:digest_tab --></a>). The selected cut sites now appear on the different abstraction
     layer views (Figure&#x00A0;<a 
href="#x1-81005r3">1.17.3.3<!--tex4ht:ref: fig:cut_sites --></a>). You can navigate to the DNA view of the different
     sites by clicking in the corresponding line of the &#8220;Digest&#8221; summary table.
                                                                         

                                                                         
     <!--l. 1886--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-81001r1"></a> <img 
src="pictures/digest_screenshots/digest_tab.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.17.3.1: </span><span  
class="content">The &#8220;Digest&#8221; summary table.</span></div><!--tex4ht:label?: x1-81001r1 -->
     <!--l. 1891--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 1894--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-81005r3"></a><!--l. 1914--><p class="noindent" ><img 
src="pictures/digest_screenshots/plasmid_view.png" alt="PIC"  
> <a 
 id="x1-81002r1"></a>
<span 
class="cmr-9">Circular view.</span>                               &#x00A0; <!--l. 1914--><p class="noindent" ><img 
src="pictures/digest_screenshots/parts_view.png" alt="PIC"  
> <a 
 id="x1-81003r2"></a>
<span 
class="cmr-9">Linear view.</span>                                 &#x00A0; <!--l. 1914--><p class="noindent" ><img 
src="pictures/digest_screenshots/dna_view.png" alt="PIC"  
> <a 
 id="x1-81004r3"></a>
<span 
class="cmr-9">DNA view.</span>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.17.3.3: </span><span  
class="content">Cut sites in the different views.</span></div><!--tex4ht:label?: x1-81005r3 -->
     <!--l. 1916--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 1919--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.18   </span> <a 
 id="x1-820001.18"></a>Primer Design</h4>
<!--l. 1924--><p class="noindent" >The Primer Design tool allows you to design primers in an easy and efficient way.
<!--l. 1926--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.18.1   </span> <a 
 id="x1-830001.18.1"></a>Opening the Primer Design Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">You can access the Primer Design dialog in two ways. Select a sequence, right click
     and select &#8221;Create primer from selection&#8221; from the drop down menu, or click the
     Primer icon in the Layers menu and select &#8221;Create primer&#8221;. The former will lead
     to auto-filled primer sequence in the dialog.
     </li></ul>
<!--l. 1934--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.18.2   </span> <a 
 id="x1-840001.18.2"></a>Primer Design Dialog</h5>
     <ul class="itemize1">
     <li class="itemize">The Primer Design dialog (Figure&#x00A0;<a 
href="#x1-84001r1">1.18.2.1<!--tex4ht:ref: fig:create_primer_dialog --></a>) is divided into three sections. Each section is
     detailed below.
     <!--l. 1953--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-84001r1"></a> <img 
src="pictures/primer_design_screenshots/create_primer_dialog.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.2.1: </span><span  
class="content">The Primer Design dialog.</span></div><!--tex4ht:label?: x1-84001r1 -->
     <!--l. 1958--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can edit the sequence ID, and add a description. Select the strand for your primer.
     If the sequence is not yet filled, click &#8221;Use selection&#8221;. This will take you to
     your project where you can select a sequence. You can edit the 3&#8217; location
     (Figure&#x00A0;<a 
href="#x1-84002r2">1.18.2.2<!--tex4ht:ref: fig:manual_primer_design_dialog_top --></a>).
     <!--l. 1962--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-84002r2"></a> <img 
src="pictures/primer_design_screenshots/manual_primer_design_dialog_top.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.2.2: </span><span  
class="content">The &#8220;Manual Primer Design&#8221; dialog: sequence ID, description, strand
and 3&#8217; location.</span></div><!--tex4ht:label?: x1-84002r2 -->
     <!--l. 1967--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">For PCR based cloning, you can add restriction &#8221;Cut Sites&#8221; to your primers from the
     drop down menu, followed by a &#8221;Leader Sequence&#8221; (Figure&#x00A0;<a 
href="#x1-84003r3">1.18.2.3<!--tex4ht:ref: fig:pcr_based_cloning --></a>).
     <!--l. 1971--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-84003r3"></a> <img 
src="pictures/primer_design_screenshots/pcr_based_cloning.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.2.3: </span><span  
class="content">The &#8220;Manual Primer Design&#8221; dialog: PCR based cloning.</span></div><!--tex4ht:label?: x1-84003r3 -->
     <!--l. 1976--><p class="noindent" ></div><hr class="endfigure">
                                                                         

                                                                         
     </li>
     <li class="itemize">The verification field displays the melting temperature (calculated according
     to <a 
href="http://www.pnas.org/content/95/4/1460.full" ><span 
class="cmtt-10">SantaLucia 1998</span></a>), the GC content as well as the length of your primers
     (Figure&#x00A0;<a 
href="#x1-84004r4">1.18.2.4<!--tex4ht:ref: fig:verification_field --></a>).
     <!--l. 1981--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-84004r4"></a> <img 
src="pictures/primer_design_screenshots/verification_field.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.2.4: </span><span  
class="content">The &#8220;Manual Primer Design&#8221; dialog: verification.</span></div><!--tex4ht:label?: x1-84004r4 -->
     <!--l. 1986--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Click &#8221;OK&#8221; will save your primers to the primer library for the project under the single
     primer tab.
     </li></ul>
<!--l. 2004--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.18.3   </span> <a 
 id="x1-850001.18.3"></a>Primer Summary Table</h5>
     <ul class="itemize1">
     <li class="itemize">To view a summary table of all primers, click the primers &#8220;Summary table&#8221; icon in
     the toolbar drop down Layers menu (Figure&#x00A0;<a 
href="#x1-85001r1">1.18.3.1<!--tex4ht:ref: fig:primers_dropdown_menu --></a>). You can remove the primers
     from the views by unchecking the box.
     <!--l. 2010--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-85001r1"></a> <img 
src="pictures/primer_design_screenshots/primers_dropdown_menu.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.3.1: </span><span  
class="content">The primers &#8221;Summary table&#8221; icon in the toolbar drop down Layers
menu.</span></div><!--tex4ht:label?: x1-85001r1 -->
     <!--l. 2015--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The primers summary table (Figure&#x00A0;<a 
href="#x1-85002r2">1.18.3.2<!--tex4ht:ref: fig:primers_summary --></a>) displays all the primers from
     the project, as well as their name, description, strand, GC content, melting
     temperature (calculated according to <a 
href="http://www.pnas.org/content/95/4/1460.full" ><span 
class="cmtt-10">SantaLucia 1998</span></a>), location, length and
     sequence. You can navigate to each primer in the DNA sequence by clicking
     on it. Click the &#8221;Edit&#8221; icon to edit a primer, or the &#8221;Delete&#8221; icon to delete
     it.
     <!--l. 2019--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-85002r2"></a>                                                       <img 
src="pictures/primer_design_screenshots/primers_summary.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.18.3.2: </span><span  
class="content">The primers summary table.</span></div><!--tex4ht:label?: x1-85002r2 -->
     <!--l. 2023--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2027--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">1.19   </span> <a 
 id="x1-860001.19"></a>Cloning Wizard</h4>
<!--l. 2033--><p class="noindent" >The Cloning Wizard tool allows you to design and prepare your cloning experiments in an
easy and efficient way.
<!--l. 2035--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.1   </span> <a 
 id="x1-870001.19.1"></a>Opening the Cloning Wizard</h5>
     <ul class="itemize1">
     <li class="itemize">To  open  the  &#8221;Cloning  Wizard&#8221;,  click  the  corresponding  icon  in  the  toolbar
     (Figure&#x00A0;<a 
href="#x1-87001r1">1.19.1.1<!--tex4ht:ref: fig:clone_icon --></a>).
     <!--l. 2041--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-87001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/clone_icon.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.1.1: </span><span  
class="content">The &#8220;Cloning Wizard&#8221; icon in the toolbar.</span></div><!--tex4ht:label?: x1-87001r1 -->
     <!--l. 2046--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2064--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.2   </span> <a 
 id="x1-880001.19.2"></a>Cloning Procedure</h5>
     <ul class="itemize1">
     <li class="itemize">Type  in  a  &#8221;Project  name&#8221;.  &#8221;Restriction  Ligation&#8221;  is  selected  as  &#8221;Cloning
     procedure&#8221;  (Figure&#x00A0;<a 
href="#x1-88001r1">1.19.2.1<!--tex4ht:ref: fig:cloning_procedure --></a>).  More  cloning  procedure  as  well  as  the  &#8221;Require
     cloning services&#8221; option will be available soon.
     <!--l. 2072--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-88001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/cloning_procedure.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.2.1: </span><span  
class="content">The &#8221;Cloning Procedure&#8221; tab.</span></div><!--tex4ht:label?: x1-88001r1 -->
     <!--l. 2077--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2082--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.3   </span> <a 
 id="x1-890001.19.3"></a>In-house Cloning</h5>
     <ul class="itemize1">
     <li class="itemize">This flow is for in-house cloning, which means that you will construct the vector
     yourself.
     </li></ul>
                                                                         

                                                                         
<!--l. 2090--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.4   </span> <a 
 id="x1-900001.19.4"></a>In-house Cloning: Destination Vector</h5>
     <ul class="itemize1">
     <li class="itemize">To select a vector, drag and drop a file from the Materials box (Figure&#x00A0;<a 
href="#x1-90001r1">1.19.4.1<!--tex4ht:ref: fig:select_vector --></a>).
     This section will already be filled if you selected your vector to open the Cloning
     Wizard.
     <!--l. 2097--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-90001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/select_vector.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.4.1: </span><span  
class="content">The &#8221;Destination Vector&#8221; tab.</span></div><!--tex4ht:label?: x1-90001r1 -->
     <!--l. 2102--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The vector will appear on the right, and its meta data will be displayed on the left
     (Figure&#x00A0;<a 
href="#x1-90002r2">1.19.4.2<!--tex4ht:ref: fig:destination_vector --></a>).
     <!--l. 2106--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-90002r2"></a>                                                       <img 
src="pictures/cloning_wizard_screenshots/destination_vector.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.4.2: </span><span  
class="content">The &#8221;Destination Vector&#8221; tab.</span></div><!--tex4ht:label?: x1-90002r2 -->
     <!--l. 2110--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2114--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.5   </span> <a 
 id="x1-910001.19.5"></a>In-house Cloning: Select Insert</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8221;Select Insert&#8221; step enables the selection of the insert sequence for cloning
     into the destination vector. You can choose to add an insert sequence of amino
     acids or DNA. To add your insert, drag and drop a file from the Materials box.
     You can reverse complement the sequence by selecting the other strand orientation
     (Figure&#x00A0;<a 
href="#x1-91001r1">1.19.5.1<!--tex4ht:ref: fig:add_insert --></a>). This section will already be filled if you selected your insert to
     open the Cloning Wizard.
     <!--l. 2121--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-91001r1"></a>                                                       <img 
src="pictures/cloning_wizard_screenshots/add_insert.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.5.1: </span><span  
class="content">The &#8221;Select Insert&#8221; tab.</span></div><!--tex4ht:label?: x1-91001r1 -->
     <!--l. 2125--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">You can codon optimize the insert or, in the case of a DNA insertion, choose to keep the
     DNA as it is. Select the appropriate option by checking its checkbox. If you would like to
     codon optimize the insert, select the host organism from the drop down menu. You can
     also select specific sites to avoid (Figure&#x00A0;<a 
href="#x1-91002r2">1.19.5.2<!--tex4ht:ref: fig:codon_optimize --></a>).
     <!--l. 2129--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-91002r2"></a> <img 
src="pictures/cloning_wizard_screenshots/codon_optimize.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.5.2: </span><span  
class="content">The &#8221;Select Insert&#8221; tab: codon optimization.</span></div><!--tex4ht:label?: x1-91002r2 -->
     <!--l. 2134--><p class="noindent" ></div><hr class="endfigure">
                                                                         

                                                                         
     </li>
     <li class="itemize">You will then need to specify the insert region by selecting restriction sites in the
     destination vector (Figure&#x00A0;<a 
href="#x1-91003r3">1.19.5.3<!--tex4ht:ref: fig:restriction_sites_selected --></a>). You can configure the restriction site list by
     opening the restriction site dialog where a preset can be configured. Your desired sites
     preset will automatically be added to the &#8221;start&#8221; and &#8221;end&#8221; restriction sites list for you
     to select. The preset will also be displayed on the vector in the right panel.
     Single and double cutters in the destination vector will be shown in black and
     other enzymes will be greyed out. As the restriction sites are selected they
     will appear in the mining cloning map along the bottom of the left panel
     (Figure&#x00A0;<a 
href="#x1-91003r3">1.19.5.3<!--tex4ht:ref: fig:restriction_sites_selected --></a>).
     <!--l. 2141--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-91003r3"></a>                                                       <img 
src="pictures/cloning_wizard_screenshots/restriction_sites_selected.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.5.3: </span><span  
class="content">The &#8221;Select Insert&#8221; tab: cut sites.</span></div><!--tex4ht:label?: x1-91003r3 -->
     <!--l. 2145--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2149--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.6   </span> <a 
 id="x1-920001.19.6"></a>In-house Cloning: Design Insert</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8221;Design Insert&#8221; step defines how the insert sequence is retrieved, either by
     submitting an &#8221;Order&#8221; from a DNA synthesis provider or choosing to &#8221;PCR the
     insert&#8221; out from a source plasmid. Select the option you would like to use by
     checking its checkbox (Figure&#x00A0;<a 
href="#x1-92001r1">1.19.6.1<!--tex4ht:ref: fig:order_pcr_insert --></a>).
     <!--l. 2155--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-92001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/order_pcr_insert.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.6.1: </span><span  
class="content">The &#8221;Design Insert&#8221; tab.</span></div><!--tex4ht:label?: x1-92001r1 -->
     <!--l. 2160--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 2162--><p class="noindent" >
     <h5 class="subsubsectionHead"><span class="titlemark">1.19.7   </span> <a 
 id="x1-930001.19.7"></a>Design Insert: Order Insert</h5>
     </li>
     <li class="itemize">Flanking sticky ends can be added to the ends of the insert sequence by adding a &#8221;5&#8217;
     location restriction site&#8221; and a &#8221;3&#8217; location restriction site.&#8221; The drop down menu of
     restriction sites consists of a smart list of enzymes, filtered according to compatible
     sticky ends to those prior selected in the destination vector. An optional &#8221;Leader
     sequence&#8221; can be added to ensure efficiency during restriction enzyme digestion
     (Figure&#x00A0;<a 
href="#x1-93001r1">1.19.7.1<!--tex4ht:ref: fig:order_insert --></a>).
     </li>
     <li class="itemize">As the flanking sequences are selected, they are added on either end of the
     insert sequence in the right panel. The insert sequence in the right panel is
     editable.
                                                                         

                                                                         
     <!--l. 2168--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-93001r1"></a>                                                       <img 
src="pictures/cloning_wizard_screenshots/order_insert.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.7.1: </span><span  
class="content">The &#8221;Design Insert&#8221; tab: order insert.</span></div><!--tex4ht:label?: x1-93001r1 -->
     <!--l. 2172--><p class="noindent" ></div><hr class="endfigure">
     <!--l. 2174--><p class="noindent" >
     <h5 class="subsubsectionHead"><span class="titlemark">1.19.8   </span> <a 
 id="x1-940001.19.8"></a>Design Insert: PCR Insert</h5>
     </li>
     <li class="itemize">If you decide to PCR your insert, then you can design PCR primers with compatible
     sticky ends for amplifying your vector from the source plasmid. When you require a
     source plasmid, you can use a plasmid from your inventory or order a plasmid. Check
     the appropriate checkbox according to your choice. The vector will then be
     automatically added to the cart in the next step. To add the plasmid, drag and drop a
     file from the Materials box (Figure&#x00A0;<a 
href="#x1-94001r1">1.19.8.1<!--tex4ht:ref: fig:source_plasmid --></a>).
     <!--l. 2178--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-94001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/source_plasmid.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.8.1: </span><span  
class="content">The &#8221;Source plasmid&#8221; drop down in the &#8221;Design Insert&#8221; tab.</span></div><!--tex4ht:label?: x1-94001r1 -->
     <!--l. 2183--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">For manual design of primers fill in all the fields like the regular primer dialog flow
     (Figure&#x00A0;<a 
href="#x1-94002r2">1.19.8.2<!--tex4ht:ref: fig:design_primers --></a>). The drop down menu for cut sites again consists of a smart list of
     enzymes, filtered according to compatible sticky ends to those prior selected in the
     destination vector. If the leader sequence adds a restriction site then you will be
     warned.
     <!--l. 2191--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-94002r2"></a> <img 
src="pictures/cloning_wizard_screenshots/design_primers.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.8.2: </span><span  
class="content">The &#8221;Design Insert&#8221; tab: design primers.</span></div><!--tex4ht:label?: x1-94002r2 -->
     <!--l. 2196--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">Selected sites and primer sequences are highlighted on the plasmid on the right
     panel (Figure&#x00A0;<a 
href="#x1-94003r3">1.19.8.3<!--tex4ht:ref: fig:primers_overview --></a>) and in the cloning minimap at the bottom of the left
     panel.
     <!--l. 2202--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-94003r3"></a>                                                       <img 
src="pictures/cloning_wizard_screenshots/primers_overview.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.8.3: </span><span  
class="content">The &#8221;Design Insert&#8221; tab: design primers.</span></div><!--tex4ht:label?: x1-94003r3 -->
     <!--l. 2206--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Verification&#8221; drop down displays information about your primers
     (Figure&#x00A0;<a 
href="#x1-94004r4">1.19.8.4<!--tex4ht:ref: fig:verification --></a>).
     <!--l. 2210--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-94004r4"></a> <img 
src="pictures/cloning_wizard_screenshots/verification.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.8.4: </span><span  
class="content">The &#8221;Verification&#8221; drop down in the &#8221;Design Insert&#8221; tab.</span></div><!--tex4ht:label?: x1-94004r4 -->
     <!--l. 2215--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 2219--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">1.19.9   </span> <a 
 id="x1-950001.19.9"></a>In-house Cloning: Finalize</h5>
     <ul class="itemize1">
     <li class="itemize">The &#8221;Finalize&#8221; step displays a summary of the cloning procedure and allows you
     to add elements to your cart and select providers by clicking the &#8221;Cart&#8221; icon
     (Figure&#x00A0;<a 
href="#x1-95001r1">1.19.9.1<!--tex4ht:ref: fig:cloning_summary --></a>). They can also be exported in text format for ease of copying
     and pasting into your preferred DNA synthesis providers website.
     <!--l. 2225--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-95001r1"></a> <img 
src="pictures/cloning_wizard_screenshots/cloning_summary.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.9.1: </span><span  
class="content">The cloning summary in the &#8221;Finalize&#8221; tab.</span></div><!--tex4ht:label?: x1-95001r1 -->
     <!--l. 2230--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The &#8221;Compilation summary&#8221; displays prospective Synthesis and Design errors
     (Figure&#x00A0;<a 
href="#x1-95002r2">1.19.9.2<!--tex4ht:ref: fig:compilation_summary --></a>).
     <!--l. 2234--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-95002r2"></a> <img 
src="pictures/cloning_wizard_screenshots/compilation_summary.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.19.9.2: </span><span  
class="content">The &#8221;Compilation summary&#8221; drop down in the &#8221;Finalize&#8221; tab.</span></div><!--tex4ht:label?: x1-95002r2 -->
     <!--l. 2239--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The final cloned product is visible on the right panel and the cloning minimap,
     displaying the inserted sequence inside the destination vector. Select &#8221;Finish&#8221; to create
     your new cloning project.
     </li></ul>
<!--l. 2369--><p class="noindent" >
                                                                         

                                                                         
<h3 class="sectionHead"><span class="titlemark">2   </span> <a 
 id="x1-960002"></a>Flows</h3>
<!--l. 2377--><p class="noindent" >
<h4 class="subsectionHead"><span class="titlemark">2.1   </span> <a 
 id="x1-970002.1"></a>Sequence Annotation</h4>
<!--l. 2379--><p class="noindent" >This simple flow describes the process of manual sequence annotation.
<!--l. 2381--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.1.1   </span> <a 
 id="x1-980002.1.1"></a>Pasting your Sequence</h5>
     <ul class="itemize1">
     <li class="itemize">Paste  your  sequence  into  a  new  project.  The  &#8221;Translation  Settings&#8221;  dialog
     (Figure&#x00A0;<a 
href="#x1-98001r1">2.1.1.1<!--tex4ht:ref: fig:translation_settings_dna_original --></a>) will pop up. Choose the appropriate option. In the example below,
     the DNA is kept as original. You can find more information about the translation
     feature in section&#x00A0;<a 
href="#x1-680001.14">1.14<!--tex4ht:ref: subsec:translation --></a>.
     <!--l. 2387--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-98001r1"></a> <img 
src="pictures/annotate_sequence_screenshots/translation_settings_dna_original.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.1.1.1: </span><span  
class="content">The &#8220;Translation Settings&#8221; dialog.</span></div><!--tex4ht:label?: x1-98001r1 -->
     <!--l. 2392--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">The whole sequence will be set as CDS by default (Figure&#x00A0;<a 
href="#x1-98002r2">2.1.1.2<!--tex4ht:ref: fig:pasted_sequence --></a>).
     <!--l. 2396--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-98002r2"></a> <img 
src="pictures/annotate_sequence_screenshots/pasted_sequence.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.1.1.2: </span><span  
class="content">The sequence set as CDS.</span></div><!--tex4ht:label?: x1-98002r2 -->
     <!--l. 2401--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">To change the feature key, go to the &#8221;Part&#8221; tab at the bottom of the screen and select a
     feature from the &#8221;Feature key&#8221; drop down menu (Figure&#x00A0;<a 
href="#x1-98003r3">2.1.1.3<!--tex4ht:ref: fig:change_feature_key --></a>).
     <!--l. 2405--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-98003r3"></a>                                                       <img 
src="pictures/annotate_sequence_screenshots/change_feature_key.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.1.1.3: </span><span  
class="content">The &#8221;Feature Key&#8221; drop down menu in the &#8221;Part&#8221; tab.</span></div><!--tex4ht:label?: x1-98003r3 -->
     <!--l. 2409--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2414--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.1.2   </span> <a 
 id="x1-990002.1.2"></a>Annotating your Sequence</h5>
                                                                         

                                                                         
     <ul class="itemize1">
     <li class="itemize">To annotate a sequence, go to the DNA view, select it, right click and select
     &#8221;Annotate selection&#8221; from the drop down menu (Figure&#x00A0;<a 
href="#x1-99001r1">2.1.2.1<!--tex4ht:ref: fig:annotate_selection --></a>).
     <!--l. 2421--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-99001r1"></a> <img 
src="pictures/annotate_sequence_screenshots/annotate_selection.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.1.2.1: </span><span  
class="content">&#8221;Annotate selection&#8221; from the right click drop down menu.</span></div><!--tex4ht:label?: x1-99001r1 -->
     <!--l. 2426--><p class="noindent" ></div><hr class="endfigure">
     </li>
     <li class="itemize">A new part will be created from your selection. It will be set as a CDS by default. You
     can change the feature key as described above.
     </li></ul>
<!--l. 2433--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.1.3   </span> <a 
 id="x1-1000002.1.3"></a>Renaming your part</h5>
     <ul class="itemize1">
     <li class="itemize">To  rename  a  part,  type  in  a  name  in  the  &#8221;Name&#8221;  section  of  the  &#8221;Part&#8221;  tab
     (Figure&#x00A0;<a 
href="#x1-100001r1">2.1.3.1<!--tex4ht:ref: fig:renaming_part --></a>).
     <!--l. 2439--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-100001r1"></a>                                                       <img 
src="pictures/annotate_sequence_screenshots/renaming_part.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.1.3.1: </span><span  
class="content">The &#8221;Name&#8221; section in the &#8221;Part&#8221; tab.</span></div><!--tex4ht:label?: x1-100001r1 -->
     <!--l. 2443--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2446--><p class="noindent" >
                                                                         

                                                                         
<h4 class="subsectionHead"><span class="titlemark">2.2   </span> <a 
 id="x1-1010002.2"></a>Gene Amplification</h4>
<!--l. 2450--><p class="noindent" >This simple flow describes the process of gene amplification.
<!--l. 2452--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.2.1   </span> <a 
 id="x1-1020002.2.1"></a>Finding your Gene of Interest</h5>
     <ul class="itemize1">
     <li class="itemize">Find your gene of interest. This can be done:
         <ul class="itemize2">
         <li class="itemize">via our NCBI search tool (see section&#x00A0;<a 
href="#x1-390001.8">1.8<!--tex4ht:ref: subsec:NCBI --></a>)
         </li>
         <li class="itemize">by finding information about the gene location in an article and using our
         &#8221;Go to&#8221; feature (see section&#x00A0;<a 
href="#x1-310001.5">1.5<!--tex4ht:ref: subsec:go_to --></a>)
         </li>
         <li class="itemize">by using our project search tool (see section&#x00A0;<a 
href="#x1-370001.7">1.7<!--tex4ht:ref: subsec:search_projects --></a>)
         </li>
         <li class="itemize">or by importing a file (see section&#x00A0;<a 
href="#x1-170001.3">1.3<!--tex4ht:ref: subsec:import_export --></a>)</li></ul>
     </li>
     <li class="itemize">In this example, the yjhQ gene is found via the project search tool (Figure&#x00A0;<a 
href="#x1-102001r1">2.2.1.1<!--tex4ht:ref: fig:yjhq_search --></a>).
     <!--l. 2466--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-102001r1"></a>                                                       <img 
src="pictures/gene_amplification_screenshots/yjhq_search.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.2.1.1: </span><span  
class="content">The yjhQ gene.</span></div><!--tex4ht:label?: x1-102001r1 -->
     <!--l. 2470--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
<!--l. 2474--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.2.2   </span> <a 
 id="x1-1030002.2.2"></a>Commenting the Part</h5>
     <ul class="itemize1">
     <li class="itemize">Add a comment to the part (Figure&#x00A0;<a 
href="#x1-103001r1">2.2.2.1<!--tex4ht:ref: fig:gene_comment --></a>) by right clicking on the part and
     selecting &#8221;Add comment&#8221; in the drop down menu (see section&#x00A0;<a 
href="#x1-480001.10">1.10<!--tex4ht:ref: subsec:user_comments --></a>).
     <!--l. 2481--><p class="noindent" ><hr class="figure"><div class="figure" 
><a 
 id="x1-103001r1"></a>                                                       <img 
src="pictures/gene_amplification_screenshots/gene_comment.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2.2.2.1: </span><span  
class="content">Comment on the yjhQ gene.</span></div><!--tex4ht:label?: x1-103001r1 -->
     <!--l. 2485--><p class="noindent" ></div><hr class="endfigure">
     </li></ul>
                                                                         

                                                                         
<!--l. 2489--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.2.3   </span> <a 
 id="x1-1040002.2.3"></a>Designing Primers</h5>
     <ul class="itemize1">
     <li class="itemize">Design primers (see section&#x00A0;<a 
href="#x1-820001.18">1.18<!--tex4ht:ref: subsec:primer_design --></a>).
     </li></ul>
<!--l. 2498--><p class="noindent" >
<h5 class="subsubsectionHead"><span class="titlemark">2.2.4   </span> <a 
 id="x1-1050002.2.4"></a>Ordering</h5>
     <ul class="itemize1">
     <li class="itemize">Place your order (see section&#x00A0;<a 
href="#x1-590001.12">1.12<!--tex4ht:ref: subsec:ordering_pricing --></a>).
     </li></ul>
<!--l. 2506--><p class="noindent" >
                                                                         

                                                                         
 
</body></html> 

                                                                         


