<span class="cmcsc-10x-x-172">G<span class="small-caps">e</span><span
class="small-caps">n</span><span class="small-caps">o</span><span
class="small-caps">m</span><span class="small-caps">e</span> C<span
class="small-caps">o</span><span class="small-caps">m</span><span
class="small-caps">p</span><span class="small-caps">i</span><span
class="small-caps">l</span><span class="small-caps">e</span><span
class="small-caps">r</span></span>\
\

\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\
\
 <span class="cmbx-12x-x-172">User Manual</span>\
\

\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\
\
 <span class="cmcsc-10x-x-144">V.0.5</span>\
\
 <span class="cmr-12">23rd</span><span
class="cmr-12"> November</span><span class="cmr-12"> 2014</span>\
\
 <span class="cmcsc-10x-x-172"><span class="small-caps">d</span><span
class="small-caps">r</span><span class="small-caps">a</span><span
class="small-caps">f</span><span class="small-caps">t</span></span>\
\
 ![PIC](pictures/genomecompiler.png)

### <span id="x1-1000"></span>Contents {.likesectionHead}

<div class="tableofcontents">

<span class="sectionToc">1 <span
id="QQ2-1-2">[Tutorials](#x1-20001)</span></span>\
 <span class="subsectionToc">1.1 <span id="QQ2-1-3">[Getting
Started](#x1-30001.1)</span></span>\
  <span class="subsubsectionToc">1.1.1 <span id="QQ2-1-4">[Creating an
Account](#x1-40001.1.1)</span></span>\
  <span class="subsubsectionToc">1.1.2 <span id="QQ2-1-6">[Logging
in](#x1-50001.1.2)</span></span>\
 <span class="subsectionToc">1.2 <span id="QQ2-1-8">[Genome Compiler
User Interface](#x1-60001.2)</span></span>\
  <span class="subsubsectionToc">1.2.1 <span id="QQ2-1-9">[Overview of
Genome Compiler User Interface](#x1-70001.2.1)</span></span>\
  <span class="subsubsectionToc">1.2.2 <span
id="QQ2-1-14">[Tooltips](#x1-80001.2.2)</span></span>\
  <span class="subsubsectionToc">1.2.3 <span id="QQ2-1-16">[Minimizing
Materials box and Properties](#x1-90001.2.3)</span></span>\
  <span class="subsubsectionToc">1.2.4 <span id="QQ2-1-19">[Maximizing
Materials box and Properties](#x1-100001.2.4)</span></span>\
  <span class="subsubsectionToc">1.2.5 <span id="QQ2-1-22">[Re-arranging
Project Tabs](#x1-110001.2.5)</span></span>\
  <span class="subsubsectionToc">1.2.6 <span id="QQ2-1-28">[Viewing
Projects](#x1-120001.2.6)</span></span>\
  <span class="subsubsectionToc">1.2.7 <span id="QQ2-1-34">[Abstraction
Layers](#x1-130001.2.7)</span></span>\
  <span class="subsubsectionToc">1.2.8 <span id="QQ2-1-39">[Numbering of
Base Pairs and Amino Acids](#x1-140001.2.8)</span></span>\
  <span class="subsubsectionToc">1.2.9 <span id="QQ2-1-43">[Properties
Section](#x1-150001.2.9)</span></span>\
  <span class="subsubsectionToc">1.2.10 <span id="QQ2-1-49">[Options
Menu](#x1-160001.2.10)</span></span>\
 <span class="subsectionToc">1.3 <span id="QQ2-1-52">[Importing and
Exporting](#x1-170001.3)</span></span>\
  <span class="subsubsectionToc">1.3.1 <span id="QQ2-1-53">[Importing
Files](#x1-180001.3.1)</span></span>\
  <span class="subsubsectionToc">1.3.2 <span id="QQ2-1-58">[Importing
Parts](#x1-190001.3.2)</span></span>\
  <span class="subsubsectionToc">1.3.3 <span id="QQ2-1-61">[Exporting
Files](#x1-200001.3.3)</span></span>\
 <span class="subsectionToc">1.4 <span
id="QQ2-1-64">[Editing](#x1-210001.4)</span></span>\
  <span class="subsubsectionToc">1.4.1 <span id="QQ2-1-65">[Basic
Copy/Cut/Paste/Delete/Undo/Redo](#x1-220001.4.1)</span></span>\
  <span class="subsubsectionToc">1.4.2 <span id="QQ2-1-69">[Drag and
Drop Functionality](#x1-230001.4.2)</span></span>\
  <span class="subsubsectionToc">1.4.3 <span id="QQ2-1-72">[Adding New
Parts](#x1-240001.4.3)</span></span>\
  <span class="subsubsectionToc">1.4.4 <span id="QQ2-1-77">[Adding Amino
Acids](#x1-250001.4.4)</span></span>\
  <span class="subsubsectionToc">1.4.5 <span id="QQ2-1-79">[Reverse
Complement](#x1-260001.4.5)</span></span>\
  <span class="subsubsectionToc">1.4.6 <span id="QQ2-1-82">[Merging and
Splitting Parts](#x1-270001.4.6)</span></span>\
  <span class="subsubsectionToc">1.4.7 <span id="QQ2-1-87">[Selecting
Strand](#x1-280001.4.7)</span></span>\
  <span class="subsubsectionToc">1.4.8 <span id="QQ2-1-93">[Mutation
Wheel](#x1-290001.4.8)</span></span>\
  <span class="subsubsectionToc">1.4.9 <span id="QQ2-1-97">[Basic Local
Alignment Search Tool (BLAST)](#x1-300001.4.9)</span></span>\
 <span class="subsectionToc">1.5 <span id="QQ2-1-99">[Go
to](#x1-310001.5)</span></span>\
  <span class="subsubsectionToc">1.5.1 <span id="QQ2-1-100">[Openging
the ”Go to” Dialog](#x1-320001.5.1)</span></span>\
  <span class="subsubsectionToc">1.5.2 <span id="QQ2-1-104">[”Go to”
Dialog](#x1-330001.5.2)</span></span>\
 <span class="subsectionToc">1.6 <span id="QQ2-1-106">[Search in the
Materials box](#x1-340001.6)</span></span>\
  <span class="subsubsectionToc">1.6.1 <span id="QQ2-1-107">[Searching
in the Materials box](#x1-350001.6.1)</span></span>\
  <span class="subsubsectionToc">1.6.2 <span
id="QQ2-1-111">[Example](#x1-360001.6.2)</span></span>\
 <span class="subsectionToc">1.7 <span id="QQ2-1-113">[Search in
Projects](#x1-370001.7)</span></span>\
  <span class="subsubsectionToc">1.7.1 <span id="QQ2-1-114">[Searching
in Projects](#x1-380001.7.1)</span></span>\
 <span class="subsectionToc">1.8 <span id="QQ2-1-119">[NCBI
Search](#x1-390001.8)</span></span>\
  <span class="subsubsectionToc">1.8.1 <span id="QQ2-1-120">[Searching
in the NCBI Database](#x1-400001.8.1)</span></span>\
  <span class="subsubsectionToc">1.8.2 <span id="QQ2-1-124">[Downloaded
and Downloading Files Window](#x1-410001.8.2)</span></span>\
 <span class="subsectionToc">1.9 <span id="QQ2-1-126">[Sharing and
Collaboration](#x1-420001.9)</span></span>\
  <span class="subsubsectionToc">1.9.1 <span id="QQ2-1-127">[Cloud
Folders](#x1-430001.9.1)</span></span>\
  <span class="subsubsectionToc">1.9.2 <span id="QQ2-1-129">[Creating
New Folders](#x1-440001.9.2)</span></span>\
  <span class="subsubsectionToc">1.9.3 <span id="QQ2-1-131">[Creating
Subfolders](#x1-450001.9.3)</span></span>\
  <span class="subsubsectionToc">1.9.4 <span id="QQ2-1-135">[Sharing
Cloud Folders](#x1-460001.9.4)</span></span>\
  <span class="subsubsectionToc">1.9.5 <span id="QQ2-1-142">[Creating
New Folders via the ”Cloud Folder Settings”
dialog](#x1-470001.9.5)</span></span>\
 <span class="subsectionToc">1.10 <span id="QQ2-1-144">[User
Comments](#x1-480001.10)</span></span>\
  <span class="subsubsectionToc">1.10.1 <span id="QQ2-1-145">[User
Comments Indicator](#x1-490001.10.1)</span></span>\
  <span class="subsubsectionToc">1.10.2 <span id="QQ2-1-147">[Navigating
to Comments](#x1-500001.10.2)</span></span>\
  <span class="subsubsectionToc">1.10.3 <span id="QQ2-1-149">[Adding
Comments](#x1-510001.10.3)</span></span>\
  <span class="subsubsectionToc">1.10.4 <span id="QQ2-1-154">[Multiple
Users making Comments in a Single
Project](#x1-520001.10.4)</span></span>\
  <span class="subsubsectionToc">1.10.5 <span id="QQ2-1-156">[User
Comments Summary Table](#x1-530001.10.5)</span></span>\
 <span class="subsectionToc">1.11 <span id="QQ2-1-159">[Track
Changes](#x1-540001.11)</span></span>\
  <span class="subsubsectionToc">1.11.1 <span id="QQ2-1-160">[Track
Changes Indicator](#x1-550001.11.1)</span></span>\
  <span class="subsubsectionToc">1.11.2 <span id="QQ2-1-162">[Navigating
to Changes](#x1-560001.11.2)</span></span>\
  <span class="subsubsectionToc">1.11.3 <span id="QQ2-1-164">[Multiple
Users making Changes in a Single
Project](#x1-570001.11.3)</span></span>\
  <span class="subsubsectionToc">1.11.4 <span id="QQ2-1-166">[Track
Changes Summary Table](#x1-580001.11.4)</span></span>\
 <span class="subsectionToc">1.12 <span id="QQ2-1-169">[Ordering and
Pricing](#x1-590001.12)</span></span>\
  <span class="subsubsectionToc">1.12.1 <span id="QQ2-1-170">[Pricing
for Projects](#x1-600001.12.1)</span></span>\
  <span class="subsubsectionToc">1.12.2 <span id="QQ2-1-173">[Pricing
for Primers](#x1-610001.12.2)</span></span>\
 <span class="subsectionToc">1.13 <span
id="QQ2-1-177">[Compilation](#x1-620001.13)</span></span>\
  <span class="subsubsectionToc">1.13.1 <span id="QQ2-1-178">[Selecting
Rules for Activation](#x1-630001.13.1)</span></span>\
  <span class="subsubsectionToc">1.13.2 <span id="QQ2-1-182">[Compiling
Projects](#x1-640001.13.2)</span></span>\
  <span class="subsubsectionToc">1.13.3 <span id="QQ2-1-185">[Error and
Warning Indicators](#x1-650001.13.3)</span></span>\
  <span class="subsubsectionToc">1.13.4 <span
id="QQ2-1-189">[Compilation Summary
Table](#x1-660001.13.4)</span></span>\
  <span class="subsubsectionToc">1.13.5 <span id="QQ2-1-192">[Navigating
to Errors and Warnings](#x1-670001.13.5)</span></span>\
 <span class="subsectionToc">1.14 <span id="QQ2-1-194">[Back
Translation](#x1-680001.14)</span></span>\
  <span class="subsubsectionToc">1.14.1 <span id="QQ2-1-195">[How to
Start Translating](#x1-690001.14.1)</span></span>\
  <span class="subsubsectionToc">1.14.2 <span
id="QQ2-1-198">[“Translation Settings”
Dialog](#x1-700001.14.2)</span></span>\
  <span class="subsubsectionToc">1.14.3 <span id="QQ2-1-203">[Proportion
of each triplet](#x1-710001.14.3)</span></span>\
 <span class="subsectionToc">1.15 <span id="QQ2-1-206">[Combinatorial
Design](#x1-720001.15)</span></span>\
  <span class="subsubsectionToc">1.15.1 <span id="QQ2-1-207">[Circuit
Part Combinatorial](#x1-730001.15.1)</span></span>\
  <span class="subsubsectionToc">1.15.2 <span id="QQ2-1-211">[Protein
Engineering Combinatorial](#x1-740001.15.2)</span></span>\
 <span class="subsectionToc">1.16 <span id="QQ2-1-217">[Managing
Restriction Sites](#x1-750001.16)</span></span>\
  <span class="subsubsectionToc">1.16.1 <span id="QQ2-1-218">[Opening
the Dialog](#x1-760001.16.1)</span></span>\
  <span class="subsubsectionToc">1.16.2 <span
id="QQ2-1-224">[”Restriction Site” Settings
Dialog](#x1-770001.16.2)</span></span>\
 <span class="subsectionToc">1.17 <span id="QQ2-1-229">[Virtual
Digest](#x1-780001.17)</span></span>\
  <span class="subsubsectionToc">1.17.1 <span id="QQ2-1-230">[Running a
Digest](#x1-790001.17.1)</span></span>\
  <span class="subsubsectionToc">1.17.2 <span
id="QQ2-1-232">[“Restriction Sites”
Window](#x1-800001.17.2)</span></span>\
  <span class="subsubsectionToc">1.17.3 <span id="QQ2-1-234">[”Digest”
Summary Table](#x1-810001.17.3)</span></span>\
 <span class="subsectionToc">1.18 <span id="QQ2-1-240">[Primer
Design](#x1-820001.18)</span></span>\
  <span class="subsubsectionToc">1.18.1 <span id="QQ2-1-241">[Opening
the Primer Design Dialog](#x1-830001.18.1)</span></span>\
  <span class="subsubsectionToc">1.18.2 <span id="QQ2-1-242">[Primer
Design Dialog](#x1-840001.18.2)</span></span>\
  <span class="subsubsectionToc">1.18.3 <span id="QQ2-1-247">[Primer
Summary Table](#x1-850001.18.3)</span></span>\
 <span class="subsectionToc">1.19 <span id="QQ2-1-250">[Cloning
Wizard](#x1-860001.19)</span></span>\
  <span class="subsubsectionToc">1.19.1 <span id="QQ2-1-251">[Opening
the Cloning Wizard](#x1-870001.19.1)</span></span>\
  <span class="subsubsectionToc">1.19.2 <span id="QQ2-1-253">[Cloning
Procedure](#x1-880001.19.2)</span></span>\
  <span class="subsubsectionToc">1.19.3 <span id="QQ2-1-255">[In-house
Cloning](#x1-890001.19.3)</span></span>\
  <span class="subsubsectionToc">1.19.4 <span id="QQ2-1-256">[In-house
Cloning: Destination Vector](#x1-900001.19.4)</span></span>\
  <span class="subsubsectionToc">1.19.5 <span id="QQ2-1-259">[In-house
Cloning: Select Insert](#x1-910001.19.5)</span></span>\
  <span class="subsubsectionToc">1.19.6 <span id="QQ2-1-263">[In-house
Cloning: Design Insert](#x1-920001.19.6)</span></span>\
  <span class="subsubsectionToc">1.19.7 <span id="QQ2-1-265">[Design
Insert: Order Insert](#x1-930001.19.7)</span></span>\
  <span class="subsubsectionToc">1.19.8 <span id="QQ2-1-267">[Design
Insert: PCR Insert](#x1-940001.19.8)</span></span>\
  <span class="subsubsectionToc">1.19.9 <span id="QQ2-1-272">[In-house
Cloning: Finalize](#x1-950001.19.9)</span></span>\
<span class="sectionToc">2 <span
id="QQ2-1-275">[Flows](#x1-960002)</span></span>\
 <span class="subsectionToc">2.1 <span id="QQ2-1-276">[Sequence
Annotation](#x1-970002.1)</span></span>\
  <span class="subsubsectionToc">2.1.1 <span id="QQ2-1-277">[Pasting
your Sequence](#x1-980002.1.1)</span></span>\
  <span class="subsubsectionToc">2.1.2 <span id="QQ2-1-281">[Annotating
your Sequence](#x1-990002.1.2)</span></span>\
  <span class="subsubsectionToc">2.1.3 <span id="QQ2-1-283">[Renaming
your part](#x1-1000002.1.3)</span></span>\
 <span class="subsectionToc">2.2 <span id="QQ2-1-285">[Gene
Amplification](#x1-1010002.2)</span></span>\
  <span class="subsubsectionToc">2.2.1 <span id="QQ2-1-286">[Finding
your Gene of Interest](#x1-1020002.2.1)</span></span>\
  <span class="subsubsectionToc">2.2.2 <span id="QQ2-1-288">[Commenting
the Part](#x1-1030002.2.2)</span></span>\
  <span class="subsubsectionToc">2.2.3 <span id="QQ2-1-290">[Designing
Primers](#x1-1040002.2.3)</span></span>\
  <span class="subsubsectionToc">2.2.4 <span
id="QQ2-1-291">[Ordering](#x1-1050002.2.4)</span></span>

</div>

### <span class="titlemark">1 </span> <span id="x1-20001"></span>Tutorials {.sectionHead}

#### <span class="titlemark">1.1 </span> <span id="x1-30001.1"></span>Getting Started {.subsectionHead}

Welcome to Genome Compiler! This tutorial will show you how to get
started with using our software.

##### <span class="titlemark">1.1.1 </span> <span id="x1-40001.1.1"></span>Creating an Account {.subsubsectionHead}

-   If you do not have an account yet, sign up at [<span
    class="cmtt-10">designer.genomecompiler.com/</span> <span
    class="cmtt-10">users/sign\_up</span>](https://designer.genomecompiler.com/users/sign_up)
    (Figure [1.1.1.1](#x1-4001r1)). You will receive an email welcoming
    you to the software. Activate your account by clicking the link
    provided in the email.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-4001r1"></span>
    ![PIC](pictures/getting_started_screenshots/sign_up.png)\
    <div class="caption">

    <span class="id">Figure 1.1.1.1: </span><span
    class="content">Signing up to Genome Compiler.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.1.2 </span> <span id="x1-50001.1.2"></span>Logging in {.subsubsectionHead}

-   Once you have an account, simply login at [<span
    class="cmtt-10">designer.genomecompiler.com</span>](https://designer.genomecompiler.com/)
    (Figure [1.1.2.1](#x1-5001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-5001r1"></span>
    ![PIC](pictures/getting_started_screenshots/login.png)\
    <div class="caption">

    <span class="id">Figure 1.1.2.1: </span><span
    class="content">Logging in to Genome Compiler.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You are now ready to go! Check out the rest of the manual to learn
    more about the different features of our software.

#### <span class="titlemark">1.2 </span> <span id="x1-60001.2"></span>Genome Compiler User Interface {.subsectionHead}

Genome Compiler offers a customizable user friendly interface.

##### <span class="titlemark">1.2.1 </span> <span id="x1-70001.2.1"></span>Overview of Genome Compiler User Interface {.subsubsectionHead}

------------------------------------------------------------------------

<div class="figure">

<span id="x1-7001r1"></span>
![PIC](pictures/UI_screenshots/general_overview.png)\

<div class="caption">

<span class="id">Figure 1.2.1.1: </span><span class="content">General
view of the software.</span>

</div>

</div>

------------------------------------------------------------------------

-   The toolbar at the top of the screen (Figure [1.2.1.2](#x1-7002r2))
    contains icons for file operations, standard editing (see
    section [1.4](#x1-210001.4)) and tools.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-7002r2"></span>
    ![PIC](pictures/UI_screenshots/toolbar_description.png)\
    <div class="caption">

    <span class="id">Figure 1.2.1.2: </span><span class="content">The
    toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The Materials box on the left of the screen
    (Figure [1.2.1.3](#x1-7003r3)) gives you access to all your files.
    The search tool helps you find your files easily (see
    section [1.6](#x1-340001.6)). You can also use it to search through
    the NCBI database (see section [1.8](#x1-390001.8)). The Materials
    box provides a number of icons for file operations and tools.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-7003r3"></span>
    ![PIC](pictures/UI_screenshots/materials_box_description.png)\
    <div class="caption">

    <span class="id">Figure 1.2.1.3: </span><span class="content">The
    Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The properties section at the bottom of the screen
    (Figure [1.2.1.4](#x1-7004r4)) contains different tabs providing
    information on your project, parts, selection, compilation and
    restriction digests (more information on the properties section can
    be found in section [1.2.9](#x1-150001.2.9)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-7004r4"></span>
    ![PIC](pictures/UI_screenshots/properties_section_description.png)\
    <div class="caption">

    <span class="id">Figure 1.2.1.4: </span><span class="content">The
    properties section.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.2 </span> <span id="x1-80001.2.2"></span>Tooltips {.subsubsectionHead}

-   Many tooltips are available, describing dialogs, icons, etc. Simply
    hover over the item to see the tooltip. For example, information
    regarding the amino acids is displayed when you hover over them
    (Figure [1.2.2.1](#x1-8001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-8001r1"></span>
    ![PIC](pictures/UI_screenshots/aa_tooltip.png)\
    <div class="caption">

    <span class="id">Figure 1.2.2.1: </span><span class="content">The
    tooltip describing the amino acids.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.3 </span> <span id="x1-90001.2.3"></span>Minimizing Materials box and Properties {.subsubsectionHead}

-   The Materials box and the properties section can be minimized by
    clicking the appropriate icon (Figures [1.2.3.1](#x1-9001r1)
    and [1.2.3.2](#x1-9002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-9001r1"></span>
    ![PIC](pictures/UI_screenshots/normal.png)\
    <div class="caption">

    <span class="id">Figure 1.2.3.1: </span><span
    class="content">Minimizing buttons for the Materials box and the
    properties section.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-9002r2"></span>
    ![PIC](pictures/UI_screenshots/minimized.png)\
    <div class="caption">

    <span class="id">Figure 1.2.3.2: </span><span
    class="content">Minimized Materials box and properties
    section.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.4 </span> <span id="x1-100001.2.4"></span>Maximizing Materials box and Properties {.subsubsectionHead}

-   The Materials box and the properties section sizes can be adjusted
    (Figure [1.2.4.1](#x1-10001r1) and [1.2.4.2](#x1-10002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-10001r1"></span>
    ![PIC](pictures/UI_screenshots/normal_before_enlarged.png)\
    <div class="caption">

    <span class="id">Figure 1.2.4.1: </span><span
    class="content">Buttons to adjust the size of the Materials box and
    the properties section.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-10002r2"></span>
    ![PIC](pictures/UI_screenshots/enlarged.png)\
    <div class="caption">

    <span class="id">Figure 1.2.4.2: </span><span
    class="content">Enlarged Materials box and properties
    section.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.5 </span> <span id="x1-110001.2.5"></span>Re-arranging Project Tabs {.subsubsectionHead}

-   Project tabs can be moved and organized. When multiple projects are
    open, they can appear in two different views. In tile view, you can
    adjust the size of each project’s pane by moving the bar in between
    them (Figure [1.2.5.1](#x1-11001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-11001r1"></span>
    ![PIC](pictures/UI_screenshots/tabs_view1.png)\
    <div class="caption">

    <span class="id">Figure 1.2.5.1: </span><span class="content">Tile
    view with multiple projects.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Alternatively, in full view, you can have only one project visible
    at a time. Navigate to each project by clicking the tab
    (Figure [1.2.5.2](#x1-11002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-11002r2"></span>
    ![PIC](pictures/UI_screenshots/tabs_view2.png)\
    <div class="caption">

    <span class="id">Figure 1.2.5.2: </span><span class="content">Full
    view with multiple projects.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Navigate between both views by moving the tabs or by clicking the
    appropriate icon (Figure [1.2.5.4](#x1-11005r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-11005r4"></span>
    ![PIC](pictures/UI_screenshots/full_view.png) <span
    id="x1-11003r1"></span> <span class="cmr-9">The swith to full view
    icon.</span>  

    ![PIC](pictures/UI_screenshots/tile_view.png) <span
    id="x1-11004r2"></span> <span class="cmr-9">The switch to tile view
    icon.</span>\

    <div class="caption">

    <span class="id">Figure 1.2.5.4: </span><span class="content">Switch
    view icons.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.6 </span> <span id="x1-120001.2.6"></span>Viewing Projects {.subsubsectionHead}

-   You can choose between two different views to visualize projects.
    The split view shows DNA on the left pane and the circular or linear
    view on the right pane (Figure [1.2.6.1](#x1-12001r1)).
    Alternatively, the expanded view allows you to select one view which
    fills the whole screen (Figure [1.2.6.2](#x1-12002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-12001r1"></span>
    ![PIC](pictures/UI_screenshots/split_view_project.png)\
    <div class="caption">

    <span class="id">Figure 1.2.6.1: </span><span class="content">The
    split view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-12002r2"></span>
    ![PIC](pictures/UI_screenshots/expanded_view_project.png)\
    <div class="caption">

    <span class="id">Figure 1.2.6.2: </span><span class="content">The
    expanded view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Navigate between the views using the ”Expand View” and ”Split View”
    buttons (Figure [1.2.6.4](#x1-12005r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-12005r4"></span>
    ![PIC](pictures/UI_screenshots/expand_view_button.png) <span
    id="x1-12003r1"></span> <span class="cmr-9">The ”Expand View”
    button.</span>  

    ![PIC](pictures/UI_screenshots/split_view_button.png) <span
    id="x1-12004r2"></span> <span class="cmr-9">The ”Split View”
    button.</span>\

    <div class="caption">

    <span class="id">Figure 1.2.6.4: </span><span class="content">Switch
    view icons.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.7 </span> <span id="x1-130001.2.7"></span>Abstraction Layers {.subsubsectionHead}

-   There are three abstraction layers: the circular view, the linear
    view and the DNA view. You can navigate to each layer view by
    clicking its icon (Figures [1.2.7.1](#x1-13001r1)
    and [1.2.7.2](#x1-13002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-13001r1"></span>
    ![PIC](pictures/UI_screenshots/linear_view.png)\
    <div class="caption">

    <span class="id">Figure 1.2.7.1: </span><span class="content">The
    linear view and its icon.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-13002r2"></span>
    ![PIC](pictures/UI_screenshots/dna_view.png)\
    <div class="caption">

    <span class="id">Figure 1.2.7.2: </span><span class="content">The
    DNA view and its icon.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The DNA view contains an amino acid layer
    (Figure [1.2.7.3](#x1-13003r3)). Methionine is shown in green, stop
    codons are represented by a red star, and incomplete amino acids are
    marked by a red exclamation point.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-13003r3"></span>
    ![PIC](pictures/UI_screenshots/aa_layer.png)\
    <div class="caption">

    <span class="id">Figure 1.2.7.3: </span><span class="content">The
    amino acid layer in the DNA view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can show or unshow the amino acid layer by checking or
    unchecking the box in the toolbar drop down Layers menu
    (Figure [1.2.7.4](#x1-13004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-13004r4"></span>
    ![PIC](pictures/UI_screenshots/aa_layers_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.2.7.4: </span><span class="content">The
    ”Amino Acid” checkbox in the toolbar drop down Layers menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.8 </span> <span id="x1-140001.2.8"></span>Numbering of Base Pairs and Amino Acids {.subsubsectionHead}

-   Base pairs and amino acids are numbered in the DNA view
    (Figure [1.2.8.1](#x1-14001r1)). After each thousand, counting
    starts again from 0 and a label indicates the current thousand
    (Figure [1.2.8.2](#x1-14002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-14001r1"></span>
    ![PIC](pictures/UI_screenshots/start.png)\
    <div class="caption">

    <span class="id">Figure 1.2.8.1: </span><span
    class="content">Numbered base pairs and amino acids in the DNA
    view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   After each thousand, counting starts again for base pairs from 0 and
    a label indicates the current thousand
    (Figure [1.2.8.2](#x1-14002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-14002r2"></span>
    ![PIC](pictures/UI_screenshots/1000.png)\
    <div class="caption">

    <span class="id">Figure 1.2.8.2: </span><span
    class="content">Numbered base pairs in the DNA view, after a
    thousand.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The number of the first base pair of each part is indicated in the
    linear view (Figure [1.2.8.3](#x1-14003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-14003r3"></span>
    ![PIC](pictures/UI_screenshots/number_part.png)\
    <div class="caption">

    <span class="id">Figure 1.2.8.3: </span><span class="content">The
    number of the first base pair of each part in the linear
    view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.9 </span> <span id="x1-150001.2.9"></span>Properties Section {.subsubsectionHead}

-   The properties section at the bottom of the screen contains
    different tabs providing information on your project
    (Figure [1.2.9.1](#x1-15001r1)), parts
    (Figure [1.2.9.2](#x1-15002r2)) (the tab is labeled ”Multiple” when
    multiple parts are selected), selection
    (Figure [1.2.9.5](#x1-15005r5)), compilation
    (Figure [1.2.9.3](#x1-15003r3)) and restriction digests
    (Figure [1.2.9.4](#x1-15004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-15001r1"></span>
    ![PIC](pictures/UI_screenshots/properties_section_description.png)\
    <div class="caption">

    <span class="id">Figure 1.2.9.1: </span><span class="content">The
    project tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-15002r2"></span>
    ![PIC](pictures/UI_screenshots/part_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.2.9.2: </span><span class="content">The
    part tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-15003r3"></span>
    ![PIC](pictures/UI_screenshots/compilation_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.2.9.3: </span><span class="content">The
    compilation tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-15004r4"></span>
    ![PIC](pictures/UI_screenshots/digest_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.2.9.4: </span><span class="content">The
    digest tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   When you select a strand of DNA in the DNA view, the selection tab
    (Figure [1.2.9.5](#x1-15005r5)) displays:

    -   the selection count
    -   the GC content
    -   the melting temperature (calculated according to [<span
        class="cmtt-10">SantaLucia
        1998</span>](http://www.pnas.org/content/95/4/1460.full))
    -   the start of the selection
    -   the end of the selection
    -   the number of parts contained in the selection
    -   the isolelectric point
    -   the hydrophobicity
    -   the molecular weight

    as well as the name, linear formula and type of the amino acid, when
    a single amino acid is selected.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-15005r5"></span>
    ![PIC](pictures/UI_screenshots/selection_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.2.9.5: </span><span class="content">The
    selection tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.2.10 </span> <span id="x1-160001.2.10"></span>Options Menu {.subsubsectionHead}

-   The ”Options” drop down menu at the top right
    (Figure [1.2.10.1](#x1-16001r1)) allows you to find information
    about the software, to give feedback and to access your preferences.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-16001r1"></span>
    ![PIC](pictures/UI_screenshots/options_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.2.10.1: </span><span class="content">The
    ”Options” drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   By default, projects left open in previous sessions are reopened in
    new sessions. This can be changed by selecting ”No” in the
    ”Preferences” dialog (Figure [1.2.10.2](#x1-16002r2)). You can also
    reset all dialogs by clicking ”Reset all dialogs”.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-16002r2"></span>
    ![PIC](pictures/UI_screenshots/preferences.png)\
    <div class="caption">

    <span class="id">Figure 1.2.10.2: </span><span class="content">The
    ”Preferences” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.3 </span> <span id="x1-170001.3"></span>Importing and Exporting {.subsectionHead}

Genome Compiler supports import of Genome Compiler (.gcproj), INSD
(.xml), GenBank (.gb), FASTA (.fasta, .fas, .fa), Raw DNA (.txt),
Geneious (.geneious) and Vector NTI archive (.ma4) file formats. Genome
Compiler supports the export of Genome Compiler, GenBank, FASTA and Raw
DNA file formats.

##### <span class="titlemark">1.3.1 </span> <span id="x1-180001.3.1"></span>Importing Files {.subsubsectionHead}

-   To import a file, click the “Import File” icon in the Materials box
    (Figure [1.3.1.1](#x1-18001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-18001r1"></span>
    ![PIC](pictures/importexport_screenshots/materials_import_icon.png)\
    <div class="caption">

    <span class="id">Figure 1.3.1.1: </span><span class="content">The
    ”Import File” icon in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   A dialog will pop up (Figure [1.3.1.2](#x1-18002r2)) to allow you to
    choose the file from your computer files.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-18002r2"></span>
    ![PIC](pictures/importexport_screenshots/computer_files.png)\
    <div class="caption">

    <span class="id">Figure 1.3.1.2: </span><span
    class="content">Choosing a file from your computer files.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Once you selected a file, a dialog will pop up
    (Figure [1.3.1.3](#x1-18003r3)) to prompt you for a location. You
    can choose to save the file in an existing folder or to create a new
    folder.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-18003r3"></span>
    ![PIC](pictures/importexport_screenshots/save_cloud_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.3.1.3: </span><span class="content">The
    ”Save on Cloud” dialog when importing a file.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To create a new folder, click “New Folder”. A dialog will pop up
    (Figure [1.3.1.4](#x1-18004r4)) to prompt you for a name and an
    optional folder description. Save your file by clicking “Save”. The
    file will now be saved as a new project in Genome Compiler and can
    be opened from the Materials box.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-18004r4"></span>
    ![PIC](pictures/importexport_screenshots/new_folder_file.png)\
    <div class="caption">

    <span class="id">Figure 1.3.1.4: </span><span class="content">The
    ”Save on Cloud” dialog when creating a new folder.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.3.2 </span> <span id="x1-190001.3.2"></span>Importing Parts {.subsubsectionHead}

-   To import a part, select “Parts” from the import drop down menu
    (Figure [1.3.2.1](#x1-19001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-19001r1"></span>
    ![PIC](pictures/importexport_screenshots/materials_import_parts.png)\
    <div class="caption">

    <span class="id">Figure 1.3.2.1: </span><span class="content">The
    import drop down menu in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The “Import Genetic Parts” dialog will appear
    (Figure [1.3.2.2](#x1-19002r2)). The “Create New Part” icon allows
    you to add a new part to the list. Select the appropriate feature
    from the “Feature Key” drop down menu and enter the part name and
    sequence. Click the “Delete” icon to delete a part.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-19002r2"></span>
    ![PIC](pictures/importexport_screenshots/import_genetic_parts_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.3.2.2: </span><span class="content">The
    ”Import Genetic Parts” dialog and its ”Feature Key” drop down
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Import the listed parts by clicking “Import”. The flow for saving
    parts is the same for saving files. Just like imported files, the
    imported part is saved as a new project in the Materials box.

##### <span class="titlemark">1.3.3 </span> <span id="x1-200001.3.3"></span>Exporting Files {.subsubsectionHead}

-   To export a file, select it and select the appropriate file format
    from the “Export” drop down menu in the Materials box
    (Figures [1.3.3.1](#x1-20001r1)) or in the toolbar
    (Figures [1.3.3.2](#x1-20002r2)). A dialog will pop up to prompt you
    for a name and a location.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-20001r1"></span>
    ![PIC](pictures/importexport_screenshots/export_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.3.3.1: </span><span class="content">The
    “Export” drop down menu in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-20002r2"></span>
    ![PIC](pictures/importexport_screenshots/export_toolbar.png)\
    <div class="caption">

    <span class="id">Figure 1.3.3.2: </span><span class="content">The
    “Export” drop down menu in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.4 </span> <span id="x1-210001.4"></span>Editing {.subsectionHead}

All the standard editing tools are available in the software. You can
copy, cut, paste, delete, drag, drop, undo and redo, as well as perform
reverse complements, create, merge and split parts or manage DNA
strands.

##### <span class="titlemark">1.4.1 </span> <span id="x1-220001.4.1"></span>Basic Copy/Cut/Paste/Delete/Undo/Redo {.subsubsectionHead}

-   You can copy and paste, cut and paste or delete parts, amino acids
    or base pairs within a project or in between different projects.
    Right click on the sequence and select from the drop down menu
    (Figure [1.4.1.1](#x1-22001r1)), or use the icons in the toolbar
    (Figure [1.4.1.2](#x1-22002r2)) or the standard keyboard shortcuts.
    You can paste strands of DNA from external sources.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-22001r1"></span>
    ![PIC](pictures/editing_screenshots/dropdown_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.4.1.1: </span><span class="content">Cut,
    copy, paste and delete in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-22002r2"></span>
    ![PIC](pictures/editing_screenshots/toolbar_copy_etc.png)\
    <div class="caption">

    <span class="id">Figure 1.4.1.2: </span><span class="content">Cut,
    copy, paste and delete icons in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can undo or redo changes via the icons in the toolbar
    (Figure [1.4.1.3](#x1-22003r3)) or the standard keyboard shortcuts.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-22003r3"></span>
    ![PIC](pictures/editing_screenshots/undo_redo.png)\
    <div class="caption">

    <span class="id">Figure 1.4.1.3: </span><span class="content">Undo
    and redo icons in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.2 </span> <span id="x1-230001.4.2"></span>Drag and Drop Functionality {.subsubsectionHead}

-   In addition to standard copy, cut, paste, delete, Genome Compiler
    also has drag and drop functionality.
-   You can drag and drop projects from the Materials box to open
    projects, or from open projects to other open projects.
-   You can also drag and drop parts, amino acids or base pairs within a
    project and between projects.
-   You can create a new project by dragging a selection to the toolbar
    (Figures [1.4.2.1](#x1-23001r1) and [1.4.2.2](#x1-23002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-23001r1"></span>
    ![PIC](pictures/editing_screenshots/drag.png)\
    <div class="caption">

    <span class="id">Figure 1.4.2.1: </span><span class="content">Drag
    and drop a selection to the toolbar to create a new project.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-23002r2"></span>
    ![PIC](pictures/editing_screenshots/drop.png)\
    <div class="caption">

    <span class="id">Figure 1.4.2.2: </span><span class="content">New
    project created from a dropped selection.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.3 </span> <span id="x1-240001.4.3"></span>Adding New Parts {.subsubsectionHead}

-   You can add new parts to your projects. Right click where you would
    like your new part to be and select “New part” from the drop down
    menu (Figure [1.4.3.1](#x1-24001r1)), or click the “Create New Part”
    icon in the toolbar (Figure [1.4.3.2](#x1-24002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-24001r1"></span>
    ![PIC](pictures/editing_screenshots/new_part_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.4.3.1: </span><span class="content">”New
    part” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-24002r2"></span>
    ![PIC](pictures/editing_screenshots/new_part_icon.png)\
    <div class="caption">

    <span class="id">Figure 1.4.3.2: </span><span class="content">The
    ”New part” icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   This will create an empty part (Figure [1.4.3.3](#x1-24003r3)) which
    can be edited in the DNA view (Figure [1.4.3.4](#x1-24004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-24003r3"></span>
    ![PIC](pictures/editing_screenshots/empty_part_parts_view.png)\
    <div class="caption">

    <span class="id">Figure 1.4.3.3: </span><span class="content">New
    empty part in the linear view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-24004r4"></span>
    ![PIC](pictures/editing_screenshots/empty_part_dna_view.png)\
    <div class="caption">

    <span class="id">Figure 1.4.3.4: </span><span class="content">New
    empty part in the DNA view.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.4 </span> <span id="x1-250001.4.4"></span>Adding Amino Acids {.subsubsectionHead}

-   You can add amino acids in your projects. Right click on the amino
    acid after which you would like the amino acids to be added and
    select “Insert Amino Acids...” from the drop down menu
    (Figure [1.4.4.1](#x1-25001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-25001r1"></span>
    ![PIC](pictures/editing_screenshots/insert_aa_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.4.4.1: </span><span
    class="content">”Insert Amino Acids” in the right click drop down
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Insert Amino Acid” dialog will appear. Type in the amino acids
    (use the 1 letter representation).
-   The ”Translation Settings” dialog will appear. You can choose to
    translate according to codon usage or to a translation table. For
    more details about translation see section [1.14](#x1-680001.14).

##### <span class="titlemark">1.4.5 </span> <span id="x1-260001.4.5"></span>Reverse Complement {.subsubsectionHead}

-   To perform a reverse complement, right click on the part and select
    “Reverse complement” from the drop down menu
    (Figure [1.4.5.1](#x1-26001r1)), or click the “Reverse Complement”
    icon in the toolbar (Figure [1.4.5.2](#x1-26002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-26001r1"></span>
    ![PIC](pictures/editing_screenshots/reverse_compl_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.4.5.1: </span><span
    class="content">”Reverse complement” in the right click drop down
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-26002r2"></span>
    ![PIC](pictures/editing_screenshots/reverse_compl_toolbar.png)\
    <div class="caption">

    <span class="id">Figure 1.4.5.2: </span><span class="content">The
    ”Reverse Complement” icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.6 </span> <span id="x1-270001.4.6"></span>Merging and Splitting Parts {.subsubsectionHead}

-   You can merge and split parts. To merge parts, select them and
    select “Merge” in the toolbar (Figure [1.4.6.1](#x1-27001r1)), or
    right click and select “Merge selected parts” from the drop down
    menu (Figure [1.4.6.2](#x1-27002r2)). A new merged part containing
    the selection will be created.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-27001r1"></span>
    ![PIC](pictures/editing_screenshots/merge.png)\
    <div class="caption">

    <span class="id">Figure 1.4.6.1: </span><span class="content">The
    ”Merge” icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-27002r2"></span>
    ![PIC](pictures/editing_screenshots/merge_right_click.png)\
    <div class="caption">

    <span class="id">Figure 1.4.6.2: </span><span class="content">”Merge
    selected parts” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To split a sequence, switch to DNA view and place the caret where
    you would like to split the sequence. Select “Split” in the toolbar
    (Figure [1.4.6.3](#x1-27003r3)), or right click and select “Split
    part” from the dropdown menu (Figure [1.4.6.4](#x1-27004r4)). New
    split parts will be created.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-27003r3"></span>
    ![PIC](pictures/editing_screenshots/split.png)\
    <div class="caption">

    <span class="id">Figure 1.4.6.3: </span><span class="content">The
    ”Split” icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-27004r4"></span>
    ![PIC](pictures/editing_screenshots/split_right_click.png)\
    <div class="caption">

    <span class="id">Figure 1.4.6.4: </span><span class="content">”Split
    part” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.7 </span> <span id="x1-280001.4.7"></span>Selecting Strand {.subsubsectionHead}

-   You can select DNA strands using the ”Minus strand” and ”Plus
    strand” icons in the ”Part” tab (Figure [1.4.7.1](#x1-28001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-28001r1"></span>
    ![PIC](pictures/editing_screenshots/select_strand.png)\
    <div class="caption">

    <span class="id">Figure 1.4.7.1: </span><span class="content">The
    ”Minus strand” and ”Plus strand” icons in the ”Part” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The selected strand will be indicated by a darker color, as well as
    identified by an arrow (Figure [1.4.7.3](#x1-28004r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-28004r3"></span>
    ![PIC](pictures/editing_screenshots/plus_strand.png) <span
    id="x1-28002r1"></span> <span class="cmr-9">Plus strand
    selected.</span>  

    ![PIC](pictures/editing_screenshots/minus_strand.png) <span
    id="x1-28003r2"></span> <span class="cmr-9">Minus strand
    selected.</span>\

    <div class="caption">

    <span class="id">Figure 1.4.7.3: </span><span class="content">DNA
    strands visualization.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can show or unshow the minus strand by checking or unchecking
    the box in the toolbar drop down Layers menu
    (Figure [1.4.7.4](#x1-28005r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-28005r4"></span>
    ![PIC](pictures/editing_screenshots/minus_strand_layers_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.4.7.4: </span><span class="content">The
    ”Minus Strand” checkbox in the toolbar drop down Layers menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.8 </span> <span id="x1-290001.4.8"></span>Mutation Wheel {.subsubsectionHead}

-   The mutation wheel allows you to mutate amino acids. To access the
    mutation wheel, click the drop down arrow below the amino acid and
    select ”Mutate...” (Figure [1.4.8.1](#x1-29001r1)) or right click on
    the amino acid and select ”Mutate...” from the drop down menu
    (Figure [1.4.8.2](#x1-29002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-29001r1"></span>
    ![PIC](pictures/editing_screenshots/access_mutation_wheel.png)\
    <div class="caption">

    <span class="id">Figure 1.4.8.1: </span><span
    class="content">”Mutate...” in the drop down arrow menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-29002r2"></span>
    ![PIC](pictures/editing_screenshots/access_mutation_wheel_right_click.png)\
    <div class="caption">

    <span class="id">Figure 1.4.8.2: </span><span
    class="content">”Mutate” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Select amino acids from the mutation wheel
    (Figure [1.4.8.3](#x1-29003r3)) and click the green tick. You can
    filter selection of amino acids with a specific property by checking
    the property checkbox on the left.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-29003r3"></span>
    ![PIC](pictures/editing_screenshots/mutation_wheel.png)\
    <div class="caption">

    <span class="id">Figure 1.4.8.3: </span><span class="content">The
    amino acids mutation wheel.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.4.9 </span> <span id="x1-300001.4.9"></span>Basic Local Alignment Search Tool (BLAST) {.subsubsectionHead}

-   You can send sequences to Basic Local Alignment Search Tool (BLAST)
    directly from the software. Select a sequence in the DNA view or a
    part in the linear view, right click and select ”Send to BLAST” from
    the drop down menu (Figure [1.4.9.1](#x1-30001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-30001r1"></span>
    ![PIC](pictures/editing_screenshots/send_to_blast.png)\
    <div class="caption">

    <span class="id">Figure 1.4.9.1: </span><span class="content">”Send
    to BLAST” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   This will open the BLAST website ([<span
    class="cmtt-10">blast.ncbi.nlm.nih.gov</span>](http://blast.ncbi.nlm.nih.gov))
    with the selected sequence pasted.

#### <span class="titlemark">1.5 </span> <span id="x1-310001.5"></span>Go to {.subsectionHead}

The go to feature allows you to navigate easily through projects.

##### <span class="titlemark">1.5.1 </span> <span id="x1-320001.5.1"></span>Openging the ”Go to” Dialog {.subsubsectionHead}

To open the “Go to” dialog, right click on your project canvas in the
protein or DNA view and select “Go to…” or “Select range…”
(Figure [1.5.1.2](#x1-32003r2)), or use the keyboard shortcut ”control
G”.

------------------------------------------------------------------------

<div class="figure">

<span id="x1-32003r2"></span>
![PIC](pictures/goto_screenshots/goto_dropdown.png) <span
id="x1-32001r1"></span> <span class="cmr-9">”Go to...”</span>  

![PIC](pictures/goto_screenshots/select_range.png) <span
id="x1-32002r2"></span> <span class="cmr-9">”Select range...”</span>\

<div class="caption">

<span class="id">Figure 1.5.1.2: </span><span class="content">”Go to...”
and ”Select range...” in the right click drop down menu.</span>

</div>

-   

    ------------------------------------------------------------------------

##### <span class="titlemark">1.5.2 </span> <span id="x1-330001.5.2"></span>”Go to” Dialog {.subsubsectionHead}

-   The “Go to” dialog will pop up (Figure [1.5.2.1](#x1-33001r1)) and
    prompt you for a nucleotide number. You can enter a single number or
    a range for selection. Clicking “Go” will take you to the sequence.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-33001r1"></span>
    ![PIC](pictures/goto_screenshots/goto_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.5.2.1: </span><span class="content">The
    ”Go to” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.6 </span> <span id="x1-340001.6"></span>Search in the Materials box {.subsectionHead}

The search bar allows you to search through your projects in the
Materials box. The search is performed not only based on the projects
names, but also according to the projects meta data.

##### <span class="titlemark">1.6.1 </span> <span id="x1-350001.6.1"></span>Searching in the Materials box {.subsubsectionHead}

Type a keyword in the search bar. Results will appear below
(Figure [1.6.1.1](#x1-35001r1)).

------------------------------------------------------------------------

<div class="figure">

<span id="x1-35001r1"></span>
![PIC](pictures/search_materialsbox_screenshots/search_name.png)\
<div class="caption">

<span class="id">Figure 1.6.1.1: </span><span class="content">Search for
a keyword in the Materials box.</span>

</div>

-   

    ------------------------------------------------------------------------

-   The search tool will look for your keyword in everything contained
    in the project meta data, including its name, comments, etc. Meta
    data for each project can be found in the project tab
    (Figure [1.6.1.2](#x1-35002r2)) located along the bottom of each
    opened project or in a tooltip while hovering over the closed
    project in the Materials box (Figure [1.6.1.3](#x1-35003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-35002r2"></span>
    ![PIC](pictures/search_materialsbox_screenshots/project_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.6.1.2: </span><span class="content">The
    project tab showing the project meta data.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-35003r3"></span>
    ![PIC](pictures/search_materialsbox_screenshots/tooltip.png)\
    <div class="caption">

    <span class="id">Figure 1.6.1.3: </span><span
    class="content">Tooltip showing the project meta data.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.6.2 </span> <span id="x1-360001.6.2"></span>Example {.subsubsectionHead}

-   For example, you can find the genome of this strand of E.coli by
    looking for the GI number that appears in the comment section
    (Figure [1.6.2.1](#x1-36001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-36001r1"></span>
    ![PIC](pictures/search_materialsbox_screenshots/search_comments.png)\
    <div class="caption">

    <span class="id">Figure 1.6.2.1: </span><span
    class="content">Example: search for the GI number.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.7 </span> <span id="x1-370001.7"></span>Search in Projects {.subsectionHead}

The search bar allows you to search for DNA sequences and for part meta
data within your projects.

##### <span class="titlemark">1.7.1 </span> <span id="x1-380001.7.1"></span>Searching in Projects {.subsubsectionHead}

-   Type a keyword or a DNA sequence in the search bar
    (Figure [1.7.1.1](#x1-38001r1)) in the top right of the screen and
    click enter. The arrows allow you to navigate through the results.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-38001r1"></span>
    ![PIC](pictures/search_project_screenshots/search_bar.png)\
    <div class="caption">

    <span class="id">Figure 1.7.1.1: </span><span class="content">The
    search bar in the top right of the screen.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The search tool will look for your keyword in everything contained
    in the part meta data, including its name, comments, etc. Meta data
    for each part can be found in the part tab
    (Figure [1.7.1.2](#x1-38002r2)) located along the bottom of each
    opened project or in a tooltip while hovering over the part
    (Figure [1.7.1.3](#x1-38003r3)). For example in this case, the
    keyword ”transposase” was found in the “Product” field.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-38002r2"></span>
    ![PIC](pictures/search_project_screenshots/part_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.7.1.2: </span><span class="content">Part
    tab showing the part meta data.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-38003r3"></span>
    ![PIC](pictures/search_project_screenshots/tooltip.png)\
    <div class="caption">

    <span class="id">Figure 1.7.1.3: </span><span
    class="content">Tooltip showing the part meta data.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   DNA sequences are searched for on both strands
    (Figure [1.7.1.4](#x1-38004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-38004r4"></span>
    ![PIC](pictures/search_project_screenshots/search_dna.png)\
    <div class="caption">

    <span class="id">Figure 1.7.1.4: </span><span class="content">Search
    for DNA sequences using the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.8 </span> <span id="x1-390001.8"></span>NCBI Search {.subsectionHead}

The NCBI search tool allows you to search through the NCBI database
directly from our software.

##### <span class="titlemark">1.8.1 </span> <span id="x1-400001.8.1"></span>Searching in the NCBI Database {.subsubsectionHead}

-   Type a keyword in the search bar and click the NCBI icon
    (Figure [1.8.1.1](#x1-40001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-40001r1"></span>
    ![PIC](pictures/NCBI_screenshots/searchbox.png)\
    <div class="caption">

    <span class="id">Figure 1.8.1.1: </span><span class="content">The
    NCBI icon and the search bar in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   NCBI search results will appear (Figure [1.8.1.2](#x1-40002r2)). The
    icon on the right of each result takes you to the NCBI website. To
    download the file, click the green “import” icon. To close the NCBI
    search results, click the NCBI icon.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-40002r2"></span>
    ![PIC](pictures/NCBI_screenshots/NCBI_results.png)\
    <div class="caption">

    <span class="id">Figure 1.8.1.2: </span><span class="content">NCBI
    search results, with - from left to right - the NCBI icon, the green
    ”import” icon and the ”See item in the NCBI website” icon.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The downloaded file will be automatically added to the cloud in the
    Materials box (Figure [1.8.1.3](#x1-40003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-40003r3"></span>
    ![PIC](pictures/NCBI_screenshots/sequence_cloud.png)\
    <div class="caption">

    <span class="id">Figure 1.8.1.3: </span><span class="content">The
    downloaded file in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   If you have already found what you are looking for in the NCBI
    database, you can copy the name or the GI number, and paste it in
    the search bar (Figure [1.8.1.1](#x1-40001r1)). Click the NCBI icon,
    and download the file.

##### <span class="titlemark">1.8.2 </span> <span id="x1-410001.8.2"></span>Downloaded and Downloading Files Window {.subsubsectionHead}

-   Downloaded and downloading files appear in a window at the bottom
    (Figure [1.8.2.1](#x1-41001r1)). You can remove files from the list
    by clicking the “remove” icon. The “view in NCBI” icon takes you to
    the corresponding page on the NCBI website. Click the “view in the
    Materials box” icon on the left of the downloaded project to view
    the file in the Materials box.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-41001r1"></span>
    ![PIC](pictures/NCBI_screenshots/whole_screen_import.png)\
    <div class="caption">

    <span class="id">Figure 1.8.2.1: </span><span class="content">The
    downloaded and downloading files window.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.9 </span> <span id="x1-420001.9"></span>Sharing and Collaboration {.subsectionHead}

The Cloud allows users to access projects from anywhere and share them
with others.

##### <span class="titlemark">1.9.1 </span> <span id="x1-430001.9.1"></span>Cloud Folders {.subsubsectionHead}

-   Folders saved in the cloud appear in the Materials box and are
    marked with a cloud icon. Refresh the cloud by clicking the “Refresh
    Cloud” icon (Figure [1.9.1.1](#x1-43001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-43001r1"></span>
    ![PIC](pictures/cloud_screenshots/materials_box_cloud.png)\
    <div class="caption">

    <span class="id">Figure 1.9.1.1: </span><span class="content">The
    ”Refresh Cloud” icon in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.9.2 </span> <span id="x1-440001.9.2"></span>Creating New Folders {.subsubsectionHead}

-   To create a new folder, right click and select “New Cloud Folder”
    (Figure [1.9.2.1](#x1-44001r1)). You can also create new folders via
    the ”Cloud Folder Settings” dialog as explained in
    section [1.9.5](#x1-470001.9.5).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-44001r1"></span>
    ![PIC](pictures/cloud_screenshots/right_click_cloud.png)\
    <div class="caption">

    <span class="id">Figure 1.9.2.1: </span><span class="content">”New
    Cloud Folder” in the right click menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.9.3 </span> <span id="x1-450001.9.3"></span>Creating Subfolders {.subsubsectionHead}

-   You can also create subfolders. Click the “New Folder” icon in the
    Materials box, or right click and select “New Folder”
    (Figure [1.9.3.2](#x1-45003r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-45003r2"></span>
    ![PIC](pictures/cloud_screenshots/new_folder_icon.png) <span
    id="x1-45001r1"></span> <span class="cmr-9">The ”New Folder” icon in
    the Materials</span> <span class="cmr-9">box.</span>  

    ![PIC](pictures/cloud_screenshots/right_click_new_folder.png) <span
    id="x1-45002r2"></span> <span class="cmr-9">”New Folder” in the
    right click drop down</span> <span class="cmr-9">menu.</span>\

    <div class="caption">

    <span class="id">Figure 1.9.3.2: </span><span class="content">Create
    Cloud subfolders in the Materials box.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.9.4 </span> <span id="x1-460001.9.4"></span>Sharing Cloud Folders {.subsubsectionHead}

-   Folders can be shared via the “Cloud Folder Settings” dialog. It can
    be accessed by clicking the “Cloud Settings” icon in the Materials
    box or in the toolbar (Figure [1.9.4.2](#x1-46003r2)). In this
    dialog select the folder from the drop down menu which you would
    like to share.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-46003r2"></span>
    ![PIC](pictures/cloud_screenshots/cloud_settings_icon.png) <span
    id="x1-46001r1"></span> <span class="cmr-9">The ”Cloud Settings”
    icon in the Materials</span> <span class="cmr-9">box.</span>  

    ![PIC](pictures/cloud_screenshots/toolbar_cloud.png) <span
    id="x1-46002r2"></span> <span class="cmr-9">The ”Cloud Settings”
    icon in the toolbar.</span>\

    <div class="caption">

    <span class="id">Figure 1.9.4.2: </span><span class="content">The
    ”Cloud Settings” icon.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can also share a folder directly by right clicking on a folder
    and selecting “Settings” (Figure [1.9.4.3](#x1-46004r3)). Please
    note that you cannot share subfolders.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-46004r3"></span>
    ![PIC](pictures/cloud_screenshots/right_click_settings.png)\
    <div class="caption">

    <span class="id">Figure 1.9.4.3: </span><span
    class="content">”Settings” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can share folders with other users. Select which folder you
    would like to share from the drop down menu
    (Figure [1.9.4.4](#x1-46005r4)). Enter their email address in the
    ”Share with” field and click “Add”. They will receive an email
    telling them that you would like to share a folder.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-46005r4"></span>
    ![PIC](pictures/cloud_screenshots/cloud_folder_settings_choosefolder.png)\
    <div class="caption">

    <span class="id">Figure 1.9.4.4: </span><span class="content">The
    ”Cloud Folder Settings” dialog: choosing a folder.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To manage permissions, select “Read/Write” or “Read” from the drop
    down menu (Figure [1.9.4.5](#x1-46006r5)). To remove users from the
    list, click “Remove”.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-46006r5"></span>
    ![PIC](pictures/cloud_screenshots/cloud_folder_settings.png)\
    <div class="caption">

    <span class="id">Figure 1.9.4.5: </span><span class="content">The
    ”Cloud Folder Settings” dialog: managing permissions.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.9.5 </span> <span id="x1-470001.9.5"></span>Creating New Folders via the ”Cloud Folder Settings” dialog {.subsubsectionHead}

-   You can also create new folders from the “Cloud Folder Settings”
    dialog (Figure [1.9.5.1](#x1-47001r1)) by clicking on the “New
    Folder” icon. Enter the new folder’s name and an optional folder
    description.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-47001r1"></span>
    ![PIC](pictures/cloud_screenshots/cloud_folder_settings_new_folder.png)\
    <div class="caption">

    <span class="id">Figure 1.9.5.1: </span><span class="content">The
    ”New Folder” icon in the ”Cloud Folder Settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.10 </span> <span id="x1-480001.10"></span>User Comments {.subsectionHead}

The user comments feature allows you to make comments on all abstraction
layers, view comments from multiple users over the cloud manipulating a
single sequence, and to view a summary table of all comments in each
project.

##### <span class="titlemark">1.10.1 </span> <span id="x1-490001.10.1"></span>User Comments Indicator {.subsubsectionHead}

-   You can make comments on all abstraction layers. Comments are
    indicated by yellow flags (Figure [1.10.1.1](#x1-49001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-49001r1"></span>
    ![PIC](pictures/comments_screenshots/yellow_flag.png)\
    <div class="caption">

    <span class="id">Figure 1.10.1.1: </span><span
    class="content">Yellow flags indicate comments.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.10.2 </span> <span id="x1-500001.10.2"></span>Navigating to Comments {.subsubsectionHead}

-   All project comments are also tracked as small indications on the
    scroll bar on the right (Figure [1.10.2.1](#x1-50001r1)). You can
    navigate to the comments by clicking on these indications.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-50001r1"></span>
    ![PIC](pictures/comments_screenshots/scrollbar_comments.png)\
    <div class="caption">

    <span class="id">Figure 1.10.2.1: </span><span class="content">The
    scroll bar allows navigation to the comments.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.10.3 </span> <span id="x1-510001.10.3"></span>Adding Comments {.subsubsectionHead}

-   To comment a whole part/gene, right click and select “Add comment”
    from the drop down menu (Figure [1.10.3.1](#x1-51001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-51001r1"></span>
    ![PIC](pictures/comments_screenshots/add_comment_part.png)\
    <div class="caption">

    <span class="id">Figure 1.10.3.1: </span><span class="content">”Add
    comment” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Add your comment in the comment tool tip
    (Figure [1.10.3.2](#x1-51002r2)). You can include direct links. To
    navigate to links, click on them while holding the control key. You
    can delete comments by clicking the “Delete” icon.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-51002r2"></span>
    ![PIC](pictures/comments_screenshots/comment_part.png)\
    <div class="caption">

    <span class="id">Figure 1.10.3.2: </span><span class="content">The
    comment tool tip: including a direct link.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To comment a stretch of nucleotides, right click on them and select
    “Add comment” from the drop down menu
    (Figure [1.10.3.3](#x1-51003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-51003r3"></span>
    ![PIC](pictures/comments_screenshots/add_comment_dna.png)\
    <div class="caption">

    <span class="id">Figure 1.10.3.3: </span><span class="content">”Add
    comment” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Add your comment in the tool tip (Figure [1.10.3.4](#x1-51004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-51004r4"></span>
    ![PIC](pictures/comments_screenshots/comment_dna.png)\
    <div class="caption">

    <span class="id">Figure 1.10.3.4: </span><span class="content">The
    comment tool tip.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.10.4 </span> <span id="x1-520001.10.4"></span>Multiple Users making Comments in a Single Project {.subsubsectionHead}

-   Multiple users can comment the same sequence
    (Figure [1.10.4.1](#x1-52001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-52001r1"></span>
    ![PIC](pictures/comments_screenshots/2_users_comment.png)\
    <div class="caption">

    <span class="id">Figure 1.10.4.1: </span><span
    class="content">Comments from multiple users on a single
    sequence.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.10.5 </span> <span id="x1-530001.10.5"></span>User Comments Summary Table {.subsubsectionHead}

-   To view a summary table of all comments, click the “Summary table”
    icon in the toolbar drop down Layers menu
    (Figure [1.10.5.1](#x1-53001r1)). You can remove the comments from
    the views by unchecking the box. However, you will still be able to
    add comments.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-53001r1"></span>
    ![PIC](pictures/comments_screenshots/summary_table_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.10.5.1: </span><span class="content">The
    comments ”Summary table” icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The “User Comments Summary” table (Figure [1.10.5.2](#x1-53002r2))
    displays all the comments, as well as their location, author and the
    time and date that they were made. You can navigate to each comment
    in the sequence by clicking on it. Click the “Show comments and
    changes” box to include in this list the track changes that were
    also made in this project.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-53002r2"></span>
    ![PIC](pictures/comments_screenshots/user_comments_summary.png)\
    <div class="caption">

    <span class="id">Figure 1.10.5.2: </span><span class="content">The
    ”User Comments Summary” table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.11 </span> <span id="x1-540001.11"></span>Track Changes {.subsectionHead}

The track changes feature allows you to track changes in real time, to
see changes made to projects by multiple users over the cloud and to
view a summary table of all changes made to each project. Changes
include additions, deletions, substitution mutations, combinatorial
changes, etc.

##### <span class="titlemark">1.11.1 </span> <span id="x1-550001.11.1"></span>Track Changes Indicator {.subsubsectionHead}

-   DNA and amino acid changes are tracked and indicated in the form of
    a blue flag on the sequence (Figure [1.11.1.1](#x1-55001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-55001r1"></span>
    ![PIC](pictures/track_changes_screenshots/blue_flag.png)\
    <div class="caption">

    <span class="id">Figure 1.11.1.1: </span><span class="content">Blue
    flags indicate changes.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.11.2 </span> <span id="x1-560001.11.2"></span>Navigating to Changes {.subsubsectionHead}

-   All project changes are also tracked as small indications on the
    scroll bar on the right (Figure [1.11.2.1](#x1-56001r1)). You can
    navigate to the tracked changes by clicking on these indications.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-56001r1"></span>
    ![PIC](pictures/track_changes_screenshots/changes_scrollbar.png)\
    <div class="caption">

    <span class="id">Figure 1.11.2.1: </span><span class="content">The
    scroll bar allows navigation to the changes.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.11.3 </span> <span id="x1-570001.11.3"></span>Multiple Users making Changes in a Single Project {.subsubsectionHead}

-   Multiple users can make changes on the same sequence
    (Figure [1.11.3.1](#x1-57001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-57001r1"></span>
    ![PIC](pictures/track_changes_screenshots/2_users_changes.png)\
    <div class="caption">

    <span class="id">Figure 1.11.3.1: </span><span
    class="content">Changes from multiple users on a single
    sequence.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.11.4 </span> <span id="x1-580001.11.4"></span>Track Changes Summary Table {.subsubsectionHead}

-   To view a summary table of all changes, click the track changes
    “Summary table” icon in the toolbar drop down Layers menu
    (Figure [1.11.4.1](#x1-58001r1)). You can remove the changes from
    the views by unchecking the box.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-58001r1"></span>
    ![PIC](pictures/track_changes_screenshots/track_changes_dropdown_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.11.4.1: </span><span class="content">The
    track changes ”Summary table” icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The “Track Changes Summary” table (Figure [1.11.4.2](#x1-58002r2))
    displays all the changes in the project, as well as their location,
    author, the time and date that they were made and prospective
    comments. You can navigate to each change in the DNA sequence by
    clicking on it. Click the “Show comments and changes” box to include
    in this list any user comments which were also made in this project.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-58002r2"></span>
    ![PIC](pictures/track_changes_screenshots/track_changes_summary.png)\
    <div class="caption">

    <span class="id">Figure 1.11.4.2: </span><span class="content">The
    “Track Changes Summary” table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.12 </span> <span id="x1-590001.12"></span>Ordering and Pricing {.subsectionHead}

Genome Compiler allows you to request quotes and order DNA directly from
the software! Pricing is available for projects as well as for primers.

##### <span class="titlemark">1.12.1 </span> <span id="x1-600001.12.1"></span>Pricing for Projects {.subsubsectionHead}

-   To request a price quote for synthesising a whole project, go to the
    ”Pricing” icon in the toolbar and click ”Project...”
    (Figure [1.12.1.1](#x1-60001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-60001r1"></span>
    ![PIC](pictures/pricing_ordering_screenshots/pricing_select_project.png)\
    <div class="caption">

    <span class="id">Figure 1.12.1.1: </span><span
    class="content">”Project” in the ”Pricing” drop down menu in the
    toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Pricing settings” dialog will appear
    (Figure [1.12.1.2](#x1-60002r2)). Select a supplier from the list
    and click ”Get quote”. A representative from Genome Compiler will
    contact you to continue with the ordering process.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-60002r2"></span>
    ![PIC](pictures/pricing_ordering_screenshots/pricing_project.png)\
    <div class="caption">

    <span class="id">Figure 1.12.1.2: </span><span class="content">The
    ”Pricing settings” dialog for projects.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.12.2 </span> <span id="x1-610001.12.2"></span>Pricing for Primers {.subsubsectionHead}

-   To get pricing for primers, go to the ”Pricing” icon in the toolbar
    and click ”Primers...” (Figure [1.12.2.1](#x1-61001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-61001r1"></span>
    ![PIC](pictures/pricing_ordering_screenshots/pricing_select_primers.png)\
    <div class="caption">

    <span class="id">Figure 1.12.2.1: </span><span
    class="content">”Primers” in the ”Pricing” drop down menu in the
    toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Primers settings” dialog will appear
    (Figure [1.12.2.2](#x1-61002r2)). Select the primers you would like
    to order from the list and click ”Pricing”.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-61002r2"></span>
    ![PIC](pictures/pricing_ordering_screenshots/settings_primers.png)\
    <div class="caption">

    <span class="id">Figure 1.12.2.2: </span><span class="content">The
    ”Primers settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Pricing settings” dialog will appear
    (Figure [1.12.2.3](#x1-61003r3)). Select a supplier from the list
    and click ”Next”. This will redirect you to the website of the
    selected supplier.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-61003r3"></span>
    ![PIC](pictures/pricing_ordering_screenshots/pricing_primers.png)\
    <div class="caption">

    <span class="id">Figure 1.12.2.3: </span><span class="content">The
    ”Pricing settings” dialog for primers.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.13 </span> <span id="x1-620001.13"></span>Compilation {.subsectionHead}

The compilation engine allows you to debug sequences. It encompasses
specific synthesis manufacturability rules as well as standard synthetic
biology constraints.

##### <span class="titlemark">1.13.1 </span> <span id="x1-630001.13.1"></span>Selecting Rules for Activation {.subsubsectionHead}

-   Before you start using the compilation engine, you should select the
    rules you would like to use. Rules can be selected from the
    “Compilation Settings” dialog (Figure [1.13.1.3](#x1-63003r3)). It
    can be accessed primarily by clicking on the corresponding
    “Settings” icon from the toolbar drop down Layers menu
    (Figure [1.13.1.1](#x1-63001r1)), or by clicking the “Settings” icon
    on the far right in the “Compilation” tab after a compilation has
    been carried out (Figure [1.13.1.2](#x1-63002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-63001r1"></span>
    ![PIC](pictures/compilation_screenshots/compilation_settings_toolbar.png)\
    <div class="caption">

    <span class="id">Figure 1.13.1.1: </span><span class="content">The
    ”Compilation Settings” dialog icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-63002r2"></span>
    ![PIC](pictures/compilation_screenshots/compilation_tab_annotated.png)\
    <div class="caption">

    <span class="id">Figure 1.13.1.2: </span><span
    class="content">”Settings” icons in the ”Compilation” summary
    table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Rules are divided into two groups: “design rules” and “synthesis
    rules”. These are predefined design rules, as well as
    manufacturability rules specific to the main DNA synthesis
    providers. You can request a new rule by emailing one of our
    customer service representatives.
-   You can select the rules you would like to apply by checking and
    unchecking their box in the dialog (Figure [1.13.1.3](#x1-63003r3)).
-   To ignore warnings, simply click the “Ignore warnings” button at the
    top of the dialog box (Figure [1.13.1.3](#x1-63003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-63003r3"></span>
    ![PIC](pictures/compilation_screenshots/compilation_settings_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.13.1.3: </span><span class="content">The
    “Compilation Settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.13.2 </span> <span id="x1-640001.13.2"></span>Compiling Projects {.subsubsectionHead}

-   To compile your project, click the compilation icon in the toolbar
    (Figure [1.13.2.1](#x1-64001r1)), or right click on your project
    canvas and click “Compile project” (Figure [1.13.2.2](#x1-64002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-64001r1"></span>
    ![PIC](pictures/compilation_screenshots/toolbar_compilation.png)\
    <div class="caption">

    <span class="id">Figure 1.13.2.1: </span><span class="content">The
    compilation icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-64002r2"></span>
    ![PIC](pictures/compilation_screenshots/right_click_compile.png)\
    <div class="caption">

    <span class="id">Figure 1.13.2.2: </span><span
    class="content">”Compile project” in the right click drop down
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.13.3 </span> <span id="x1-650001.13.3"></span>Error and Warning Indicators {.subsubsectionHead}

-   Errors and warnings correspond to violation of design rules. Errors
    (red X flags) and warnings (yellow exclamation mark flags) will
    appear in the sequence (Figure [1.13.3.2](#x1-65003r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-65003r2"></span>
    ![PIC](pictures/compilation_screenshots/error.png) <span
    id="x1-65001r1"></span> <span class="cmr-9">Error.</span>  

    ![PIC](pictures/compilation_screenshots/warning.png) <span
    id="x1-65002r2"></span> <span class="cmr-9">Warning.</span>\

    <div class="caption">

    <span class="id">Figure 1.13.3.2: </span><span
    class="content">Errors and warnings.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.13.4 </span> <span id="x1-660001.13.4"></span>Compilation Summary Table {.subsubsectionHead}

-   A summary of the errors and warnings in the project is displayed in
    the “Compilation” summary table (Figure [1.13.4.2](#x1-66002r2)) at
    the bottom of the screen. This table can be accessed by clicking the
    corresponding icon from the toolbar drop down Layers menu
    (Figure [1.13.4.1](#x1-66001r1)). You can remove the viewing of
    errors and warnings from the different abstraction layers views by
    unchecking the compilation box from the toolbar drop down Layers
    menu.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-66001r1"></span>
    ![PIC](pictures/compilation_screenshots/access_track_summary_table.png)\
    <div class="caption">

    <span class="id">Figure 1.13.4.1: </span><span class="content">The
    ”Compilation” summary table icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-66002r2"></span>
    ![PIC](pictures/compilation_screenshots/compilation_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.13.4.2: </span><span class="content">The
    ”Compilation” summary table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.13.5 </span> <span id="x1-670001.13.5"></span>Navigating to Errors and Warnings {.subsubsectionHead}

-   You can navigate to each error or warning in the DNA sequence by
    clicking the desired rule violation in the “Compilation” summary
    table (Figure [1.13.4.2](#x1-66002r2)) or by using the project
    scroll bar on the right (Figure [1.13.5.1](#x1-67001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-67001r1"></span>
    ![PIC](pictures/compilation_screenshots/scrollbar_compilation.png)\
    <div class="caption">

    <span class="id">Figure 1.13.5.1: </span><span class="content">The
    scroll bar allows navigation to the errors and warnings.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.14 </span> <span id="x1-680001.14"></span>Back Translation {.subsectionHead}

This feature allows you to back translate according to standard
translation tables or codon usage.

-   Translation can occur in four different situations:
    -   Back translation of a whole project
    -   Back translation of a part/gene
    -   Drag & drop a part/gene/AA/DNA
    -   Copy & paste part/gene/AA/DNA

##### <span class="titlemark">1.14.1 </span> <span id="x1-690001.14.1"></span>How to Start Translating {.subsubsectionHead}

-   To back translate a whole project, go to the “Project” tab
    (Figure [1.14.1.1](#x1-69001r1)) at the bottom of the opened
    project. You can select the appropriate host organism and
    translation table from the ‘Translation Settings’ dialog which will
    appear upon selecting the host organism.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-69001r1"></span>
    ![PIC](pictures/translation_screenshots/project_tab_host_organism.png)\
    <div class="caption">

    <span class="id">Figure 1.14.1.1: </span><span class="content">The
    ”Project” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To back translate a part or a gene, right click on it and select
    “Back Translate…” (Figure [1.14.1.2](#x1-69002r2)). This will take
    you to the “Translation Settings” dialog
    (Figure [1.14.2.2](#x1-70003r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-69002r2"></span>
    ![PIC](pictures/translation_screenshots/right_click_back_translate.png)\
    <div class="caption">

    <span class="id">Figure 1.14.1.2: </span><span class="content">”Back
    Translate...” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Dragging and dropping sequence, or copying and pasting sequence from
    outside the project will also prompt the ”Translation Settings”
    dialog (Figure [1.14.2.2](#x1-70003r2)).

##### <span class="titlemark">1.14.2 </span> <span id="x1-700001.14.2"></span>“Translation Settings” Dialog {.subsubsectionHead}

-   You can translate according to codon usage or to a translation table
    (Figure [1.14.2.2](#x1-70003r2)). For codon usage, select the
    appropriate organism from the drop down menu. Similarly, you can
    select the appropriate translation table from the corresponding drop
    down menu.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-70003r2"></span>
    ![PIC](pictures/translation_screenshots/translation_settings_dialog_codon_usage.png)
    <span id="x1-70001r1"></span> <span class="cmr-9">Codon
    usage.</span>  

    ![PIC](pictures/translation_screenshots/translation_settings_dialog_transation_table.png)
    <span id="x1-70002r2"></span> <span class="cmr-9">Translation
    table.</span>\

    <div class="caption">

    <span class="id">Figure 1.14.2.2: </span><span class="content">The
    “Translation Settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   In the case of translation of a strand of DNA, you can also choose
    to keep the DNA as original (Figure [1.14.2.3](#x1-70004r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-70004r3"></span>
    ![PIC](pictures/translation_screenshots/translation_settings_dna_as_original.png)\
    <div class="caption">

    <span class="id">Figure 1.14.2.3: </span><span class="content">The
    “Translation Settings” dialog: keep the DNA as original.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Once you click the “Don’t ask again” box, translation will occur
    automatically according to your settings when you drag and drop or
    copy and paste a sequence.

##### <span class="titlemark">1.14.3 </span> <span id="x1-710001.14.3"></span>Proportion of each triplet {.subsubsectionHead}

-   After a back translation, the percentages of each triplet for the
    organism are shown in the amino acid drop down menu
    (Figure [1.14.3.1](#x1-71001r1)) as well as in the amino acid
    combinatorial wheel (Figure [1.14.3.2](#x1-71002r2)) and the amino
    acid mutation wheel.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-71001r1"></span>
    ![PIC](pictures/translation_screenshots/percentages_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.14.3.1: </span><span
    class="content">Percentages in the amino acid drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-71002r2"></span>
    ![PIC](pictures/translation_screenshots/percentages_wheel.png)\
    <div class="caption">

    <span class="id">Figure 1.14.3.2: </span><span
    class="content">Percentages shown in the amino acid combinatorial
    wheel.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.15 </span> <span id="x1-720001.15"></span>Combinatorial Design {.subsectionHead}

The combinatorial design feature allows you to easily generate and
filter multiple combinations within your construct.

##### <span class="titlemark">1.15.1 </span> <span id="x1-730001.15.1"></span>Circuit Part Combinatorial {.subsubsectionHead}

-   Drag and drop parts on the part that you would like to change. You
    can add part alternatives to as many different parts in the project
    as desired. The ”Translation Settings” dialog will pop up
    (Figure [1.15.1.1](#x1-73001r1)). You can choose to translate your
    sequence according to codon usage or to a translation table or to
    keep the DNA as original (more details about the translation feature
    can be found in section [1.14](#x1-680001.14)). In the example
    below, the sequence is translated according to codon usage in E.
    coli.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-73001r1"></span>
    ![PIC](pictures/combinatorial_screenshots/translation_settings_dialog_codon_usage.png)\
    <div class="caption">

    <span class="id">Figure 1.15.1.1: </span><span class="content">The
    “Translation Settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   A dropdown arrow will appear on the part. Click on it to see the
    dropdown menu showing all the different sequences
    (Figure [1.15.1.2](#x1-73002r2)). To view a summary of all
    combinations, click ”Combinatorial Summary...”.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-73002r2"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_parts_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.15.1.2: </span><span class="content">The
    dropdown menu showing the different parts.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Combinatorial Summary - Parts” will appear
    (Figure [1.15.1.3](#x1-73003r3)). It displays the different
    combinations, which can be selected. You can choose which part to
    view in the main view by selecting it and clicking ”Set as Current”.
    Get a pricing of your projects by clicking ”Pricing”. The statistics
    of your selection are shown at the bottom of the dialog. On
    exhaustive mode, the library consists of all the possible
    combinations, whereas in one step mutation mode, the library
    consists of all combinations whereby only one alternative is
    replaced each time.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-73003r3"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_summary_parts.png)\
    <div class="caption">

    <span class="id">Figure 1.15.1.3: </span><span class="content">The
    ”Combinatorial Summary - Parts”.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.15.2 </span> <span id="x1-740001.15.2"></span>Protein Engineering Combinatorial {.subsubsectionHead}

-   To test different amino acids on one location, either click the drop
    down arrow below the amino acid and select ”Combinatorial...”
    (Figure [1.15.2.1](#x1-74001r1)) or right click on the amino acid
    and select ”Combinatorial...” from the drop down menu
    (Figure [1.15.2.2](#x1-74002r2)). You can add amino acid
    alternatives to as many different amino acids in the part as
    desired.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-74001r1"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.15.2.1: </span><span
    class="content">”Combinatorial...” in the drop down arrow
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-74002r2"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_right_click.png)\
    <div class="caption">

    <span class="id">Figure 1.15.2.2: </span><span
    class="content">”Combinatorial...” in the right click dropdown
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Select amino acids from the combinatorial wheel
    (Figure [1.15.2.3](#x1-74003r3)) and click the green tick. You can
    filter selection of amino acids with a specific property by checking
    the property checkbox on the left.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-74003r3"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_aa.png)\
    <div class="caption">

    <span class="id">Figure 1.15.2.3: </span><span class="content">The
    amino acids combinatorial wheel.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   A drop down arrow will appear on the amino acid. Click on it to see
    the drop down menu showing all the different amino acids
    (Figure [1.15.2.4](#x1-74004r4)). To view a summary of all
    combinations, click ”Combinatorial Summary...”.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-74004r4"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_aa_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.15.2.4: </span><span class="content">The
    drop down menu showing the different amino acids.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Combinatorial Summary - Proteins” will appear
    (Figure [1.15.2.5](#x1-74005r5)). It has the same features as for
    parts.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-74005r5"></span>
    ![PIC](pictures/combinatorial_screenshots/combinatorial_summary_proteins.png)\
    <div class="caption">

    <span class="id">Figure 1.15.2.5: </span><span class="content">The
    ”Combinatorial Summary - Proteins”.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.16 </span> <span id="x1-750001.16"></span>Managing Restriction Sites {.subsectionHead}

The “Restriction Sites” dialog allows you to manage restriction enzymes.
You can filter them for different criteria and save presets for further
use.

##### <span class="titlemark">1.16.1 </span> <span id="x1-760001.16.1"></span>Opening the Dialog {.subsubsectionHead}

-   To open the “Restriction Sites” dialog, click on the Layers menu in
    the toolbar and select the “Restriction sites” settings from the
    drop down menu (Figure [1.16.1.1](#x1-76001r1)). Selected cut sites
    will be visible throughout the different abstraction layers views
    (Figure [1.16.1.3](#x1-76005r3)). To remove them from the views,
    uncheck the corresponding checkbox in the Layers menu.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-76001r1"></span>
    ![PIC](pictures/restriction_sites_dialog_screenshots/dropdown_menu_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.16.1.1: </span><span class="content">The
    ”Restriction Sites” settings icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-76005r3"></span>
    ![PIC](pictures/digest_screenshots/plasmid_view.png) <span
    id="x1-76002r1"></span> <span class="cmr-9">Circular view.</span>  

    ![PIC](pictures/digest_screenshots/parts_view.png) <span
    id="x1-76003r2"></span> <span class="cmr-9">Linear view.</span>  

    ![PIC](pictures/digest_screenshots/dna_view.png) <span
    id="x1-76004r3"></span> <span class="cmr-9">DNA view.</span>\

    <div class="caption">

    <span class="id">Figure 1.16.1.3: </span><span class="content">Cut
    sites in the different views.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.16.2 </span> <span id="x1-770001.16.2"></span>”Restriction Site” Settings Dialog {.subsubsectionHead}

-   The “Restriction Site” settings dialog
    (Figure [1.16.2.1](#x1-77001r1)) will appear, displaying the
    following information about the restriction enzymes:
    -   Number of cuts in the project
    -   Length of the recognition sequence
    -   Hang size
    -   Ambiguity, i.e. whether different sequences can be recognized by
        the enzyme
-   In order to display the restriction enzymes cut sites in the
    project, select the check box for each enzyme in the main table and
    they will be automatically added to the ”Selected” list on the
    right. You can remove enzymes from the list by unchecking their
    checkbox or clicking on their “Delete” icon.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-77001r1"></span>
    ![PIC](pictures/restriction_sites_dialog_screenshots/restriction_sites_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.16.2.1: </span><span class="content">The
    ”Restriction Sites” settings dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The drop down menu from each column heading can be used to filter
    your enzyme selection. In the example below
    (Figure [1.16.2.2](#x1-77002r2)), we selected for double cutters
    only, so that only double cutters will appear on the list.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-77002r2"></span>
    ![PIC](pictures/restriction_sites_dialog_screenshots/double_cutters_only.png)\
    <div class="caption">

    <span class="id">Figure 1.16.2.2: </span><span
    class="content">Example: show double cutters only.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The “Preset” drop down menu allows you to switch between presets. To
    create a new preset, select restriction enzymes (they will appear on
    the right) and click save. A dialog will pop up
    (Figure [1.16.2.3](#x1-77003r3)) to prompt you for a new ”Preset
    name”. To delete a preset, click the corresponding delete icon .

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-77003r3"></span>
    ![PIC](pictures/restriction_sites_dialog_screenshots/new_preset.png)\
    <div class="caption">

    <span class="id">Figure 1.16.2.3: </span><span class="content">The
    ”Save Preset” dialog (forefront).</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The small drop down arrow at the far right of the main table for
    each enzyme displays the restriction site locations
    (Figure [1.16.2.4](#x1-77004r4)). Clicking on the locations will
    take you to the corresponding area in the sequence view and
    highlight the location in the plasmid view.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-77004r4"></span>
    ![PIC](pictures/restriction_sites_dialog_screenshots/restriction_site_locations_dropdown.png)\
    <div class="caption">

    <span class="id">Figure 1.16.2.4: </span><span class="content">The
    ”Restriction Sites” dialog: locations of the restriction
    sites.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   If you check the “Show 1 cutters only” box at the bottom left, only
    1 cutters will be visible throughout the different views.

#### <span class="titlemark">1.17 </span> <span id="x1-780001.17"></span>Virtual Digest {.subsectionHead}

This feature allows you to perform a hypothetical digest on a sequence
selection or on the whole of your sequence.

##### <span class="titlemark">1.17.1 </span> <span id="x1-790001.17.1"></span>Running a Digest {.subsubsectionHead}

-   Right click on your project canvas or the project tab and select
    “Run Digest” (Figure [1.17.1.1](#x1-79001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-79001r1"></span>
    ![PIC](pictures/digest_screenshots/right_click_digest.png)\
    <div class="caption">

    <span class="id">Figure 1.17.1.1: </span><span class="content">”Run
    Digest” in the right click drop down menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.17.2 </span> <span id="x1-800001.17.2"></span>“Restriction Sites” Window {.subsubsectionHead}

-   The “Restriction Sites” window will appear
    (Figure [1.17.2.1](#x1-80001r1)). Select the desired restriction
    enzymes for your digest. This window works in a similar fashion to
    the “Restriction Sites” dialog (see section [1.16](#x1-750001.16)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-80001r1"></span>
    ![PIC](pictures/digest_screenshots/restriction_sites_window.png)\
    <div class="caption">

    <span class="id">Figure 1.17.2.1: </span><span class="content">The
    “Restriction Sites” window.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.17.3 </span> <span id="x1-810001.17.3"></span>”Digest” Summary Table {.subsubsectionHead}

-   Click “Run Digest” and all the information regarding digested
    fragment sizes will be displayed in the “Digest” summary table at
    the bottom of your screen (Figure [1.17.3.1](#x1-81001r1)). The
    selected cut sites now appear on the different abstraction layer
    views (Figure [1.17.3.3](#x1-81005r3)). You can navigate to the DNA
    view of the different sites by clicking in the corresponding line of
    the “Digest” summary table.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-81001r1"></span>
    ![PIC](pictures/digest_screenshots/digest_tab.png)\
    <div class="caption">

    <span class="id">Figure 1.17.3.1: </span><span class="content">The
    “Digest” summary table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-81005r3"></span>
    ![PIC](pictures/digest_screenshots/plasmid_view.png) <span
    id="x1-81002r1"></span> <span class="cmr-9">Circular view.</span>  

    ![PIC](pictures/digest_screenshots/parts_view.png) <span
    id="x1-81003r2"></span> <span class="cmr-9">Linear view.</span>  

    ![PIC](pictures/digest_screenshots/dna_view.png) <span
    id="x1-81004r3"></span> <span class="cmr-9">DNA view.</span>\

    <div class="caption">

    <span class="id">Figure 1.17.3.3: </span><span class="content">Cut
    sites in the different views.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.18 </span> <span id="x1-820001.18"></span>Primer Design {.subsectionHead}

The Primer Design tool allows you to design primers in an easy and
efficient way.

##### <span class="titlemark">1.18.1 </span> <span id="x1-830001.18.1"></span>Opening the Primer Design Dialog {.subsubsectionHead}

-   You can access the Primer Design dialog in two ways. Select a
    sequence, right click and select ”Create primer from selection” from
    the drop down menu, or click the Primer icon in the Layers menu and
    select ”Create primer”. The former will lead to auto-filled primer
    sequence in the dialog.

##### <span class="titlemark">1.18.2 </span> <span id="x1-840001.18.2"></span>Primer Design Dialog {.subsubsectionHead}

-   The Primer Design dialog (Figure [1.18.2.1](#x1-84001r1)) is divided
    into three sections. Each section is detailed below.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-84001r1"></span>
    ![PIC](pictures/primer_design_screenshots/create_primer_dialog.png)\
    <div class="caption">

    <span class="id">Figure 1.18.2.1: </span><span class="content">The
    Primer Design dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can edit the sequence ID, and add a description. Select the
    strand for your primer. If the sequence is not yet filled, click
    ”Use selection”. This will take you to your project where you can
    select a sequence. You can edit the 3’ location
    (Figure [1.18.2.2](#x1-84002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-84002r2"></span>
    ![PIC](pictures/primer_design_screenshots/manual_primer_design_dialog_top.png)\
    <div class="caption">

    <span class="id">Figure 1.18.2.2: </span><span class="content">The
    “Manual Primer Design” dialog: sequence ID, description, strand and
    3’ location.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   For PCR based cloning, you can add restriction ”Cut Sites” to your
    primers from the drop down menu, followed by a ”Leader Sequence”
    (Figure [1.18.2.3](#x1-84003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-84003r3"></span>
    ![PIC](pictures/primer_design_screenshots/pcr_based_cloning.png)\
    <div class="caption">

    <span class="id">Figure 1.18.2.3: </span><span class="content">The
    “Manual Primer Design” dialog: PCR based cloning.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The verification field displays the melting temperature (calculated
    according to [<span class="cmtt-10">SantaLucia
    1998</span>](http://www.pnas.org/content/95/4/1460.full)), the GC
    content as well as the length of your primers
    (Figure [1.18.2.4](#x1-84004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-84004r4"></span>
    ![PIC](pictures/primer_design_screenshots/verification_field.png)\
    <div class="caption">

    <span class="id">Figure 1.18.2.4: </span><span class="content">The
    “Manual Primer Design” dialog: verification.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Click ”OK” will save your primers to the primer library for the
    project under the single primer tab.

##### <span class="titlemark">1.18.3 </span> <span id="x1-850001.18.3"></span>Primer Summary Table {.subsubsectionHead}

-   To view a summary table of all primers, click the primers “Summary
    table” icon in the toolbar drop down Layers menu
    (Figure [1.18.3.1](#x1-85001r1)). You can remove the primers from
    the views by unchecking the box.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-85001r1"></span>
    ![PIC](pictures/primer_design_screenshots/primers_dropdown_menu.png)\
    <div class="caption">

    <span class="id">Figure 1.18.3.1: </span><span class="content">The
    primers ”Summary table” icon in the toolbar drop down Layers
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The primers summary table (Figure [1.18.3.2](#x1-85002r2)) displays
    all the primers from the project, as well as their name,
    description, strand, GC content, melting temperature (calculated
    according to [<span class="cmtt-10">SantaLucia
    1998</span>](http://www.pnas.org/content/95/4/1460.full)), location,
    length and sequence. You can navigate to each primer in the DNA
    sequence by clicking on it. Click the ”Edit” icon to edit a primer,
    or the ”Delete” icon to delete it.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-85002r2"></span>
    ![PIC](pictures/primer_design_screenshots/primers_summary.png)\
    <div class="caption">

    <span class="id">Figure 1.18.3.2: </span><span class="content">The
    primers summary table.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">1.19 </span> <span id="x1-860001.19"></span>Cloning Wizard {.subsectionHead}

The Cloning Wizard tool allows you to design and prepare your cloning
experiments in an easy and efficient way.

##### <span class="titlemark">1.19.1 </span> <span id="x1-870001.19.1"></span>Opening the Cloning Wizard {.subsubsectionHead}

-   To open the ”Cloning Wizard”, click the corresponding icon in the
    toolbar (Figure [1.19.1.1](#x1-87001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-87001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/clone_icon.png)\
    <div class="caption">

    <span class="id">Figure 1.19.1.1: </span><span class="content">The
    “Cloning Wizard” icon in the toolbar.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.19.2 </span> <span id="x1-880001.19.2"></span>Cloning Procedure {.subsubsectionHead}

-   Type in a ”Project name”. ”Restriction Ligation” is selected as
    ”Cloning procedure” (Figure [1.19.2.1](#x1-88001r1)). More cloning
    procedure as well as the ”Require cloning services” option will be
    available soon.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-88001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/cloning_procedure.png)\
    <div class="caption">

    <span class="id">Figure 1.19.2.1: </span><span class="content">The
    ”Cloning Procedure” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.19.3 </span> <span id="x1-890001.19.3"></span>In-house Cloning {.subsubsectionHead}

-   This flow is for in-house cloning, which means that you will
    construct the vector yourself.

##### <span class="titlemark">1.19.4 </span> <span id="x1-900001.19.4"></span>In-house Cloning: Destination Vector {.subsubsectionHead}

-   To select a vector, drag and drop a file from the Materials box
    (Figure [1.19.4.1](#x1-90001r1)). This section will already be
    filled if you selected your vector to open the Cloning Wizard.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-90001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/select_vector.png)\
    <div class="caption">

    <span class="id">Figure 1.19.4.1: </span><span class="content">The
    ”Destination Vector” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The vector will appear on the right, and its meta data will be
    displayed on the left (Figure [1.19.4.2](#x1-90002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-90002r2"></span>
    ![PIC](pictures/cloning_wizard_screenshots/destination_vector.png)\
    <div class="caption">

    <span class="id">Figure 1.19.4.2: </span><span class="content">The
    ”Destination Vector” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.19.5 </span> <span id="x1-910001.19.5"></span>In-house Cloning: Select Insert {.subsubsectionHead}

-   The ”Select Insert” step enables the selection of the insert
    sequence for cloning into the destination vector. You can choose to
    add an insert sequence of amino acids or DNA. To add your insert,
    drag and drop a file from the Materials box. You can reverse
    complement the sequence by selecting the other strand orientation
    (Figure [1.19.5.1](#x1-91001r1)). This section will already be
    filled if you selected your insert to open the Cloning Wizard.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-91001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/add_insert.png)\
    <div class="caption">

    <span class="id">Figure 1.19.5.1: </span><span class="content">The
    ”Select Insert” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You can codon optimize the insert or, in the case of a DNA
    insertion, choose to keep the DNA as it is. Select the appropriate
    option by checking its checkbox. If you would like to codon optimize
    the insert, select the host organism from the drop down menu. You
    can also select specific sites to avoid
    (Figure [1.19.5.2](#x1-91002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-91002r2"></span>
    ![PIC](pictures/cloning_wizard_screenshots/codon_optimize.png)\
    <div class="caption">

    <span class="id">Figure 1.19.5.2: </span><span class="content">The
    ”Select Insert” tab: codon optimization.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   You will then need to specify the insert region by selecting
    restriction sites in the destination vector
    (Figure [1.19.5.3](#x1-91003r3)). You can configure the restriction
    site list by opening the restriction site dialog where a preset can
    be configured. Your desired sites preset will automatically be added
    to the ”start” and ”end” restriction sites list for you to select.
    The preset will also be displayed on the vector in the right panel.
    Single and double cutters in the destination vector will be shown in
    black and other enzymes will be greyed out. As the restriction sites
    are selected they will appear in the mining cloning map along the
    bottom of the left panel (Figure [1.19.5.3](#x1-91003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-91003r3"></span>
    ![PIC](pictures/cloning_wizard_screenshots/restriction_sites_selected.png)\
    <div class="caption">

    <span class="id">Figure 1.19.5.3: </span><span class="content">The
    ”Select Insert” tab: cut sites.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.19.6 </span> <span id="x1-920001.19.6"></span>In-house Cloning: Design Insert {.subsubsectionHead}

-   The ”Design Insert” step defines how the insert sequence is
    retrieved, either by submitting an ”Order” from a DNA synthesis
    provider or choosing to ”PCR the insert” out from a source plasmid.
    Select the option you would like to use by checking its checkbox
    (Figure [1.19.6.1](#x1-92001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-92001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/order_pcr_insert.png)\
    <div class="caption">

    <span class="id">Figure 1.19.6.1: </span><span class="content">The
    ”Design Insert” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ##### <span class="titlemark">1.19.7 </span> <span id="x1-930001.19.7"></span>Design Insert: Order Insert {.subsubsectionHead}

-   Flanking sticky ends can be added to the ends of the insert sequence
    by adding a ”5’ location restriction site” and a ”3’ location
    restriction site.” The drop down menu of restriction sites consists
    of a smart list of enzymes, filtered according to compatible sticky
    ends to those prior selected in the destination vector. An optional
    ”Leader sequence” can be added to ensure efficiency during
    restriction enzyme digestion (Figure [1.19.7.1](#x1-93001r1)).
-   As the flanking sequences are selected, they are added on either end
    of the insert sequence in the right panel. The insert sequence in
    the right panel is editable.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-93001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/order_insert.png)\
    <div class="caption">

    <span class="id">Figure 1.19.7.1: </span><span class="content">The
    ”Design Insert” tab: order insert.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

    ##### <span class="titlemark">1.19.8 </span> <span id="x1-940001.19.8"></span>Design Insert: PCR Insert {.subsubsectionHead}

-   If you decide to PCR your insert, then you can design PCR primers
    with compatible sticky ends for amplifying your vector from the
    source plasmid. When you require a source plasmid, you can use a
    plasmid from your inventory or order a plasmid. Check the
    appropriate checkbox according to your choice. The vector will then
    be automatically added to the cart in the next step. To add the
    plasmid, drag and drop a file from the Materials box
    (Figure [1.19.8.1](#x1-94001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-94001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/source_plasmid.png)\
    <div class="caption">

    <span class="id">Figure 1.19.8.1: </span><span class="content">The
    ”Source plasmid” drop down in the ”Design Insert” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   For manual design of primers fill in all the fields like the regular
    primer dialog flow (Figure [1.19.8.2](#x1-94002r2)). The drop down
    menu for cut sites again consists of a smart list of enzymes,
    filtered according to compatible sticky ends to those prior selected
    in the destination vector. If the leader sequence adds a restriction
    site then you will be warned.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-94002r2"></span>
    ![PIC](pictures/cloning_wizard_screenshots/design_primers.png)\
    <div class="caption">

    <span class="id">Figure 1.19.8.2: </span><span class="content">The
    ”Design Insert” tab: design primers.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   Selected sites and primer sequences are highlighted on the plasmid
    on the right panel (Figure [1.19.8.3](#x1-94003r3)) and in the
    cloning minimap at the bottom of the left panel.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-94003r3"></span>
    ![PIC](pictures/cloning_wizard_screenshots/primers_overview.png)\
    <div class="caption">

    <span class="id">Figure 1.19.8.3: </span><span class="content">The
    ”Design Insert” tab: design primers.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Verification” drop down displays information about your primers
    (Figure [1.19.8.4](#x1-94004r4)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-94004r4"></span>
    ![PIC](pictures/cloning_wizard_screenshots/verification.png)\
    <div class="caption">

    <span class="id">Figure 1.19.8.4: </span><span class="content">The
    ”Verification” drop down in the ”Design Insert” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">1.19.9 </span> <span id="x1-950001.19.9"></span>In-house Cloning: Finalize {.subsubsectionHead}

-   The ”Finalize” step displays a summary of the cloning procedure and
    allows you to add elements to your cart and select providers by
    clicking the ”Cart” icon (Figure [1.19.9.1](#x1-95001r1)). They can
    also be exported in text format for ease of copying and pasting into
    your preferred DNA synthesis providers website.

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-95001r1"></span>
    ![PIC](pictures/cloning_wizard_screenshots/cloning_summary.png)\
    <div class="caption">

    <span class="id">Figure 1.19.9.1: </span><span class="content">The
    cloning summary in the ”Finalize” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The ”Compilation summary” displays prospective Synthesis and Design
    errors (Figure [1.19.9.2](#x1-95002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-95002r2"></span>
    ![PIC](pictures/cloning_wizard_screenshots/compilation_summary.png)\
    <div class="caption">

    <span class="id">Figure 1.19.9.2: </span><span class="content">The
    ”Compilation summary” drop down in the ”Finalize” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The final cloned product is visible on the right panel and the
    cloning minimap, displaying the inserted sequence inside the
    destination vector. Select ”Finish” to create your new cloning
    project.

### <span class="titlemark">2 </span> <span id="x1-960002"></span>Flows {.sectionHead}

#### <span class="titlemark">2.1 </span> <span id="x1-970002.1"></span>Sequence Annotation {.subsectionHead}

This simple flow describes the process of manual sequence annotation.

##### <span class="titlemark">2.1.1 </span> <span id="x1-980002.1.1"></span>Pasting your Sequence {.subsubsectionHead}

-   Paste your sequence into a new project. The ”Translation Settings”
    dialog (Figure [2.1.1.1](#x1-98001r1)) will pop up. Choose the
    appropriate option. In the example below, the DNA is kept as
    original. You can find more information about the translation
    feature in section [1.14](#x1-680001.14).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-98001r1"></span>
    ![PIC](pictures/annotate_sequence_screenshots/translation_settings_dna_original.png)\
    <div class="caption">

    <span class="id">Figure 2.1.1.1: </span><span class="content">The
    “Translation Settings” dialog.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   The whole sequence will be set as CDS by default
    (Figure [2.1.1.2](#x1-98002r2)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-98002r2"></span>
    ![PIC](pictures/annotate_sequence_screenshots/pasted_sequence.png)\
    <div class="caption">

    <span class="id">Figure 2.1.1.2: </span><span class="content">The
    sequence set as CDS.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   To change the feature key, go to the ”Part” tab at the bottom of the
    screen and select a feature from the ”Feature key” drop down menu
    (Figure [2.1.1.3](#x1-98003r3)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-98003r3"></span>
    ![PIC](pictures/annotate_sequence_screenshots/change_feature_key.png)\
    <div class="caption">

    <span class="id">Figure 2.1.1.3: </span><span class="content">The
    ”Feature Key” drop down menu in the ”Part” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">2.1.2 </span> <span id="x1-990002.1.2"></span>Annotating your Sequence {.subsubsectionHead}

-   To annotate a sequence, go to the DNA view, select it, right click
    and select ”Annotate selection” from the drop down menu
    (Figure [2.1.2.1](#x1-99001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-99001r1"></span>
    ![PIC](pictures/annotate_sequence_screenshots/annotate_selection.png)\
    <div class="caption">

    <span class="id">Figure 2.1.2.1: </span><span
    class="content">”Annotate selection” from the right click drop down
    menu.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

-   A new part will be created from your selection. It will be set as a
    CDS by default. You can change the feature key as described above.

##### <span class="titlemark">2.1.3 </span> <span id="x1-1000002.1.3"></span>Renaming your part {.subsubsectionHead}

-   To rename a part, type in a name in the ”Name” section of the ”Part”
    tab (Figure [2.1.3.1](#x1-100001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-100001r1"></span>
    ![PIC](pictures/annotate_sequence_screenshots/renaming_part.png)\
    <div class="caption">

    <span class="id">Figure 2.1.3.1: </span><span class="content">The
    ”Name” section in the ”Part” tab.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

#### <span class="titlemark">2.2 </span> <span id="x1-1010002.2"></span>Gene Amplification {.subsectionHead}

This simple flow describes the process of gene amplification.

##### <span class="titlemark">2.2.1 </span> <span id="x1-1020002.2.1"></span>Finding your Gene of Interest {.subsubsectionHead}

-   Find your gene of interest. This can be done:
    -   via our NCBI search tool (see section [1.8](#x1-390001.8))
    -   by finding information about the gene location in an article and
        using our ”Go to” feature (see section [1.5](#x1-310001.5))
    -   by using our project search tool (see
        section [1.7](#x1-370001.7))
    -   or by importing a file (see section [1.3](#x1-170001.3))
-   In this example, the yjhQ gene is found via the project search tool
    (Figure [2.2.1.1](#x1-102001r1)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-102001r1"></span>
    ![PIC](pictures/gene_amplification_screenshots/yjhq_search.png)\
    <div class="caption">

    <span class="id">Figure 2.2.1.1: </span><span class="content">The
    yjhQ gene.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">2.2.2 </span> <span id="x1-1030002.2.2"></span>Commenting the Part {.subsubsectionHead}

-   Add a comment to the part (Figure [2.2.2.1](#x1-103001r1)) by right
    clicking on the part and selecting ”Add comment” in the drop down
    menu (see section [1.10](#x1-480001.10)).

    ------------------------------------------------------------------------

    <div class="figure">

    <span id="x1-103001r1"></span>
    ![PIC](pictures/gene_amplification_screenshots/gene_comment.png)\
    <div class="caption">

    <span class="id">Figure 2.2.2.1: </span><span
    class="content">Comment on the yjhQ gene.</span>

    </div>

    </div>

    ------------------------------------------------------------------------

##### <span class="titlemark">2.2.3 </span> <span id="x1-1040002.2.3"></span>Designing Primers {.subsubsectionHead}

-   Design primers (see section [1.18](#x1-820001.18)).

##### <span class="titlemark">2.2.4 </span> <span id="x1-1050002.2.4"></span>Ordering {.subsubsectionHead}

-   Place your order (see section [1.12](#x1-590001.12)).

</div>

</div>
